<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.5.1/jquery-3.5.1.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<script>
  window.onload = choosePic;
  var myPix = new Array("images/image1.jpg","images/image2.jpg","images/image3.jpg","images/image4.jpg","images/image5.jpg","images/image6.jpg","images/image7.jpg","images/image8.jpg","images/image8.jpg","images/image10.jpg");
  function choosePic() {
     var randomNum = Math.floor(Math.random() * myPix.length);
     document.getElementById("myPicture").src = myPix[randomNum];
  }
</script>

<div class="navbar navbar-default  navbar-float-top" role="navigation">
    <div class="container" overflow="auto" align="left" style="width: inherit;">
      <img id="myPicture" alt="Brand" src="" align='center'>
    </div>

  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>

      </button>

      <a class="navbar-brand" href="index.html"></a>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav" align="center">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs/wiki">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
        <li><a href="press.html">Press</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<title>
<span class="math inline">\(if(title-prefix)\)</span><span
class="math inline">\(title-prefix\)</span> - <span
class="math inline">\(endif\)</span><span
class="math inline">\(pagetitle\)</span>
</title>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<p><a href="http://scholar.google.com/citations?user=5SzRq1oAAAAJ"><img src="images/scholar.png" style="width: 100px;"></a></p>
<SCRIPT language="JavaScript" SRC='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></SCRIPT>
<p>Journal names have been intentionally excluded. Lab member names are
in bold. Links are provided to the published article and when possible
an open-access version (pre- or post-print) of the paper. Click on a
triangle for the article abstract.</p>
<p><br></p>
<div id="preprints" class="section level3">
<h3>Preprints</h3>
<details>
<summary>
Allele-specific expression reveals multiple paths to highland adaptation
in maize. [<a href="">preprint</a>]<br />
Hu H, Crow T, Nojoomi S, …[5 authors]… Estévez-Palmas JM,
<strong>Ross-Ibarra J</strong>, Runcie DE.
</summary>
<p style="margin-left: 30px">
Maize is a staple food of smallholder farmers living in highland regions
up to 4,000 meters above sea level worldwide, and Mexican and South
American highlands are two major highland maize growing regions. Maize
landraces show numerous adaptations to highland environments, and
population genetic data suggests the highland populations largely
independently from local lowland populations on each continent. To
better understand the mechanistic basis of highland adaptation, we
crossed maize landraces from 108 highland and lowland sites of Mexico
and South America (MSA) with the inbred line B73 to produce F1 hybrids
and grew them in both highland and lowland sites in Mexico. We
identified thousand of genes with divergent expression between highland
and lowland populations. Hundreds of these genes show patterns of
convergent evolution etween Mexico and South America, including 17
flowering-related genes. To dissect the genetic architecture of the
divergent gene expression, we developed a novel allele-specific
expression analysis pipeline to detect genes with divergent functional
cis-regulatory variation between highland and lowland populations. We
identified hudnreds of genes with divergent cis-regulation between
highland and lowland landrace alleles, with 20 in common between
regions, further suggesting convergence in the genes underlying
adaptation. Principal component analyses of these 20 loci provided
evidence of multiple mechanisms underlying this convergent adaptation:
the spread of adaptive allele through migration and the recruitment of
different, but functionally similar, alleles in different populations.
Our findings reveal a complex genetic architecture of cis-regulatory
alleles underlying adaptation to highlands in maize. Although the vast
majority of evolutionary changes associated with highland adaptation
were region-specific, our findings highlight an important role for
convergence at the gene expression and gene regulation levels as well.
</p>
</details>
<p><br></p>
<details>
<summary>
<p>A happy accident: a novel turfgrass reference genome. [<a
href="https://www.biorxiv.org/content/10.1101/2022.03.08.483531v1">preprint</a>]<br />
<strong>Phillips AR</strong>*, Seetharam AR*, AuBuchon-Elder T …[6
authors]… Kellogg E, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Poa pratensis, commonly known as Kentucky bluegrass, is a popular
cool-season grass species used as turf in lawns and recreation areas
globally. Despite its substantial economic value, a reference genome had
not previously been assembled due to the genome’s relatively large size
and biological complexity that includes apomixis, polyploidy, and
interspecific hybridization. We report here a fortuitous de novo
assembly and annotation of a P. pratensis genome. The draft assembly
consists of 6.09 Gbp with an N50 scaffold length of 65.1 Mbp, and a
total of 118 scaffolds, generated using PacBio long reads and Bionano
optical map technology. We annotated 256K gene models and found 58% of
the genome to be composed of transposable elements. To demonstrate the
applicability of the reference genome, we evaluated population structure
and estimated genetic diversity in three North American wild P.
pratensis populations. Our results support previous studies that found
high genetic diversity and population structure within the species. The
reference genome and annotation will be an important resource for
turfgrass breeding and biologists interested in this complex species.
</p>
</details>
<p><br></p>
<details>
<summary>
<p>Not so local: the population genetics of convergent adaptation in
maize and teosinte. [<a
href="https://www.biorxiv.org/content/10.1101/2021.09.09.459637v1">preprint</a>]<br />
<strong>Tittes S</strong>, <strong>Lorant A</strong>, <strong>McGinty
S</strong>, …[4 authors]… Tenaillon MI, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
What is the genetic architecture of local adaptation and what is the
geographic scale that it operates over? We investigated patterns of
local and convergent adaptation in five sympatric population pairs of
traditionally cultivated maize and its wild relative teosinte (Zea mays
subsp. parviglumis). We found that signatures of local adaptation based
on the inference of adaptive fixations and selective sweeps are
frequently exclusive to individual populations, more so in teosinte
compared to maize. However, for both maize and teosinte, selective
sweeps are frequently shared by several populations, and often between
the subspecies. We were further able to infer that selective sweeps were
shared among populations most often via migration, though sharing via
standing variation was also common. Our analyses suggest that teosinte
has been a continued source of beneficial alleles for maize, post
domestication, and that maize populations have facilitated adaptation in
teosinte by moving beneficial alleles across the landscape. Taken
together, out results suggest local adaptation in maize and teosinte has
an intermediate geographic scale, one that is larger than individual
populations, but smaller than the species range.
</p>
</details>
<p><br></p>
<details>
<summary>
<p>Strengthened mutualistic adaptation between teosinte and its
rhizosphere biota in cold climates [<a
href="https://www.biorxiv.org/content/10.1101/2021.04.20.440703v1">preprint</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Gasca-Pineda J, Baxter, I,
Eguiarte LE, <strong>Ross-Ibarra J</strong>, Strauss SY</p>
</summary>
<p style="margin-left: 30px">
<p>While abiotic environments consistently shape local adaptation, the
strength of local adaptation to biotic interactions may vary more. One
theory, COCO (CO-evolutionary Outcomes across Conditionality), predicts
it may be strongest where species experience greater stress, because
stress increases fitness impacts of species interactions. For example,
in plant interactions with rhizosphere biota, positive outcomes increase
with stress from low soil fertility, drought and cold.</p>
<p>To investigate the influence of abiotic stress gradients on
adaptation between plants and rhizosphere biota, we used a greenhouse
common garden experiment recombining teosinte, Zea mays ssp. mexicana
(wild relative of maize), and rhizosphere biota, collected across a
stress gradient (elevational variation in temperature, precipitation,
and nutrients).</p>
<p>We found stronger local adaptation between teosinte and rhizosphere
biota from colder, more stressful sites, as expected by COCO. However,
biota from less stressful, warmer sites provided greater average
benefits across teosinte populations. Links between plant traits and
20-element profiles of plant leaves explained fitness variation,
persisted in the field, were influenced by both plants and biota, and
largely reflected patterns of local adaptation.</p>
In sum, we uncovered greater local adaptation to biotic interactions in
colder sites, and that both plants and rhizosphere biota affect the
expression of plant phenotypes.
</p>
</details>
<p><br></p>
<details>
<summary>
<p>Conflict over fertilization underlies the transient evolution of
reinforcement. [<a
href="https://www.biorxiv.org/content/10.1101/2020.11.10.377481v1">preprint</a>]<br />
<strong>Rushworth C</strong>, Wardlaw AW, <strong>Ross-Ibarra
J</strong>, Brandvain YB.</p>
</summary>
<p style="margin-left: 30px">
When two populations or species hybridize, their offspring often
experience reductions in fitness relative to either parental population.
The production of low fitness hybrids may be prevented by the evolution
of increased prezygotic isolation; a process known as reinforcement.
Theoretical challenges to the evolution of reinforcement are generally
cast as a coordination problem — e.g., linkage disequilibrium between
trait and preference loci is difficult to maintain in the face of
recombination. However, the evolution of reinforcement also poses a
potential conflict between mates. For example, the opportunity costs to
hybridization may differ between the sexes or species. This is
particularly likely for postmating prezygotic isolation, as the ability
to fertilize both conspecific and heterospecific eggs is beneficial to
male gametes, but heterospecific mating may incur a cost for female
gametes. Motivated by this problem, we develop a population genetic
model of interspecific conflict over reinforcement, inspired by
`gametophytic factors’, which act as postmating prezygotic barriers
among Zea mays subspecies. We demonstrate that this conflict results in
the transient evolution of reinforcement –– after female preference for
a conspecific gamete trait rises to high frequency, male traits
adaptively introgress into the other population. Ultimately the male
gamete trait fixes in both species, and prezygotic isolation returns to
pre-reinforcement levels. We interpret geographic patterns of isolation
among Z. mays subspecies in light of these findings, and suggest when
and how this conflict can be mediated. Our results suggest that sexual
conflict may pose an understudied obstacle to the evolution of
reinforcement via postmating prezygotic isolation.
</p>
</details>
<p><br></p>
<details>
<summary>
<p>Single gene resolution of locally adaptive genetic variation in
Mexican maize [<a
href="https://www.biorxiv.org/content/10.1101/706739v1">preprint</a>]<br />
<strong>Gates DJ</strong>, Runcie D, Janzen GM, Romero Navarro A …[4
authors]… Buckler ES, Hearne S, Hufford MB, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We present a high-resolution genome-wide association analysis to
identify loci exhibiting adaptive patterns in a large panel of more than
4500 traditional maize landraces representing the breadth of genetic
diversity of maize in Mexico. We evaluate associations between genotype
and plant performance in 13 common gardens across a range of
environments, identifying hundreds of candidate genes underlying
genotype by environment interaction. We further identify genetic
associations with environment across Mexico and show that such loci are
associated with variation in yield and flowering time in our field
trials and predict performance in independent drought trials. Our
results indicate that the variation necessary to adapt crops to changing
climate exists in traditional landraces that have been subject to
ongoing environmental adaptation and can be identified by both
phenotypic and environmental association.
</p>
</details>
<p><br></p>
</div>
<div id="section" class="section level3">
<h3>2022</h3>
<details>
<summary>
Portrait of a genus: genome sequencing reveals evidence of adaptive
variation in Zea [<a
href="https://www.biorxiv.org/content/10.1101/2022.06.03.494450v1">preprint</a>]<br />
Chen L, Luo J, Minliang Jin, <strong>Yang N</strong> …[27 authors
including <strong>Phillips AR</strong>]… , Warburton ML,
<strong>Ross-Ibarra J</strong>, Yan J.
</summary>
<p style="margin-left: 30px">
Maize is a globally valuable commodity and one of the most extensively
studied genetic model organisms. However, we know surprisingly little
about the extent and potential utility of the genetic variation found in
the wild relatives of maize. Here, we characterize a high-density
genomic variation map from 744 genomes encompassing maize and all wild
taxa of the genus Zea, identifying over 70 million single nucleotide
polymorphisms (SNPs) and nearly 9 million Insertion/Deletion (InDel)
polymorphisms. The variation map reveals evidence of selection within
taxa displaying novel adaptations to traits such as waterlogging,
perenniality and regrowth. We focus in detail on adaptive alleles in
highland teosinte and temperate maize and highlight the key role of
flowering time related pathways in highland and high latitude
adaptation. To show how this data can identify useful genetic variants,
we generated and characterized novel mutant alleles for two flowering
time candidate genes. This work provides the most extensive sampling to
date of the genetic diversity of the genus Zea, resolving questions on
evolution and identifying adaptive variants for direct use in modern
breeding.
</p>
</details>
<p><br></p>
<details>
<summary>
Genomic Insights into Historical Improvement of Heterotic Groups during
Modern Hybrid Maize Breeding.<br />
Li C, Guan H, Jing X, Li Y …[18 authors]… <strong>Ross-Ibarra
J</strong>, Li Y, Wang T, Wang H
</summary>
<p style="margin-left: 30px">
Single-cross maize hybrids display superior heterosis and are produced
from crossing two parental inbred lines belonging to genetically
different heterotic groups. To dissect the genomic basis underlying
breeding improvement of heterotic groups, we assembled 1,604
historically utilized maize inbred lines belonging to various female and
male heterotic groups, and conducted phenotyping and genomic sequencing
analyses. We find that the female heterotic groups (FHGs) and male
heterotic groups (MHGs) have undergone convergent changes in a common
set of plant architecture- and yield-related traits across the breeding
eras for increased tolerance to higher planting density and increased
yield. In addition, we find that the FHGs and MHGs have experienced
divergent changes for three ear-related traits associated with kernel
dehydration rate and mechanical harvesting. Using genome-wide selection
scans and association analyses, we identify a large number of candidate
genes that contributed to the improvement of agronomic traits of the
female and male parental lines. Moreover, we observe increased genetic
differentiation between the FHGs and MHGs across the breeding eras, and
find positive correlation between increase of heterozygosity levels in
the differentiated genes with heterosis in hybrids. Further, we validate
the function of two selected genes affecting flowering time/ear height
and kernel length/kernel weight in inbred lines, and a differentiated
gene promoting ear length/ear weight in hybrids. This study provides
insights into the genomic basis of hybrid maize breeding, and should
enable future genomics-informed maize breeding.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://onlinelibrary.wiley.com/doi/10.1002/evl3.285">The
genomic signature of wild-to-crop introgression during the domestiation
of scarlet runner bean (<em>Phaseolus coccineus L.</em>)</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.02.03.429668v1">preprint</a>]<br />
Guerra-Garcia A, Rojas-Barrera IC, <strong>Ross-Ibarra J</strong>, Papa
R, Piñero D.
</summary>
<p style="margin-left: 30px">
The scarlet runner bean is an open-pollinated legume from the highlands
of Mesoamerica that is cultivated in small-scale agriculture for its dry
seeds and immature pods. Demographic bottlenecks associated with
domestication might reduce genetic diversity and facilitate the
accumulation of deleterious mutations. Conversely, introgression from
wild relatives could be a source of variation. Using Genotyping by
Sequencing data (79,286 SNVs) from 237 cultivated and wild samples, we
evaluated the demographic history of traditional varieties from
different regions of Mexico and looked for evidence of introgression
between sympatric wild and cultivated populations. Traditional varieties
have high levels of diversity, even though there is evidence of a severe
initial genetic bottleneck, followed by a population expansion.
Introgression from wild to domesticated populations was detected, but
not in the opposite direction. This asymmetric introgression might
contribute to the recovery of genetic variation and it has occurred at
different times: constantly in the center of Mexico; recently in the
North West; and anciently in the South. Several factors are acting
together to increase and maintain genetic diversity in P. coccineus
cultivars, such as demographic expansion and introgression. Wild
relatives represent a valuable genetic resource and have played a key
role in scarlet runner bean evolution via introgression into traditional
varieties.
</p>
</details>
<p><br></p>
<details>
<summary>
An adaptive teosinte mexicana introgression modulates
phosphatidylcholine levels and is associated with maize flowering time
[<a
href="https://www.biorxiv.org/content/10.1101/2021.01.25.426574v4">preprint</a>]<br />
Barnes AC*, Rodríguez-Zapata F*, Blöcher-Juárez KA*, <strong>Gates
DJ</strong> …[22 authors]… <strong>Ross-Ibarra J</strong>, Hufford M,
Sawers RJH, Rellán-Álvarez R.
</summary>
<p style="margin-left: 30px">
Native Americans domesticated maize (Zea mays ssp. mays) from lowland
teosinte parviglumis (Zea mays ssp.parviglumis) in the warm Mexican
southwest and brought it to the highlands of México and South America
where it was exposed to lower temperatures that imposed strong selection
on flowering time. Phospholipids are important metabolites in plant
responses to low-temperature and phosphorus availability, and have been
suggested to influence flowering time. Here, we combined linkage mapping
with genome scans to identify High PhosphatidylCholine 1 (HPC1), a gene
that encodes a phospholipase A1 enzyme, as a major driver of
phospholipid variation in highland maize. Common garden experiments
demonstrated strong genotype-by-environment interactions associated with
variation at HPC1, with the highland HPC1 allele leading to higher
fitness in highlands, possibly by hastening flowering. The highland
maize HPC1 variant resulted in impaired function of the encoded protein
due to a polymorphism in a highly conserved sequence. A meta-analysis
across HPC1 orthologs indicated a strong association between the
identity of the amino acid at this position and optimal growth in
prokaryotes. Mutagenesis of HPC1 via genome editing validated its role
in regulating phospholipid metabolism. Finally, we showed that the
highland HPC1 allele entered cultivated maize by introgression from the
wild highland teosinte Zea mays ssp. mexicana and has been maintained in
maize breeding lines from the Northern US, Canada and Europe. Thus, HPC1
introgressed from teosinte mexicana underlies a large metabolic QTL that
modulates phosphatidylcholine levels and has an adaptive effect at least
in part via induction of early flowering time.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://academic.oup.com/gbe/article/14/2/evac016/6519160?login=false">Controlling
for variable transposition rate with an age-adjusted site frequency
spectrum</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.08.16.456262v1">preprint</a>]<br />
<strong>Horvath R</strong>, <strong>Menon M</strong>, Stitzer MC,
<strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Recognition of the important role of transposable elements (TEs) in
eukaryotic genomes quickly led to a burgeoning literature modeling and
estimating the effects of selection on TEs. Much of the empirical work
on selection has focused on analyzing the site frequency spectrum (SFS)
of TEs. But TEs differ from standard evolutionary models in a number of
ways that can impact the power and interpretation of the SFS. For
example, rather than mutating under a clock-like model, transposition
often occurs in bursts which can inflate particular frequency categories
compared to expectations under a standard neutral model. If a TE burst
has been recent, the excess of low frequency polymorphisms can mimic the
effect of purifying selection. Here, we investigate how transposition
bursts affect the frequency distribution of TEs and the correlation
between age and allele frequency. Using information on the TE age
distribution, we propose an age-adjusted site frequency spectrum to
compare TEs and neutral polymorphisms to more effectively evaluate
whether TEs are under selective constraints. We show that our approach
can minimize instances of false inference of selective constraint, but
also allows for a correct identification of even weak selection
affecting TEs which experienced a transposition burst and is robust to
at least simple demographic changes. The results presented here will
help researchers working on TEs to more reliably identify the effects of
selection on TEs without having to rely on the assumption of a constant
transposition rate.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://academic.oup.com/g3journal/article/12/3/jkac013/6520465">Analysis
of genotype by environment interactions in a maize mapping
population</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.07.21.453280v1">preprint</a>]<br />
<strong>Hudson AI</strong>, <strong>Odell SG</strong>, Dubreuil P,
Tixier M-H, Praud S, Runcie DE, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Genotype by environment interactions are a significant challenge for
crop breeding as well as being important for understanding the genetic
basis of environmental adaptation. In this study, we analyzed genotype
by environment interaction in a maize multi-parent advanced generation
intercross population grown across five environments. We found that
genotype by environment interactions contributed as much as genotypic
effects to the variation in some agronomically important traits. In
order to understand how genetic correlations between traits change
across environments, we estimated the genetic variance-covariance matrix
in each environment. Changes in genetic covariances between traits
across environments were common, even among traits that show low
genotype by environment variance. We also performed a genome-wide
association study to identify markers associated with genotype by
environment interactions but found only a small number of significantly
associated markers, possibly due to the highly polygenic nature of
genotype by environment interactions in this population.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009797">Domestication
reshaped the genetic basis of inbreeding depression in a maize landrace
compared to its wild relative, teosinte</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.09.01.458502v1">preprint</a>]<br />
Holland JB, Samayoa LF, Olukolu BA, Yang CJ, …[7 authors]… <strong>Yang
J</strong>, <strong>Ross-Ibarra J</strong>, Buckler ES, Doebley JF.</p>
</summary>
<p style="margin-left: 30px">
Inbreeding depression is the reduction in fitness and vigor resulting
from mating of close relatives observed in many plant and animal
species. The extent to which the genetic load of mutations contributing
to inbreeding depression is due to rare large-effect variation versus
potentially more common variants with very small individual effects is
unknown and may be affected by population history. We compared the
effects of outcrossing and self-fertilization on 18 traits in a landrace
population of maize, which underwent a population bottleneck during
domestication, and a neighboring population of its wild relative
teosinte. Inbreeding depression was greater in maize than teosinte for
15 of 18 traits, congruent with the greater segregating genetic load
predicted from sequence data in the maize population. For many traits -
and more commonly in maize - genetic variation among self-fertilized
families was less than expected based on additive and dominance variance
estimated in outcrossed families, suggesting that a negative covariance
between additive and homozygous dominance effects limits the variation
available to selection under partial inbreeding. We identified
quantitative trait loci (QTL) representing large-effect rare variants
carried by only a single parent, which were more important in teosinte
than maize. Teosinte also carried more putative juvenile-acting lethal
variants identified by segregation distortion. These results suggest a
mixture of mostly polygenic, small-effect recessive variation underlying
inbreeding depression, with an additional contribution from rare
larger-effect variants that was more important in teosinte but depleted
in maize following to the domestication bottleneck. Purging associated
with the maize domestication bottleneck may have selected against large
effect variants, but polygenic load is harder to purge and segregating
mutational burden increased in maize compared to teosinte.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8896015/">A B73
x Palomero Toluqueo mapping population reveals local adaptation in
Mexican highland maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.09.15.460568v1">preprint</a>]<br />
Perez-Limon S, Li M, …[13 authors]… <strong>Ross-Ibarra J</strong>,
Flint-Garcia S, Diaz-Garcia L, Rellán-Álvarez R, Sawers RJH</p>
</summary>
<p style="margin-left: 30px">
Generations of farmer selection have produced a unique collection of
traditional maize varieties adapted to the environmental challenges of
the central Mexican highlands. In addition to agronomic and cultural
value, Mexican highland maize represents a good system for the study of
local adaptation and acquisition of adaptive phenotypes under
cultivation. In this study, we characterized a recombinant inbred line
population derived from the cross of the B73 reference line and the
Mexican highland maize variety Palomero Toluqueo. Evaluation over
multiple years in lowland and highland field sites in Mexico identified
genomic regions linked to yield components and putatively adaptive
morphological traits. A region on chromosome 7 associated with ear
weight showed antagonistic allelic effects in lowland and highland
fields, suggesting a trade-off consistent with local adaptation. We
identified several alleles of highland origin associated with
characteristic highland traits, including reduced tassel branching,
increased stem pigmentation and the presence of stem macrohairs. The
oligogenic architecture of characteristic morphological traits supports
their role in adaptation, suggesting they have arisen from consistent
directional selection acting at distinct points across the genome. We
discuss these results in the context of the origin of phenotypic novelty
during selection, commenting on the role of de novo mutation and the
acquisition of adaptive variation by gene flow from endemic wild
relatives.
</p>
</details>
<p><br></p>
</div>
<div id="section-1" class="section level3">
<h3>2021</h3>
<details>
<summary>
<p><a
href="https://academic.oup.com/g3journal/article/12/3/jkac011/6509518?login=true">Modeling
Allelic Diversity of Multi-parent Mapping Populations Affects Detection
of Quantitative Trait Loci</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.07.14.452335v1">preprint</a>]<br />
<strong>Odell SG</strong>, <strong>Hudson AI</strong>, Praud S, Dubreuil
P, Tixier M-H, <strong>Ross-Ibarra J</strong>, Runcie DE</p>
</summary>
<p style="margin-left: 30px">
The search for quantitative trait loci (QTL) that explain complex traits
such as yield and flowering time has been ongoing in all crops. Methods
such as bi-parental QTL mapping and genome-wide association studies
(GWAS) each have their own advantages and limitations. Multi-parent
advanced generation intercross (MAGIC) populations contain more
recombination events and genetic diversity than bi-parental mapping
populations and reduce the confounding effect of population structure
that is an issue in association mapping populations. Here we discuss the
results of using a MAGIC population of doubled haploid (DH) maize lines
created from 16 diverse founders to perform QTL mapping. We compare
three models that assume bi-allelic, founder, and ancestral haplotype
allelic states for QTL. The three methods have different power to detect
QTL for a variety of agronomic traits. Although the founder approach
finds the most QTL, there are also QTL unique to each method, suggesting
that each model has advantages for traits with different genetic
architectures. A closer look at a well-characterized flowering time QTL,
qDTA8, which contains vgt1, suggests a potential epistatic interaction
and highlights the strengths and weaknesses of each method. Overall, our
results reinforce the importance of considering different approaches to
analyzing genotypic datasets, and show the limitations of binary SNP
data for identifying multi-allelic QTL.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://journals.plos.org/plosgenetics/article/comments?id=10.1371/journal.pgen.1009810">Selective
sorting of ancestral introgression in maize and teosinte along an
elevational cline</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.03.05.434040v1">preprint</a>]<br />
Calfee E, <strong>Gates DJ</strong>, <strong>Lorant A</strong>,
<strong>Perkins MT</strong>, Coop G, <strong>Ross-Ibarra J</strong>.</p>
</summary>
<p style="margin-left: 30px">
While often deleterious, hybridization can also be a key source of
genetic variation and pre-adapted haplotypes, enabling rapid evolution
and niche expansion. Here we evaluate these opposing selection forces on
introgressed ancestry between maize (Zea mays ssp. mays), and its wild
teosinte relative. Introgression from ecologically diverse teosinte may
have facilitated maize’s global range expansion, in particular to
challenging high elevation regions (&gt; 1500 m). We generated
low-coverage genome sequencing data for 348 maize and mexicana
individuals to evaluate patterns of introgression in 14 sympatric
population pairs, spanning the elevational range of Zea mays ssp.
mexicana, a teosinte endemic to the mountains of Mexico. While recent
hybrids are commonly observed in sympatric populations and mexicana
demonstrates fine-scale local adaptation, we find that the majority of
mexicana ancestry tracts introgressed &gt;1000 generations ago. This
mexicana ancestry seems to have maintained much of its diversity and
likely came from a common ancestral source, rather than contemporary
sympatric populations, resulting in relatively low Fst between mexicana
ancestry tracts sampled from geographically distant maize populations.
Introgressed mexicana ancestry is reduced in lower-recombination rate
quintiles of the genome and around domestication genes, consistent with
pervasive selection against introgression. However, we also find
mexicana ancestry increases across the sampled elevational gradient and
that high introgression peaks are most commonly shared among
high-elevation maize populations, consistent with introgression from
mexicana facilitating adaptation to the highland environment. In the
other direction, we find patterns consistent with adaptive and clinal
introgression of maize ancestry into sympatric mexicana at many loci
across the genome, suggesting that maize also contributes to adaptation
in mexicana, especially at the lower end of its elevational range. In
sympatric maize, in addition to high introgression regions we find many
genomic regions where selection for local adaptation maintains steep
gradients in introgressed mexicana ancestry across elevation, including
at least two inversions: the well-characterized Inv4m and a new 3 Mb
inversion Inv9f surrounding the macrohairless1 locus on chromosome 9.
The bulk of our ancestry selection outliers show no signals of sweeps or
local sourcing from sympatric populations and so likely represent
ancestral introgression sorted by selection, resulting in correlated but
distinct outcomes of introgression in different contemporary maize
landrace populations.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009768">The
Genomic Ecosystem of Transposable Elements in Maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/559922v1">preprint</a>][<a
href="https://github.com/mcstitzer/maize_genomic_ecosystem">github</a>][<a
href="https://mcstitzer.shinyapps.io/maize_te_families/">interactive</a>]<br />
<strong>Stitzer MC</strong>, Anderson SN, Springer NM,
<strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Transposable elements (TEs) constitute the majority of flowering plant
DNA, reflecting their tremendous success in subverting, avoiding, and
surviving the defenses of their host genomes to ensure their selfish
replication. More than 85% of the sequence of the maize genome can be
ascribed to past transposition, providing a major contribution to the
structure of the genome. Evidence from individual loci has informed our
understanding of how transposition has shaped the genome, and a number
of individual TE insertions have been causally linked to dramatic
phenotypic changes. But genome-wide analyses in maize and other taxa
have frequently represented TEs as a relatively homogeneous class of
fragmentary relics of past transposition, obscuring their evolutionary
history and interaction with their host genome. Using an updated
annotation of structurally intact TEs in the maize reference genome, we
investigate the family-level ecological and evolutionary dynamics of TEs
in maize. Integrating a variety of data, from descriptors of individual
TEs like coding capacity, expression, and methylation, as well as
similar features of the sequence they inserted into, we model the
relationship between these attributes of the genomic environment and the
survival of TE copies and families. Our analyses reveal a diversity of
ecological strategies of TE families, each representing the evolution of
a distinct ecological niche allowing survival of the TE family. In
contrast to the wholesale relegation of all TEs to a single category of
junk DNA, these differences generate a rich ecology of the genome,
suggesting families of TEs that coexist in time and space compete and
cooperate with each other. We conclude that while the impact of
transposition is highly family- and context-dependent, a family-level
understanding of the ecology of TEs in the genome can refine our ability
to predict the role of TEs in generating genetic and phenotypic
diversity.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://science.sciencemag.org/content/373/6555/655.abstract"><em>De
novo</em> assembly, annotation, and comparative analysis of 26 diverse
maize genomes</a>. [<a
href="https://www.biorxiv.org/content/10.1101/2021.01.14.426684v1">preprint</a>]<br />
Hufford MB, Seetharam AS, …[40 authors including <strong>Hudson
AI</strong> and <strong>Tittes S</strong>]… <strong>Ross-Ibarra
J</strong>, Yu J, Gent JI, Hirsch CN, Ware D, Dawe RK.</p>
</summary>
<p style="margin-left: 30px">
We report de novo genome assemblies, transcriptomes, annotations, and
methylomes for the 26 inbreds that serve as the founders for the maize
nested association mapping population. The data indicate that the number
of pan-genes exceeds 103,000 and that the ancient tetraploid character
of maize continues to degrade by fractionation to the present day.
Excellent contiguity over repeat arrays and complete annotation of
centromeres further reveal the locations and internal structures of
major cytological landmarks. We show that combining structural variation
with SNPs can improve the power of quantitative mapping studies.
Finally, we document variation at the level of DNA methylation, and
demonstrate that unmethylated regions are enriched for cis-regulatory
elements that overlap QTL and contribute to changes in gene expression.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://genome.cshlp.org/content/early/2021/05/27/gr.266528.120.abstract#aff-4">Conserved
noncoding sequences provide insights into regulatory sequence and loss
of gene expression in maize</a>. [<a
href="https://www.biorxiv.org/content/10.1101/2020.07.11.192575v2">preprint</a>]<br />
Song B, Buckler ES, Wang H, Wu Y, …[4 authors including <strong>D
Gates</strong>]… Bradbury PJ, <strong>Ross-Ibarra J</strong>, Hufford
MB, Romay MC.</p>
</summary>
<p style="margin-left: 30px">
DNA sequencing technology has advanced so quickly, identifying key
functional regions using evolutionary approaches is required to
understand how those genomes work. This research develops a sensitive
sequence alignment approach to identify functional constrained
non-coding sequences in the Andropogoneae tribe. The grass tribe
Andropogoneae contains several crop species descended from a common
ancestor ~18 million years ago. Despite broadly similar phenotypes, they
have tremendous genomic diversity with a broad range of ploidy levels
and transposons. These features make Andropogoneae a powerful system for
studying conserved non-coding sequence (CNS), here we used it to
understand the function of CNS in maize. We find that 86% of CNS
comprise known genomic elements e.g., cis-regulatory elements,
chromosome interactions, introns, several transposable element
superfamilies, and are linked to genomic regions related to DNA
replication initiation, DNA methylation and histone modification. In
maize, we show that CNSs regulate gene expression and variants in CNS
are associated with phenotypic variance, and rare CNS absence
contributes to loss of gene expression. Furthermore, we find the
evolution of CNS is associated with the functional diversification of
duplicated genes in the context of the maize subgenomes. Our results
provide a quantitative understanding of constrained non-coding elements
and identify functional non-coding variation in maize.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab119/6255756">Molecular
Parallelism Underlies Convergent Highland Adaptation of Maize
Landraces</a> [<a
href="https://www.biorxiv.org/content/10.1101/2020.07.31.227629v1">preprint</a>]<br />
<strong>Wang L</strong>, <strong>Josephs EB</strong>, Lee KM, Roverts
LM, Rellán-Álvarez R, <strong>Ross-Ibarra J</strong>, Hufford MB.</p>
</summary>
<p style="margin-left: 30px">
Convergent phenotypic evolution provides some of the strongest evidence
for adaptation. However, the extent to which recurrent phenotypic
adaptation has arisen via parallelism at the molecular level remains
unresolved, as does the evolutionary origin of alleles underlying such
adaptation. Here, we investigate genetic mechanisms of convergent
highland adaptation in maize landrace populations and evaluate the
genetic sources of recurrently selected alleles. Population branch
excess statistics reveal strong evidence of parallel adaptation at the
level of individual SNPs, genes and pathways in four independent
highland maize populations, even though most SNPs show unique patterns
of local adaptation. The majority of selected SNPs originated via
migration from a single population, most likely in the Mesoamerican
highlands. Polygenic adaptation analyses of quantitative traits reveal
that alleles affecting flowering time are significantly associated with
elevation, indicating the flowering time pathway was targeted by
highland adaptation. In addition, repeatedly selected genes were
significantly enriched in the flowering time pathway, indicating their
significance in adapting to highland conditions. Overall, our study
system represents a promising model to study convergent evolution in
plants with potential applications to crop adaptation across
environmental gradients.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab061/6237897?searchresult=1">Gene
body methylation is under selection in <em>Arabidopsis
thaliana</em>.</a> [<a
href="https://www.biorxiv.org/content/10.1101/2020.09.04.283333v1">preprint</a>]<br />
Muyle A, <strong>Ross-Ibarra J</strong>, Seymour DK, Gaut BS.</p>
</summary>
<p style="margin-left: 30px">
In plants, mammals and insects, some genes are DNA methylated in the CG
dinucleotide context, a phenomenon called gene body methylation. It has
long been controversial whether this phenomenon has any functional role.
Here, we took advantage of the availability of 868 leaf methylomes in
Arabidopsis thaliana to characterize the population frequency of gene
body methylation at the gene level. We used two outgroups to infer the
ancestral methylation state of orthologs. Using the A. thaliana gene
body methylation site frequency spectrum and a population genetics model
specifically designed for epigenetic data, we find that genes with
ancestral gene body methylation are under significant selection to
remain methylated. Conversely, ancestrally unmethylated genes are under
selection to remain unmethylated. The estimated selection coefficients
are small, on the order of magnitude of selection acting on codon usage.
We also find that A. thaliana is losing gene body methylation more often
than gaining it, which could be due to a recent reduction in the
efficacy of selection after a switch to selfing. We investigated the
potential function of gene body methylation through its link with gene
expression level. We show that, within genes with polymorphic
methylation states in A. thaliana wild populations, the gene body
methylated state is consistently associated with the highest gene
expression level. Our work suggests that gene body methylation has a
small effect on fitness, but substantial enough for natural selection to
act on it.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://www.nature.com/articles/s41477-020-00834-5.epdf?sharing_token=_mN9oSJzgPbS9Ldsr01s8dRgN0jAjWel9jnR3ZoTv0OIqw-nLs7dqtihevHgjRdBgKyJmnWQSkGOUEc4hulR2jnTLjP4fqtKbSUSzA7iWbyA0E0YEvXdfVUO_1_dmi2uNdJtrPRcKz9O_jp58VwFcP02Yhzr7I3YNxgg8Oit1Xk%3D">Comparative
evolutionary analysis and prediction of deleterious mutation patterns
between sorghum and maize</a> [<a
href="https://doi.org/10.1101/777623">preprint</a>][<a
href="https://github.com/GoreLab/Sorghum-HapMap">github</a>]<br />
Lozano R, Gazave E, dos Santos JPR, Stetter MG …[4 authors]… Mockler T,
Buckler ES, <strong>Ross-Ibarra J</strong>, Gore MA</p>
</summary>
<p style="margin-left: 30px">
Sorghum and maize share a close evolutionary history that can be
explored through comparative genomics. To perform a large-scale
comparison of the genomic variation between these two species, we
analyzed 13M variants identified from whole genome resequencing (WGS) of
468 sorghum lines together with 25M variants previously identified in
1,218 maize lines. Deleterious mutations in both species were prevalent
in pericentromeric regions, enriched in non-syntenic genes, and present
at low allele frequencies. A comparison of deleterious burden between
sorghum and maize revealed that sorghum, in contrast to maize, departed
from the “domestication cost” hypothesis that predicts a higher
deleterious burden among domesticates compared to wild lines.
Additionally, sorghum and maize population genetic summary statistics
were used to predict a gene deleterious index with an accuracy higher
than 0.5. This research represents a key step towards understanding the
evolutionary dynamics of deleterious variants in sorghum and provides a
comparative genomics framework to start prioritizing them for removal
through genome editing and breeding.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Cathy Rushworth</strong></p>
<pre><code>Rushworth, C.A., and T. Mitchell-Olds. 2021. The evolution of sex is tempered by costly hybridization in Boechera (rock cress). Journal of Heredity 112(1): 67-77 https://doi.org/10.1093/jhered/esaa041.

Rushworth, C.A., R.S. Baucom, B.K. Blackman, M. Neiman, M.E. Orive, A. Sethuraman, J. Ware, and D.R. Matute. 2021. Who are we now? A demographic assessment of the evolution societies. Evolution 75(2): 208-218 https://onlinelibrary.wiley.com/doi/10.1111/evo.14168.

Carley, L.N., J.P. Mojica, B. Wang, C.-Y. Chen, Y.-P. Lin, K.V.S.K. Prasad, E. Chan, C.-W. Hsu, R. Keith, C.L. Nuñez, C.F. Olson-Manning, C.A. Rushworth, M.R. Wagner, J. Wang, P.-M. Yeh, M. Reichelt, K. Ghattas, J. Gershenzon, C.-R. Lee, and T. Mitchell-Olds. 2021. Ecological factors influence balancing selection on leaf chemical profiles of a wildflower. Nature Ecology and Evolution https://doi.org/10.1038/s41559-021-01486-0.</code></pre>
<p><strong>Mitra Menon</strong></p>
<pre><code>Menon, M., Bagley, J.C., Page, G.F.M. et al. Adaptive evolution in a conifer hybrid zone is driven by a mosaic of recently introgressed and background genetic variants. Commun Biol 4, 160 (2021). https://doi.org/10.1038/s42003-020-01632-7</code></pre>
<p><strong>Silas Tittes</strong></p>
<pre><code>O’Hara, N. B., Franks, S. J., Kane, N. C., Tittes, S., &amp; Rest, J. S. (2021). Evolution of pathogen response genes associated with increased disease susceptibility during adaptation to an extreme drought in a Brassica rapa plant population. BMC Ecology and Evolution, 21(1), 1-11.  </code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-2" class="section level3">
<h3>2020</h3>
<details>
<summary>
<a href="https://www.nature.com/articles/s41467-020-19333-4">Adaptive
evolution of DNA methylation reshaped gene regulation in maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/2020.03.13.991117v1">preprint</a>]
[<a href="https://github.com/jyanglab/msfs_teo">github</a>]<br />
Xu G, Lyu J, Li Q, Liu H, Wang D, Zhang M, Springer NM,
<strong>Ross-Ibarra J</strong>, Yang J
</summary>
<p style="margin-left: 30px">
DNA methylation is a ubiquitous chromatin feature – in maize, more than
25% of cytosines in the genome are methylated. Recently, major progress
has been made in describing the molecular mechanisms driving
methylation, yet variation and evolution of the methylation landscape
during maize domestication remain largely unknown. Here we leveraged
whole-genome sequencing (WGS) and whole-genome bisulfite sequencing
(WGBS) on populations of modern maize, landrace, and teosinte (Zea mays
ssp. parviglumis) to investigate the adaptive and phenotypic
consequences of methylation variations in maize. By using a novel
estimation approach, we inferred the methylome site frequency spectrum
(mSFS) to estimate forward and backward methylation mutation rates and
selection coefficients. We only found weak evidence for direct
selections on methylations in any context, but thousands of
differentially methylated regions (DMRs) were identified in
population-wide that are correlated with recent selections. Further
investigation revealed that DMRs are enriched in 5’ untranslated
regions, and that maize hypomethylated DMRs likely helped rewire distal
gene regulation. For two trait-associated DMRs, vgt1-DMR and tb1-DMR,
our HiChIP data indicated that the interactive loops between DMRs and
respective downstream genes were present in B73, a modern maize line,
but absent in teosinte. And functional analyses suggested that these
DMRs likely served as cis-acting elements that modulated gene regulation
after domestication. Our results enable a better understanding of the
evolutionary forces acting on patterns of DNA methylation and suggest a
role of methylation variation in adaptive evolution.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008791">The
genetic architecture of the maize progenitor, teosinte, and how it was
altered during maize domestication</a><br />
Chen Q, Fernando Samayoa L, Yang CJ, …[5 authors]… <strong>Lorant
A</strong>, Buckler ES, <strong>Ross-Ibarra J</strong>, Holland JB,
Doebley JF
</summary>
<p style="margin-left: 30px">
The genetics of domestication has been extensively studied ever since
the rediscovery of Mendel’s law of inheritance and much has been learned
about the genetic control of trait differences between crops and their
ancestors. Here, we ask how domestication has altered genetic
architecture by comparing the genetic architecture of 18 domestication
traits in maize and its ancestor teosinte using matched populations. We
observed a strongly reduced number of QTL for domestication traits in
maize relative to teosinte, which is consistent with the previously
reported depletion of additive variance by selection during
domestication. We also observed more dominance in maize than teosinte,
likely a consequence of selective removal of additive variants. We
observed that large effect QTL have low minor allele frequency (MAF) in
both maize and teosinte. Regions of the genome that are strongly
differentiated between teosinte and maize (high F ST ) explain less
quantitative variation in maize than teosinte, suggesting that, in these
regions, allelic variants were brought to (or near) fixation during
domestication. We also observed that genomic regions of high
recombination explain a disproportionately large proportion of heritable
variance both before and after domestication. Finally, we observed that
about 75% of the additive variance in both teosinte and maize is
“missing” in the sense that it cannot be ascribed to detectable QTL and
only 25% of variance maps to specific QTL. This latter result suggests
that morphological evolution during domestication is largely
attributable to very large numbers of QTL of very small effect.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="https://www.g3journal.org/content/ggg/early/2020/05/28/g3.120.401196.full.pdf">Loss
of diversity and accumulation of genetic load in doubled-haploid lines
from European maize landraces</a> [<a
href="https://www.biorxiv.org/content/10.1101/817791v1">preprint</a>][<a
href="https://github.com/LZeitler/eurodh-scripts/blob/master/README.org">github</a>]<br />
<strong>Zeitler L</strong>, <strong>Ross-Ibarra J</strong>,
<strong>Stetter MGS</strong>
</summary>
<p style="margin-left: 30px">
Maize landraces are well adapted to their local environments and present
valuable sources of genetic diversity for breeding and conservation. But
the maintenance of open-pollinated landraces in programs is challenging,
as regeneration of seed can often lead to inbreeding depression and the
loss of diversity due to genetic drift. Recent reports suggest that the
production of doubled-haploid (DH) lines from landraces may serve as a
convenient means to preserve useful genetic diversity in a homozygous
form that is immediately useful for modern breeding. The production of
doubled-haploid (DH) lines presents an extreme case of inbreeding which
results in instantaneous homozygosity genome-wide. Here we analyzed the
effect of DH production on genetic diversity, using genome-wide SNP data
from hundreds of individuals of five European landraces and their
related DH lines. In contrast to previous findings, we observe a
dramatic loss of diversity at both the haplotype level and that of
individual SNPs. We identify thousands of SNPs that exhibit allele
frequency differences larger than expected under models of neutral
genetic drift, and document patterns of heterozygosity and polymorphism
at conserved sites that suggest an important role for deleterious
recessive load in determining diversity differences between landrace and
DH populations. Although we were unable to uncover more details about
the mode of selection, we conclude that while landrace DH lines may be a
valuable tool for the introduction of variation into maize breeding
programs they come at the cost of decreased genetic diversity and
increased genetic load.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="https://drive.google.com/file/d/1UxPN_Yf2HQ0OOJdqbP537Bwxlm2N1EkL/view?usp=sharing">Genome-wide
selection and genetic improvement during modern maize breeding</a><br />
Wang B, Lin Z, Li X, Zhao Y …[17 authors]… Hufford MB,
<strong>Ross-Ibarra J</strong>, He H, Wang H.
</summary>
<p style="margin-left: 30px">
Modern crop breeding has made a profound impact on food production to
sustain population growth during the past century; however, systematic
analyses of the genomic basis underlying the dramatic increase in crop
yields during modern breeding remain limited in scope. Here we report an
analysis of the genomic and phenotypic changes associated with modern
maize breeding across a chronological sampling of 350 elite inbred lines
representing multiple eras of both Chinese and U.S. germplasm. We
document a number of convergent phenotypic changes in both countries.
Using genome-wide association and selection scan methods, we identify
160 loci underlying adaptive agronomic phenotypes and more than 1,800
genomic regions representing the targets of selection during modern
breeding. We functionally validate two candidate genes underpinning
variation in ear height and tassel branch number using CRISPR knockout
mutants. In sum, this work demonstrates the utility of breeding-era
approach for identifying breeding signatures and lays the foundation for
future genomics-enabled maize breeding.”
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="https://www.sciencedirect.com/science/article/pii/S1369526620300339">Evolutionary
insights into plant breeding</a> [<a
href="https://agrixiv.org/akdt8">preprint</a>]<br />
<strong>Turner-Hissong SD</strong>, Mabry ME, Beissinger TM,
<strong>Ross-Ibarra J</strong>, Pires JC
</summary>
<pre><code>&lt;p style=&quot;margin-left: 30px&quot;&gt;Crop domestication is a fascinating area of study, as evidencedby a multitude of recent reviews. Coupled with the increasing availability of genomic and phenomic resources in numerous crop species, insights from evolutionary biology will enable a deeper understanding of the genetic architecture and short-term evolution of complex traits, which can be used to inform selection strategies. Future advances in crop improvement will rely on the integration of population genetics with  plant  breeding  methodology,  and  the  development  of  community  resources  to  support research in a variety of crop life histories and reproductive strategies. We highlight recent advances in the role of selective sweeps and demographic history in shaping genetic architecture, how these breakthroughs can inform selection strategies, and the application of precision gene editing to leverage these connections.</code></pre>
</p>
</details>
<br><br />
<details>
<summary>
<p><a
href="http://library.oapen.org/bitstream/handle/20.500.12657/23339/1006816.pdf?sequence=1#page=292">Genomics
of long- and short- term adaptation in maize and teosinte</a> [<a
href="https://peerj.com/manuscripts/31241/">preprint</a>]<br />
<strong>Lorant A</strong>, <strong>Ross-Ibarra J</strong>, Tenaillon
M</p>
</summary>
<p style="margin-left: 30px">
We review studies on short and long term adaptation, both natural and
artificial, in maize and teosinte.
</p>
</details>
<br>
<details>
<summary>
<p><a href="https://www.genetics.org/content/214/4/1019">The temporal
dynamics of background selection in non-equilibrium populations</a> [<a
href="https://www.biorxiv.org/content/10.1101/618389v1">preprint</a>][<a
href="https://github.com/RILAB/BGS_sims/">github</a>]<br />
Torres R, <strong>Stetter MG</strong>, Hernandez R, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We use simulations to show that the effects of nonequilibrium population
dynamics on linked selection differ qualitatively from predictions under
equilibrium models, calling into question inferences about the efficacy
of selection made from summary statistics.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Catherine Rushworth</strong></p>
<pre><code>Rushworth, C.A., R.S. Baucom, B.K. Blackman, M. Neiman, M.E. Orive, A. Sethuraman, J. Ware, and D.R. Matute. 2021. Who are we now? A demographic assessment of the evolution societies. In press, Evolution. Early View: https://onlinelibrary.wiley.com/doi/10.1111/evo.14168.  

Rushworth, C.A., and T. Mitchell-Olds. 2021. The evolution of sex is tempered by costly hybridization in Boechera (rock cress). Journal of Heredity Early View: https://doi.org/10.1093/jhered/esaa041.  

Rushworth, C.A., Y. J. Brandvain, and T. Mitchell-Olds. 2020. Identifying the fitness consequences of sex in complex natural environments. Evolution Letters 4-6: 516-529. https://dx.doi.org/10.1002/evl3.194  

Jordon-Thaden, I., J. Beck, C.A. Rushworth, M.D. Windham, N. Diaz, J. Cantley, C. Martine, and C.J. Rothfels. 2020. RADseq for any lab: a basic two-enzyme protocol performs well in herbarium and silica-dried tissues across four genera. Applications in Plant Sciences 8(4): e11344.  </code></pre>
<p><strong>Silas Tittes</strong></p>
<pre><code>Tittes, Silas. &quot;rdmc: an open source R package implementing convergent adaptation models of Lee and Coop (2017).&quot; G3: Genes, Genomes, Genetics 10.9 (2020): 3041-3046.  

Wooliver, R., Tittes, S. B., &amp; Sheth, S. N. (2020). A resurrection study reveals limited evolution of thermal performance in response to recent climate change across the geographic range of the scarlet monkeyflower. Evolution, 74(8), 1699-1710.  

Comeault, A. A., Wang, J., Tittes, S., Isbell, K., Ingley, S., Hurlbert, A. H., &amp; Matute, D. R. (2020). Genetic diversity and thermal performance in invasive and native populations of African fig flies. Molecular biology and evolution, 37(7), 1893-1906.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-3" class="section level3">
<h3>2019</h3>
<details>
<summary>
<p><a
href="http://www.g3journal.org/cgi/pmidlookup?view=long&amp;pmid=31506319">Dynamic
patterns of transcript abundance of transposable element families in
maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/668558v1">preprint</a>]<br />
Anderson SN, <strong>Stitzer MC</strong>, Zhou P, <strong>Ross-Ibarra
J</strong>, Hirsch CD, Springer NM</p>
</summary>
<p style="margin-left: 30px">
We develop an approach to estiamte family-level expression in
transposable elements. While a relatively small proportion of TE
families are transcribed, expression is highly dynamic with most
families exhibiting tissue-specific expression. We also assay expression
among different maize genotypes and use a mapping population to identify
what proportion of copies in a family contribute to expression.”
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.14489">Transposable
elements contribute to dynamic genome content in maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/547398v1?rss=1">preprint</a>]<br />
Anderson SN*, <strong>Stitzer MC</strong>*, Brohammer AB*, Peng Zhou P,
Noshay JM, Hirsch CD, <strong>Ross-Ibarra J</strong>, Hirsch CN,
Springer NM</p>
</summary>
<p style="margin-left: 30px">
We identify hundreds of thousands of variable transposable elements
among a small set of maize genome assemblies. We find evidence of a
large number of genes impacted by TEs, including TEs in genes and genes
in TEs, and evidence of recent transposition as well.””
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="https://www.genetics.org/content/213/1/143">Hybrid decay: a
transgenerational epigenetic decline in vigor and viability triggered in
backcross populations of teosinte with maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/588715v1">preprint</a>]<br />
Wei X, Anderson SN, Wang X, …[<strong>Bilinski P</strong>,
<strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong> and 4 other
authors]… Chen X, Springer NM, Doebley JF</p>
</summary>
<p style="margin-left: 30px">
We describe a phenomenon we name ‘hybrid decay’ triggered by
backcrossing between domesticated maize and a specific teosinte
population, resulting in genome instability, activation of transposable
elements, and altered epigenetics .”
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://onlinelibrary.wiley.com/doi/full/10.1111/evo.13818">Adaptive
phenotypic divergence in teosinte differs across biotic contexts</a> [<a
href="https://www.biorxiv.org/content/10.1101/382770v2">preprint</a>]
[<a
href="https://onlinelibrary.wiley.com/doi/full/10.1111/evo.13857">perspective</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Strauss SY, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We show that rhizosphere biota impact adaptive divergence of phenotypic
traits in teosinte along an environmental cline. This work suggests an
important role for biotic interactions in determining the outcome of
local adaptation.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="https://peerj.com/articles/6815/">Characterization of
introgression from the teosinte <em>Zea mays</em> ssp. <em>mexicana</em>
to Mexican highland maize</a> [<a
href="https://peerj.com/preprints/27489/">preprint</a>]<br />
Gonzalez-Segovia E, Pérez-Limon S, C Cíntora-Martínez C, Guerrero-Zavala
A, Jansen G, Hufford MB, <strong>Ross-Ibarra J</strong>, Ruairidh J H
Sawers</p>
</summary>
<p style="margin-left: 30px">
We look for introgression from teosinte in the genomes of two highland
landraces and use QTL mapping to test for functional relevance of
introgressed regions.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="https://www.genetics.org/content/211/3/989">Detecting
adaptive differentiation in structured populations with genomic data and
common gardens</a> [<a
href="https://www.biorxiv.org/content/early/2018/07/13/368506">preprint</a>]
[<a href="https://github.com/emjosephs/qpc-maize">github</a>]<br />
<strong>Josephs EB</strong>, Berg JJ, <strong>Ross-Ibarra J</strong>,
Coop G</p>
</summary>
<p style="margin-left: 30px">
We develop methods to detect polygenic adaptation on phenotypes from
common gardens and GWAS panels.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Taylor Perkins</strong></p>
<pre><code>Perkins, M. Taylor, Tetyana Zhebentyayeva, Paul H. Sisco, and J. Hill Craddock. “Genome-wide sequence-based genotyping supports a nonhybrid origin of Castanea alabamensis.” bioRxiv (2019): 680371.   

Craddock, J. Hill, and M. Taylor Perkins. “Chestnut (Castanea spp. Miller) Breeding.” In Advances in Plant Breeding Strategies, vol. 4. (J.M. Al-Khayri et al., eds.)(in press), 55 pages. Springer Nature Switzerland AG, Basel.  </code></pre>
<p><strong>Michelle Stitzer</strong></p>
<pre><code>Roessler, Kyria, Aline Muyle, Concepcion M. Diez, Garren RJ Gaut, Alexandros Bousios, Michelle C. Stitzer, Danelle K. Seymour, John F. Doebley, Qingpo Liu, and Brandon S. Gaut. &quot;The genome-wide dynamics of purging during selfing in maize.&quot; Nature plants (2019): 1-11.  

Noshay, Jaclyn M., Sarah N. Anderson, Peng Zhou, Lexiang Ji, William Ricci, Zefu Lu, Michelle C. Stitzer et al. &quot;Monitoring the interplay between transposable element families and DNA methylation in maize.&quot; PLoS genetics 15, no. 9 (2019): e1008291.     

Marcela K. Tello-Ruiz, Cristina F. Marco, Fei-Man Hsu, Rajdeep S. Khangura, Pengfei Qiao, Sirjan Sapkota, Michelle C. Stitzer, Rachael Wasikowski, Hao Wu, Junpeng Zhan, Kapeel Chougule, Lindsay C. Barone, Cornel Ghiban, Demitri Muna, Andrew C. Olson, Liya C. Wang, Doreen C. Ware, David A. Micklos bioRxiv 654848; doi: https://doi.org/10.1101/654848  

Stitzer, Michelle C., and Philipp Brand. &quot;Digest: Hybrid incompatibilities and introgression in wild monkeyflowers.&quot; Evolution 72, no. 11 (2018): 2565-2566.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-4" class="section level3">
<h3>2018</h3>
<details>
<summary>
<p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007794">Genetic
architecture and selective sweeps after polygenic adaptation to distant
trait optima</a> [<a
href="https://www.biorxiv.org/content/early/2018/05/03/313247">preprint</a>]
[<a
href="https://mgstetter.shinyapps.io/quantgensimAPP/">interactive</a>]<br />
<strong>Stetter MG</strong>, Thornton K, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We use simulations to study the population genetics of selection on a
quantitative trait and learn which parameters drive quantitative trait
evollution.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://www.journals.uchicago.edu/doi/10.1086/700118">Evolutionary
responses to conditionality in species interactions across environmental
gradients</a> [<a
href="http://biorxiv.org/content/early/2017/03/01/031195">preprint</a>][<a
href="https://www.authorea.com/inst/14743?articles_format=list&amp;page=5">review</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, <strong>Ross-Ibarra J</strong>,
Strauss SY</p>
</summary>
<p style="margin-left: 30px">
We present the CoCoA hypothesis which describes how patterns of biotic
coadpation change across varying abiotic environments
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://books.google.com/books?id=ZZl7DwAAQBAJ&amp;pg=PA318&amp;lpg=PA318&amp;dq=maize+genome+snodgrass&amp;source=bl&amp;ots=jD5IvHVlpG&amp;sig=JtkrFv2iBqp4Noq5ud81Y0aIQrU&amp;hl=en&amp;sa=X&amp;ved=2ahUKEwigrdeIkf3eAhXnwlQKHQqgBEEQ6AEwAXoECAMQAQ#v=onepage&amp;q=maize%20genome%20snodgrass&amp;f=false">Evolution
and Adaptation in the Maize Genome</a><br />
Manchanda N, Snodgrass SJ, <strong>Ross-Ibarra J</strong>, Hufford
MB</p>
</summary>
<p style="margin-left: 30px">
We review progress thus far in genomic research of maize domestication
and adaptation. We discuss the insights genomics has shed on our
understanding of these processes and conclude with a future outlook for
how genomics might be further applied.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.15350">Maize
domestication and gene interaction</a> [<a
href="https://peerj.com/preprints/26502/">preprint</a>]<br />
<strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
We review genetic work looking at the role of dominance, epistasis, and
pleiotropy during maize domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">A
novel maize kinesin causes neocentromere activity and meiotic drive,
altering inheritance patterns across the genome</a> [<a
href="http://sci-hub.tw/http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">pdf</a>]
[<a
href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;cad=rja&amp;uact=8&amp;ved=0ahUKEwiIk5W14eHSAhVO92MKHbseACgQtwIIHDAA&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DbcCxsVPwOE8&amp;usg=AFQjCNGJ2xxs5xSbskrcavIoJ0D8ve3fjA&amp;sig2=gmS2FYQxQlrQJRw8PGPDFw">YouTube</a>]<br />
Dawe RK, Lowry EG, Gent JI, <strong>Stitzer MC</strong>, Higgins DM,
<strong>Ross-Ibarra J</strong>, Wallace J, Kanizay LB … [7 authors]</p>
</summary>
<p style="margin-left: 30px">
We identify a small gene family Kindr as the causal locus for meiotic
drive in maize.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162">Parallel
altitudinal clines reveal adaptive evolution of genome size in <em>Zea
mays</em></a> [<a
href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007249">perspective</a>]
[<a
href="http://www.biorxiv.org/content/early/2017/07/13/134528">preprint</a>]
[<a
href="https://github.com/paulbilinski/GenomeSizeAnalysis">github</a>]
[<a
href="https://www.slideshare.net/jrossibarra/parallel-altitudinal-clines-reveal-adaptive-evolution-of-genome-size-in-zea-mays">slides</a>]<br />
<strong>Bilinski P</strong>, Albert P, Berg JJ, Birchler JA, Grote M,
<strong>Lorant A</strong>, <strong>Quezada J</strong>, Swarts K,
<strong>Yang J</strong>, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
We show that genome size can be analyzed as a quantitative trait and
that selection on flowering time in maize has likely driven adaptive
changes in genome size along multiple altitudinal clines.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajb2.1002">Adaptation
in plant genomes: bigger is different</a> [<a
href="https://www.biorxiv.org/content/early/2017/10/02/196501">preprint</a>][<a
href="https://github.com/RILAB/AJB_MutationalTargetSize_GenomeSize">github</a>][<a
href="https://www.youtube.com/watch?v=aXLuWruOmO4_">YouTube</a>][<a
href="https://www.slideshare.net/jrossibarra/adaptation-in-plant-genomes-bigger-is-different">slides</a>]<br />
<strong>Mei W</strong>, <strong>Stetter MG</strong>, <strong>Gates
DJ</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We develop the functional space hypothesis in which we posit that
adaptation in large genomes uses more noncoding variation and is more
likely to proceed via soft sweeps.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://www.sciencedirect.com/science/article/pii/S1462901117310602">Harnessing
cross-border resources to confront climate change</a><br />
Aburto-Oropeza O …[<strong>Ross-Ibarra J</strong> and 85 authors in
alphabetical order]… Taylor JE</p>
</summary>
<p style="margin-left: 30px">
Discusses the challenges of changing climates pose for society and the
environment along the Mexican-US border.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Wenbin Mei</strong> and <strong>Michelle Stitzer</strong></p>
<pre><code>Springer, Nathan M., et al. &quot;The maize W22 genome provides a foundation for functional genomics and transposon biology.&quot; Nature genetics (2018): 1.</code></pre>
<p><strong>Markus Stetter</strong></p>
<pre><code>Joshi, Dinesh C., et al. &quot;From zero to hero: the past, present and future of grain amaranth breeding.&quot; Theoretical and Applied Genetics (2018): 1-17.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B. &quot;Determining the evolutionary forces shaping G× E.&quot; New Phytologist (2018).

Uzunović, Jasmina, et al. &quot;Transposable elements are important contributors to standing variation in gene expression in Capsella grandiflora.&quot; bioRxiv (2018): 289173.</code></pre>
<p><strong>Anne Lorant</strong></p>
<pre><code>Kremling, Karl AG, et al. &quot;Dysregulation of expression correlates with rare-allele burden and fitness loss in maize.&quot; Nature 555.7697 (2018): 520.</code></pre>
<p><strong>Daniel Gates</strong></p>
<pre><code>Gates, Daniel J., Diana Pilson, and Stacey D. Smith. &quot;Filtering of target sequence capture individuals facilitates species tree construction in the plant subtribe Iochrominae (Solanaceae).&quot; Molecular phylogenetics and evolution 123 (2018): 26-34.

Gates, Daniel J., et al. &quot;A novel R3 MYB transcriptional repressor associated with the loss of floral pigmentation in Iochroma.&quot; New Phytologist 217.3 (2018): 1346-1356.  </code></pre>
<p><strong>Dianne Velasco</strong></p>
<pre><code>Riaz, Summaira, et al. &quot;Genetic diversity analysis of cultivated and wild grapevine (Vitis vinifera L.) accessions around the Mediterranean basin and Central Asia.&quot; BMC plant biology 18.1 (2018): 137.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-5" class="section level3">
<h3>2017</h3>
<details>
<summary>
<p><a
href="https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix134/4782225?searchresult=1">Construction
of the third generation <em>Zea mays</em> haplotype map</a> [<a
href="http://biorxiv.org/content/early/2015/09/16/026963">preprint</a>]
[<a href="http://gigadb.org/dataset/100339">sequence data</a>] [<a
href="https://www.panzea.org/genotypes">genotype data</a>] [<a
href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=23">alignment
data</a>] [<a
href="https://bitbucket.org/bukowski1/maize_hapmap3_code">code</a>]<br />
Bukowski R …[16 authors]… <strong>Ross-Ibarra J</strong>, <strong>Lorant
A</strong>, <strong>Buffalo V</strong>, Romay MC, Buckler ES, Ware D,
Lai J, Sun Q, Xu Y</p>
</summary>
<p style="margin-left: 30px">
We present a SNP dataset of 80M variants from a sample of 1200 maize and
teosinte genomes.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1346-4">The
interplay of demography and selection during maize domestication and
diffusion</a> [<a
href="http://www.google.com/url?q=http%3A%2F%2Fbiorxiv.org%2Fcontent%2Fearly%2F2017%2F03%2F07%2F114579&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHr4cDzAxS_7BXVZOfyuIWzXiHr0w">preprint</a>][<a
href="https://github.com/HuffordLab/Wang_et_al._Demography">github</a>][<a
href="https://www.youtube.com/watch?v=KgIUGHEZ2nM">YouTube</a>]<br />
Wang L, <strong>Beissinger TM</strong>, <strong>Lorant A</strong>,
<strong>Ross-Ibarra C</strong>, <strong>Ross-Ibarra J</strong>, Hufford
MB.</p>
</summary>
<p style="margin-left: 30px">
We show how population bottlenecks and gene flow have shaped patterns of
genetic load across the maize genome and among populatons.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007019">Incomplete
dominance of deleterious alleles contribute substantially to trait
variation and heterosis in maize</a> [<a
href="http://biorxiv.org/content/early/2016/11/07/086132">preprint</a>]
[<a href="https://github.com/yangjl/GERP-diallel">github</a>]<br />
<strong>Yang J</strong>, <strong>Mezmouk S</strong>, Baumgarten A,
Buckler ES, Guill KE, McMullen MD, Mumm RH, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
We combine phenotypic data from crosses between multiple maize inbred
lines and genome-wide characterization of deleterious alleles to show
that a simple model of incomplete dominance may help explain hybrid
vigor.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184202">The
potential role of genetic assimilation during maize domestication</a>
[<a
href="http://biorxiv.org/content/early/2017/02/03/105940">preprint</a>]<br />
<strong>Lorant A</strong>, Pedersen, S, Holst I, Hufford MB, Winter K,
Piperno D, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Gene expression data from maize and teosinte grown in mid-Holocene
environments suggests that for some genes plastic changes in expression
may have preceded genetic changes during domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a href="https://peerj.com/articles/3737/">Allele specific expression
analysis identifies regulatory variation associated with stress-related
genes in the Mexican highland maize landrace Palomero Toluqueño</a> [<a
href="http://biorxiv.org/content/early/2017/06/20/152397">preprint</a>]<br />
Aguilar-Rangel MR, Chávez Montes RA, Gonzalez-Segovia E,
<strong>Ross-Ibarra J</strong>, Simpson JK, Sawers RJH</p>
</summary>
<p style="margin-left: 30px">
<p>Background Gene regulatory variation has been proposed to play an
important role in the adaptation of plants to environmental stress. In
the central highlands of Mexico, farmer selection has generated a unique
group of maize landraces adapted to the challenges of the highland
niche. In this study, gene expression in Mexican highland maize and a
reference maize breeding line were compared to identify evidence of
regulatory variation in stress-related genes. It was hypothesised that
local adaptation in Mexican highland maize would be associated with a
transcriptional signature observable even under benign conditions.</p>
<p>Methods Allele specific expression analysis was performed using the
seedling-leaf transcriptome of an F1 individual generated from the cross
between the highland adapted Mexican landrace Palomero Toluqueño and the
reference line B73, grown under benign conditions. Results were compared
with a published dataset describing the transcriptional response of B73
seedlings to cold, heat, salt and UV treatments.</p>
<p>Results A total of 2,386 genes were identified to show allele
specific expression. Of these, 277 showed an expression difference
between Palomero Toluqueño and B73 alleles under benign conditions that
anticipated the response of B73 cold, heat, salt and/or UV treatments,
and, as such, were considered to display a prior stress response. Prior
stress response candidates included genes associated with plant hormone
signaling and a number of transcription factors. Construction of a gene
co-expression network revealed further signaling and stress-related
genes to be among the potential targets of the transcription factors
candidates.</p>
Discussion Prior activation of responses may represent the best strategy
when stresses are severe but predictable. Expression differences
observed here between Palomero Toluqueño and B73 alleles indicate the
presence of cis-acting regulatory variation linked to stress-related
genes in Palomero Toluqueño. Considered alongside gene annotation and
population data, allele specific expression analysis of plants grown
under benign conditions provides an attractive strategy to identify
functional variation potentially linked to local adaptation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://science.sciencemag.org/content/357/6350/512.full">Genomic
estimation of complex traits reveals ancient maize adaptation to
temperate North America</a><br />
Swarts K, Gutaker RM …[<strong>Ross-Ibarra J</strong> and 14 authors in
alphabetical order]… Krause J, Matson RG, Weigel D, Buckler ES, Burbano
HA
</summary>
<p style="margin-left: 30px">
By 4000 years ago, people had introduced maize to the southwestern
United States; full agriculture was established quickly in the lowland
deserts but delayed in the temperate highlands for 2000 years. We test
if the earliest upland maize was adapted for early flowering, a
characteristic of modern temperate maize. We sequenced fifteen
1900-year-old maize cobs from Turkey Pen Shelter in the temperate
Southwest. Indirectly validated genomic models predicted that Turkey Pen
maize was marginally adapted with respect to flowering, as well as
short, tillering, and segregating for yellow kernel color. Temperate
adaptation drove modern population differentiation and was selected in
situ from ancient standing variation. Validated prediction of polygenic
traits improves our understanding of ancient phenotypes and the dynamics
of environmental adaptation. <a
href="http://www.nature.com/news/ancient-genomes-show-how-maize-adapted-to-life-at-high-altitudes-1.22403">Nature
News</a> [<a
href="https://phys.org/news/2017-08-precision-corn-climate.html">Phys.org</a>]
[<a
href="https://www.sciencedaily.com/releases/2017/08/170803141056.htm">Science
Daily</a>]
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://www.sciencedirect.com/science/article/pii/S0960982217307856">How
to make a domesticate</a> [<a
href="https://peerj.com/preprints/2902/">preprint</a>]<br />
<strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Mei
W</strong>, <strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
A perspective that seeks to predict what factors influence the success
of domestication, how many genes contributed to the process, where these
genes originated and the implications for de novo domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature22971.html">Improved
maize reference genome with single-molecule technologies</a> [<a
href="http://biorxiv.org/content/early/2016/12/19/079004">preprint</a>]
[<a
href="https://github.com/mcstitzer/maize_v4_TE_annotation">github</a>]<br />
Jiao Y, Peluso P, Shi J, Liang T, <strong>Stitzer MC</strong>, …[18
authors]… <strong>Ross-Ibarra J</strong>, Dawe K, Hastie A, Rank DR,
Ware D</p>
</summary>
<p style="margin-left: 30px">
We present a high-quality long-read reference genome of the maize inbred
B73 and the first detailed annotation of retroelements in any grass
species. [<a
href="https://news.science360.gov/obj/story/2caf076f-1759-40c2-9c56-62e3fbbdf44b/tracking-down-jumping-genes-maize">NSF
research news</a>] [<a href="https://t.co/3MkC6eRDyD">UCD press
release</a>]
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="https://academic.oup.com/mbe/article/34/8/1825/3742530/Gene-Fractionation-and-Function-in-the-Ancient?guestAccessKey=391f8f04-5fd6-4bc7-bac0-0057fb3102ab">Gene
fractionation and function in the ancient subgenomes of maize</a> [<a
href="http://biorxiv.org/content/early/2016/12/19/095547">preprint</a>]<br />
<strong>Renny-Byfield S</strong>, Rodgers-Melnick E, <strong>Ross-Ibarra
J</strong></p>
</summary>
<p style="margin-left: 30px">
The maize genome experienced an ancient whole genome duplication ∼10 MYA
and the duplicate subgenomes have since experienced reciprocal gene loss
such that many genes have returned to single-copy status. This process
has not affected the subgenomes equally; reduced gene expression in one
of the subgenomes mitigates the consequences of mutations and gene
deletions and is thought to drive higher rates of fractionation. Here,
we use published data to show that, in accordance with predictions of
this model, paralogs with greater expression contribute more to
phenotypic variation compared with their lowly expressed counterparts.
Furthermore, paralogous genes in the least-fractionated subgenome
account for a greater degree of phenotypic diversity than those resident
on the more-fractionated subgenome. Intriguingly, analysis of singleton
genes reveals this difference persists even after fractionation is
complete. Additionally, we show that the two subgenomes of maize may
differ in their epigenetic profiles.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><a
href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177896">Genomic
abundance is not predictive of tandem repeat localization in grass
genomes</a> [<a href="https://peerj.com/preprints/2314/">preprint</a>]
[<a
href="https://github.com/paulbilinski/Github_centrepeat">github</a>]<br />
<strong>Bilinski P</strong>, Han Y, <strong>Hufford MB</strong>,
<strong>Lorant A</strong>, Zhang P, Estep MC, Jiang J,
<strong>Ross-Ibarra J</strong></p>
</summary>
<p style="margin-left: 30px">
Highly repetitive regions have historically posed a challenge when
investigating sequence variation and content. High-throughput sequencing
has enabled researchers to use whole-genome shotgun sequencing to
estimate the abundance of repetitive sequence, and these methodologies
have been recently applied to centromeres. Previous research has
investigated variation in centromere repeats across eukaryotes, positing
that the highest abundance tandem repeat in a genome is often the
centromeric repeat. To test this assumption, we used shotgun sequencing
and a bioinformatic pipeline to identify common tandem repeats across a
number of grass species. We find that de novo assembly and subsequent
abundance ranking of repeats can successfully identify tandem repeats
with homology to known tandem repeats. Fluorescent in-situ hybridization
shows that de novo assembly and ranking of repeats from non-model taxa
identifies chromosome domains rich in tandem repeats both near
pericentromeres and elsewhere in the genome.
</p>
</details>
<p><br></p>
<details>
<summary>
<strong>Other papers by lab members</strong>
</summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, et al. &quot;Genetic and ecological insights into glacial refugia of walnut (Juglans regia L.).&quot; PloS one 12.10 (2017): e0185974.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B., et al. &quot;The relationship between selection, network connectivity, and regulatory variation within a population of Capsella grandiflora.&quot; Genome biology and evolution 9.4 (2017): 1099-1109.

Josephs, Emily B., John R. Stinchcombe, and Stephen I. Wright. &quot;What can genome‐wide association studies tell us about the evolutionary forces maintaining genetic variation for quantitative traits?.&quot; New Phytologist 214.1 (2017): 21-33.</code></pre>
<p><strong>Wenbin Mei</strong></p>
<pre><code>Mei, Wenbin, et al. &quot;A comprehensive analysis of alternative splicing in paleopolyploid maize.&quot; Frontiers in plant science 8 (2017): 694.

Liu, Xiaoxian, et al. &quot;Detecting alternatively spliced transcript isoforms from single‐molecule long‐read sequences without a reference genome.&quot; Molecular ecology resources 17.6 (2017): 1243-1256.

Gault, Christine M., et al. &quot;Aberrant splicing in maize rough endosperm3 reveals a conserved role for U12 splicing in eukaryotic multicellular development.&quot; Proceedings of the National Academy of Sciences (2017): 201616173.

Feng, Guanqiao, et al. &quot;Evolution of the 3R-MYB gene family in plants.&quot; Genome biology and evolution 9.4 (2017): 1013-1029.

Mei, Wenbin, et al. &quot;Evolutionarily conserved alternative splicing across monocots.&quot; Genetics (2017): genetics-300189.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-6" class="section level3">
<h3>2016</h3>
<details>
<summary>
<a
href="http://www.cell.com/current-biology/abstract/S0960-9822(16)31120-4">Genome
sequence of a 5310-year-old maize cob provides insights into the early
stages of maize domestication</a><br />
Ramos-Madrigal J, Smith BD, Moreno-Mayar JV, Gopalakrishnan S,
<strong>Ross-Ibarra J</strong>, Gilbert MTP, Wales N
</summary>
<p style="margin-left: 30px">
Genome sequence of a 5,000 year old maize cob reveals it was partially
domesticated and helps identify the timing of adaptation during
domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.g3journal.org/content/6/12/3985.short?rss=1&amp;ssource=mfr">Evolutionary
genomics of peach and almond domestication</a> [<a
href="http://biorxiv.org/content/early/2016/09/27/060160">preprint</a>]<br />
<strong>Velasco D</strong>, <strong>Hough J</strong>, Aradhya M,
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Resequencing data from a number of peach and almond varieties show
differences in population size and mating system, but little evidence
for convergent selection during domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full">ANGSD-wrapper:
utilities for analyzing next generation sequencing data</a> [<a
href="https://github.com/mojaveazure/angsd-wrapper">github</a>] [<a
href="https://peerj.com/preprints/1472/">preprint</a>]<br />
<strong>Durvasula A</strong>, Hoffman PJ, <strong>Kent TV</strong>, Liu
C, Kono TJY, Morrell PL, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We provide a set of scripts and configuration files to facilitate easy
popgen analysis of short-read sequencing data.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rdcu.be/kHEB">Recent demography drives changes in linked
selection across the maize genome</a> [<a
href="https://github.com/timbeissinger/Maize-Teo-Scripts/">github</a>]
[<a
href="http://biorxiv.org/content/early/2015/11/13/031666">preprint</a>]<br />
<strong>Beissinger TM</strong>, Wang L, <strong>Crosby K</strong>,
<strong>Durvasula A</strong>, Hufford MB, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We find little evidence for widespread hard sweeps in maize or teosinte
and show that differences in demography during maize domestication and
subsequent expansion have have changed the impacts of linked selection
on surrounding diversity in the maize genome.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4834384/">Maize
diversity associated with social origin and environmental variation in
southern Mexico</a> [<a
href="https://peerj.com/preprints/1192/">preprint</a>]<br />
Orozco-Ramírez Q, Santacruz-Varela A, <strong>Ross-Ibarra J</strong>,
Brush S
</summary>
<p style="margin-left: 30px">
While prevailing theories of crop evolution suggest that crop diversity
and cultural diversity should be linked, empirical evidence for such a
link remains inconclusive. In particular, few studies have investigated
such patterns on a local scale. Here, we address this issue by examining
the determinants of maize diversity in a local region of high cultural
and biological richness in Southern Mexico. We collected maize samples
from villages at low and middle elevations in two adjacent
municipalities of differing ethnicity: Mixtec or Chatino. Although
morphological traits show few patterns of population structure, we see
clear genetic differentiation among villages, with municipality
explaining a larger proportion of the differentiation than altitude.
Consistent with an important role of social origin in patterning seed
exchange, metapopulation model-based estimates of differentiation match
the genetic data within village and ethnically distinct municipalities,
but underestimate differentiation when all four villages are taken
together. Our research provides insights about the importance of social
origin in structuring maize diversity at the local scale.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://journal.frontiersin.org/article/10.3389/fpls.2016.00308/full">High
quality maize centromere 10 sequence reveals evidence of frequent
recombination events</a><br />
Wolfgruber TK, Nakashima MM …[5 authors]… <strong>Bilinski P</strong>,
Dawe RK, <strong>Ross-Ibarra J</strong>, Birchler JA, Presting G
</summary>
<p style="margin-left: 30px">
The ancestral centromeres of maize contain long stretches of the
tandemly arranged CentC repeat. The abundance of tandem DNA repeats and
centromeric retrotransposons (CR) has presented a significant challenge
to completely assembling centromeres using traditional sequencing
methods. Here, we report a nearly complete assembly of the 1.85 Mb maize
centromere 10 from inbred B73 using PacBio technology and BACs from the
reference genome project. The error rates estimated from overlapping BAC
sequences are 7 × 10−6 and 5 × 10−5 for mismatches and indels,
respectively. The number of gaps in the region covered by the reassembly
was reduced from 140 in the reference genome to three. Three expressed
genes are located between 92 and 477 kb from the inferred ancestral
CentC cluster, which lies within the region of highest centromeric
repeat density. The improved assembly increased the count of full-length
CR from 5 to 55 and revealed a 22.7 kb segmental duplication that
occurred approximately 121,000 years ago. Our analysis provides evidence
of frequent recombination events in the form of partial
retrotransposons, deletions within retrotransposons, chimeric
retrotransposons, segmental duplications including higher order CentC
repeats, a deleted CentC monomer, centromere-proximal inversions, and
insertion of mitochondrial sequences. Double-strand DNA break (DSB)
repair is the most plausible mechanism for these events and may be the
major driver of centromere repeat evolution and diversity. In many cases
examined here, DSB repair appears to be mediated by microhomology,
suggesting that tandem repeats may have evolved to efficiently repair
frequent DSBs in centromeres.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Migicovsky, Zoë, et al. &quot;Genomic ancestry estimation quantifies use of wild species in grape breeding.&quot; BMC genomics 17.1 (2016): 478.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B., and Stephen I. Wright. &quot;On the Trail of Linked Selection.&quot; PLoS genetics 12.8 (2016): e1006240.</code></pre>
<p><strong>Tim Beissinger</strong></p>
<pre><code>Morota, Gota, Timothy M. Beissinger, and Francisco Peñagaricano. &quot;MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products in chicken.&quot; G3: Genes, Genomes, Genetics (2016): g3-116.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-7" class="section level3">
<h3>2015</h3>
<details>
<summary>
<a href="https://pubmed.ncbi.nlm.nih.gov/26385980/">The genomic impacts
of drift and selection for hybrid performance in maize</a> [<a
href="http://arxiv.org/abs/1307.7313">preprint</a>]<br />
Gerke JP, Edwards JW, Guill KE, <strong>Ross-Ibarra J</strong>, McMullen
MD
</summary>
<p style="margin-left: 30px">
Although maize is naturally an outcrossing organism, modern breeding
utilizes highly inbred lines in controlled crosses to produce hybrids.
The U.S. Department of Agriculture’s reciprocal recurrent selection
experiment between the Iowa Stiff Stalk Synthetic (BSSS) and the Iowa
Corn Borer Synthetic No. 1 (BSCB1) populations represents one of the
longest running experiments to understand the response to selection for
hybrid performance. To investigate the genomic impact of this selection
program, we genotyped the progenitor lines and &gt;600 individuals
across multiple cycles of selection using a genome-wide panel of ∼40,000
SNPs. We confirmed previous results showing a steady temporal decrease
in genetic diversity within populations and a corresponding increase in
differentiation between populations. Thanks to detailed historical
information on experimental design, we were able to perform extensive
simulations using founder haplotypes to replicate the experiment in the
absence of selection. These simulations demonstrate that while most of
the observed reduction in genetic diversity can be attributed to genetic
drift, heterozygosity in each population has fallen more than expected.
We then took advantage of our high-density genotype data to identify
extensive regions of haplotype fixation and trace haplotype ancestry to
single founder inbred lines. The vast majority of regions showing such
evidence of selection differ between the two populations, providing
evidence for the dominance model of heterosis. We discuss how this
pattern is likely to occur during selection for hybrid performance and
how it poses challenges for dissecting the impacts of modern breeding
and selection on the maize genome.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://www.nature.com/articles/ng.3422">Seed filling in
domesticated maize and rice depends on SWEET-mediated hexose
transport</a> [<a
href="http://sci-hub.tw/https://www.nature.com/articles/ng.3422">pdf</a>]<br />
Sosso D, Luo D, Li Q-B, Sasse J, <strong>Yang J</strong> …[6 authors]…
<strong>Ross-Ibarra J</strong>, Yang B, Frommer WB
</summary>
<p style="margin-left: 30px">
Carbohydrate import into seeds directly determines seed size and must
have been increased through domestication. However, evidence of the
domestication of sugar translocation and the identities of seed-filling
transporters have been elusive. Maize ZmSWEET4c, as opposed to its
sucrose-transporting homologs, mediates transepithelial hexose transport
across the basal endosperm transfer layer (BETL), the entry point of
nutrients into the seed, and shows signatures indicative of selection
during domestication. Mutants of both maize ZmSWEET4c and its rice
ortholog OsSWEET4 are defective in seed filling, indicating that a lack
of hexose transport at the BETL impairs further transfer of sugars
imported from the maternal phloem. In both maize and rice, SWEET4 was
likely recruited during domestication to enhance sugar import into the
endosperm.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://www.genetics.org/content/200/4/1297">Independent
molecular basis of convergent highland adaptation in maize</a> [<a
href="http://biorxiv.org/content/early/2015/01/09/013607">preprint</a>]
[<a href="https://github.com/rossibarra/hilo_paper">github</a>] [<a
href="https://dx.doi.org/10.6084/m9.figshare.746990.v1">data</a>]<br />
<strong>Takuno S</strong>, Ralph P, Swarts K, Elshire RJ, Glaubitz JC,
Buckler ES, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We find that maize adaptation to the highlands in Mexico and South
America is largely independent and has made considerable use of standing
genetic variation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://peerj.com/articles/900/">Natural variation in teosinte
at the domestication locus teosinte branched1 (tb1)</a> [<a
href="https://peerj.com/preprints/685/">preprint</a>]<br />
<strong>Vann LE</strong>, <strong>Kono TJ</strong>, <strong>Pyhäjärvi
T</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>.
</summary>
<p style="margin-left: 30px">
We find that variation in the domestication locus tb1 does not explain a
substantial proportion of phenotypic variation for tillering in natural
populations of teosinte.
</p>
</details>
<p><br> <!--  [[github](https://github.com/lvann/tb1)] --></p>
<details>
<summary>
<a href="http://elifesciences.org/content/4/e05861">Genetic,
evolutionary and plant breeding insights from the domestication of
maize</a><br />
Hake S, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
The natural history of maize began nine thousand years ago when Mexican
farmers started to collect the seeds of the wild grass, teosinte.
Invaluable as a food source, maize permeated Mexican culture and
religion. Its domestication eventually led to its adoption as a model
organism, aided in large part by its large chromosomes, ease of
pollination and growing agricultural importance. Genome comparisons
between varieties of maize, teosinte and other grasses are beginning to
identify the genes responsible for the domestication of modern maize and
are also providing ideas for the breeding of more hardy varieties.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rdcu.be/kHD8">The origin and evolution of maize in the
American Southwest</a> [<a
href="http://biorxiv.org/content/early/2015/01/11/013540">preprint</a>]
[<a
href="http://popgen.dk/fonseca/NatPlants/Capture-c2/">data</a>]<br />
Fonseca RR, Smith B, Wales N, Cappellini E …[12 authors]… Hufford MB,
Albrechtsen A, <strong>Ross-Ibarra J</strong>, Gilbert MT
</summary>
<p style="margin-left: 30px">
We use sequence data from ancient maize cobs to reveal the timing and
geography of maize introduction into the Southwest US. One of the <a
href="https://www.nature.com/collections/dnshkjhpxx/content/most-read">most
read papers</a> from the last four years of the journal! <a
href="http://news.ucdavis.edu/search/news_detail.lasso?id=11124">UC
Davis Press Release</a> <a
href="http://www.scienceworldreport.com/articles/21122/20150112/origins-maize-dna-tests-reveal-two-paths.htm">Science
World Report</a> <a
href="http://www.archaeology.org/news/2879-150108-southwest-corn-dna">Archaeology</a>
<a
href="http://www.agenciasinc.es/Noticias/El-maiz-entro-en-el-suroeste-de-EE-UU-hace-4.000-anos-a-traves-de-la-montana">SiNC</a>,
<a
href="http://news.sciencemag.org/archaeology/2015/01/how-corn-became-corn">Science
magazine</a> <a
href="http://www.nature.com/articles/nplants20149">Nature Plants
commentary</a> <a
href="http://www.the-scientist.com/?articles.view/articleNo/41865/title/Corn-Chronicle/">The
Scientist</a>
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/dyer2015.pdf">Reply to Brush
<em>et al.</em>: Wake-up call for crop conservation science</a><br />
Dyer GA, Lopez-Feldman A, Yunez-Naude A, Taylor JE, <strong>Ross-Ibarra
J</strong>
</summary>
<p style="margin-left: 30px">
A response to <a
href="https://www.pnas.org/doi/10.1073/pnas.1422010112">Brush et al
2014</a>
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004915">Transposable
Elements Contribute to Activation of Maize Genes in Response to Abiotic
Stress</a> [<a
href="http://biorxiv.org/content/early/2014/08/15/008052">preprint</a>]<br />
Makarevitch I, Waters M, West P, <strong>Stitzer M</strong>,
<strong>Ross-Ibarra J</strong>, Springer NM
</summary>
<p style="margin-left: 30px">
We show that transposable elements can act as regulatory sequences that
up- or down-regulate genes in maize in response to abiotic stress.
</p>
</details>
<p><br></p>
<details>
<summary>
<strong>Other papers by lab members</strong>
</summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, J. E. Preece, and Dianne Velasco. &quot;Multivariate analysis of molecular and morphological diversity in fig (Ficus carica L.).&quot; V International Symposium on Fig 1173. 2015.</code></pre>
<p><strong>Paul Bilinski</strong></p>
<pre><code>Leung, Wilson, et al. &quot;Drosophila Muller F elements maintain a distinct set of genomic properties over 40 million years of evolution.&quot; G3: Genes, Genomes, Genetics 5.5 (2015): 719-740.</code></pre>
<p><strong>Anna O’Brien</strong></p>
<pre><code>Persson, Tomas, et al. &quot;Candidatus Frankia datiscae Dg1, the actinobacterial microsymbiont of Datisca glomerata, expresses the canonical nod genes nodABC in symbiosis with its host plant.&quot; PLoS One 10.5 (2015): e0127630.</code></pre>
<p><strong>Shohei Takuno</strong></p>
<pre><code>Le, Tu N., et al. &quot;Epigenetic regulation of intragenic transposable elements impacts gene transcription in Arabidopsis thaliana.&quot; Nucleic acids research 43.8 (2015): 3911-3921.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-8" class="section level3">
<h3>2014</h3>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/tiffin2014.pdf">Advances and
limits of using population genetics to understand local adaptation</a>
[<a href="https://peerj.com/preprints/488/">preprint</a>] [<a
href="https://www.cell.com/trends/ecology-evolution/fulltext/S0169-5347(17)30188-X">correction</a>]<br />
Tiffin P, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We discuss some of the outstanding questions and key challenges facing
the use of population genetics to study local adaptation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/Bilinski2014.pdf">Diversity and
evolution of centromere repeats in the maize genome</a> [<a
href="http://biorxiv.org/content/early/2014/05/12/005058">preprint</a>]<br />
<strong>Bilinski P</strong>, <strong>Distor KD</strong>,
<strong>Gutierez-Lopez J</strong>, <strong>Mendoza Mendoza G</strong>,
Shi J, Dawe K, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Centromere repeats are found in most eukaryotes and play a critical role
in kinetochore formation. Though centromere repeats exhibit considerable
diversity both within and among species, little is understood about the
mechanisms that drive centromere repeat evolution. Here, we use maize as
a model to investigate how a complex history involving polyploidy,
fractionation, and recent domestication has im- pacted the diversity of
the maize centromeric repeat CentC. We first validate the existence of
long tandem arrays of repeats in maize and other taxa in the genus Zea.
Although we find considerable sequence diversity among CentC copies
genome-wide, genetic similarity among repeats is highest within these
arrays, suggesting that tandem du- plications are the primary mechanism
for the generation of new copies. Nonetheless, clustering analyses
identify similar sequences among distant repeats, and simulations
suggest that this pattern may be due to homoplasious mutation. Although
the two ancestral subgenomes of maize have contributed nearly equal
numbers of centro- meres, our analysis shows that the majority of all
CentC repeats derive from one of the parental genomes, with an even
stronger bias when examining the largest as- sembled contiguous
clusters. Finally, by comparing maize with its wild progenitor teosinte,
we find that the abundance of CentC likely decreased after domesti-
cation, while the pericentromeric repeat Cent4 has drastically
increased.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://www.g3journal.org/content/4/1/163.full">The pattern and
distribution of deleterious mutations in maize</a> [<a
href="http://arxiv.org/abs/1308.0380">preprint</a>] [<a
href="https://github.com/RILAB/siftmappR">github</a>]<br />
<strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We show that deleterious mutations are abundant in modern maize breeding
material and that genes with deleterious mutations are likely important
contributors to phenotypic variation.
</p>
</details>
<p><br></p>
<details>
<summary>
<strong>Other papers by lab members</strong>
</summary>
<p style="margin-left: 30px">
</p>
</details>
<p><br></p>
</div>
<div id="section-9" class="section level3">
<h3>2013</h3>
<details>
<summary>
<a
href="http://www.pnas.org/content/110/48/19639.abstract">Comprehensive
analysis of imprinted genes in maize reveals allelic variation and
limited conservation with other species</a> [<a
href="http://arxiv.org/abs/1307.7678">preprint</a>] [<a
href="https://github.com/paulbilinski/Waters_etal_SupplementalTableAnalysis">github</a>]<br />
Waters AJ, <strong>Bilinski P</strong>, Eichten SR, Vaugh MW,
<strong>Ross-Ibarra J</strong>, Gehring M, Springer NM
</summary>
<p style="margin-left: 30px">
In plants, a subset of genes exhibit imprinting in endosperm tissue such
that expression is primarily from the maternal or paternal allele.
Imprinting may arise as a consequence of mechanisms for silencing of
transposons during reproduction, and in some cases imprinted expression
of particular genes may provide a selective advantage such that it is
conserved across species. Separate mechanisms for the origin of
imprinted expression patterns and maintenance of these patterns may
result in substantial variation in the targets of imprinting in
different species. Here we present deep sequencing of RNAs isolated from
reciprocal crosses of four diverse maize genotypes, providing a
comprehensive analysis that allows evaluation of imprinting at more than
95% of endosperm-expressed genes. We find that over 500 genes exhibit
statistically significant parent-of-origin effects in maize endosperm
tissue, but focused our analyses on a subset of these genes that had
&gt;90% expression from the maternal allele (69 genes) or from the
paternal allele (108 genes) in at least one reciprocal cross. Over 10%
of imprinted genes show evidence of allelic variation for imprinting. A
comparison of imprinting in maize and rice reveals that 13% of genes
with syntenic orthologs in both species exhibit conserved imprinting.
Genes that exhibit conserved imprinting between maize and rice have
elevated nonsynonymous to synonymous substitution ratios compared with
other imprinted genes, suggesting a history of more rapid evolution.
Together, these data suggest that imprinting only has functional
relevance at a subset of loci that currently exhibit imprinting in
maize.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://gbe.oxfordjournals.org/content/5/9/1594">Complex
patterns of local adaptation in teosinte</a> [<a
href="http://datadryad.org/resource/doi:10.5061/dryad.8m648/1">data</a>]
[<a href="http://arxiv.org/abs/1208.0634">preprint</a>]<br />
<strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>,
<strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Populations of widely distributed species encounter and must adapt to
local environmental conditions. However, comprehensive characterization
of the genetic basis of adaptation is demanding, requiring genome-wide
genotype data, multiple sampled populations, and an understanding of
population structure and potential selection pressures. Here, we used
single-nucleotide polymorphism genotyping and data on numerous
environmental variables to describe the genetic basis of local
adaptation in 21 populations of teosinte, the wild ancestor of maize. We
found complex hierarchical genetic structure created by altitude,
dispersal events, and admixture among subspecies, which complicated
identification of locally beneficial alleles. Patterns of linkage
disequilibrium revealed four large putative inversion polymorphisms
showing clinal patterns of frequency. Population differentiation and
environmental correlations suggest that both inversions and intergenic
polymorphisms are involved in local adaptation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003604">From
many, one: genetic control of prolificacy during maize domestication</a>
[<a
href="http://figshare.com/articles/Wills_et_al_2013/775358">data</a>]
[<a href="http://arxiv.org/abs/1303.0882">preprint</a>]<br />
Wills DM, Whipple C, <strong>Takuno S</strong>, Kursel LE, Shannon LM,
<strong>Ross-Ibarra J</strong>, Doebley JF.
</summary>
<p style="margin-left: 30px">
A reduction in number and an increase in size of inflorescences is a
common aspect of plant domestication. When maize was domesticated from
teosinte, the number and arrangement of ears changed dramatically.
Teosinte has long lateral branches that bear multiple small ears at
their nodes and tassels at their tips. Maize has much shorter lateral
branches that are tipped by a single large ear with no additional ears
at the branch nodes. To investigate the genetic basis of this difference
in prolificacy (the number of ears on a plant), we performed a
genome-wide QTL scan. A large effect QTL for prolificacy (prol1.1) was
detected on the short arm of chromosome 1 in a location that has
previously been shown to influence multiple domestication traits. We
fine-mapped prol1.1 to a 2.7 kb “causative region” upstream of the
grassy tillers1 (gt1) gene, which encodes a homeodomain leucine zipper
transcription factor. Tissue in situ hybridizations reveal that the
maize allele of prol1.1 is associated with up-regulation of gt1
expression in the nodal plexus. Given that maize does not initiate
secondary ear buds, the expression of gt1 in the nodal plexus in maize
may suppress their initiation. Population genetic analyses indicate
positive selection on the maize allele of prol1.1, causing a partial
sweep that fixed the maize allele throughout most of domesticated maize.
This work shows how a subtle cis-regulatory change in tissue specific
gene expression altered plant architecture in a way that improved the
harvestability of maize.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/McCouch2013.pdf">Agriculture:
Feeding the future</a><br />
McCouch S, Baute GJ, Bradeen J, Bramel P …[36 authors]… Ward J, Waugh R,
Wenzl P, Zamir.
</summary>
<p style="margin-left: 30px">
We must mine the biodiversity in seed banks to help to overcome food
shortages, urge Susan McCouch and colleagues.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003477">The
genomic signature of crop-wild introgression in maize</a> [<a
href="http://figshare.com/articles/Hufford_et_al_2013/766356">data</a>]
[<a href="http://arxiv.org/abs/1208.3894">preprint</a>]<br />
<strong>Hufford MB</strong>, Lubinsky P, <strong>Pyhäjärvi T</strong>,
<strong>Devengenzo MT</strong>, Ellstrand NC, <strong>Ross-Ibarra
J</strong>
</summary>
<p style="margin-left: 30px">
The evolutionary significance of hybridization and subsequent
introgression has long been appreciated, but evaluation of the
genome-wide effects of these phenomena has only recently become
possible. Crop-wild study systems represent ideal opportunities to
examine evolution through hybridization. For example, maize and the
conspecific wild teosinte Zea mays ssp. mexicana (hereafter, mexicana)
are known to hybridize in the fields of highland Mexico. Despite
widespread evidence of gene flow, maize and mexicana maintain distinct
morphologies and have done so in sympatry for thousands of years.
Neither the genomic extent nor the evolutionary importance of
introgression between these taxa is understood. In this study we
assessed patterns of genome-wide introgression based on 39,029 single
nucleotide polymorphisms genotyped in 189 individuals from nine
sympatric maize-mexicana populations and reference allopatric
populations. While portions of the maize and mexicana genomes appeared
resistant to introgression (notably near known cross-incompatibility and
domestication loci), we detected widespread evidence for introgression
in both directions of gene flow. Through further characterization of
these genomic regions and preliminary growth chamber experiments, we
found evidence suggestive of the incorporation of adaptive mexicana
alleles into maize during its expansion to the highlands of central
Mexico. In contrast, very little evidence was found for adaptive
introgression from maize to mexicana. The methods we have applied here
can be replicated widely, and such analyses have the potential to
greatly inform our understanding of evolution through introgressive
hybridization. Crop species, due to their exceptional genomic resources
and frequent histories of spread into sympatry with relatives, should be
particularly influential in these studies.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/provance2013.pdf">Population
genetics and ethnobotany of cultivated <em>Diospyros riojae</em>
(Ebenaceae), an endangered fruit crop from Mexico</a> [<a
href="http://figshare.com/articles/Population_Genetics_and_Ethnobotany_of_Cultivated_Diospyros_riojae_G_mez_Pompa_Ebenaceae_an_Endangered_Fruit_Crop_from_Mexico/105841">preprint</a>]<br />
<strong>Provance MC</strong>, Garcia Ruiz I, <strong>Thommes C</strong>,
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
The traditional cultivation of an endan- gered species, Diospyros
riojae, in North-Central Mexico, and the sale of its fruit, is described
for the first time. This is complemented by the first examina- tion of
genetic structure in New World Ebenaceae. Genetic comparisons are made
among 27 accessions of D. riojae from across its known range of
cultivation, and individuals of Diospyros conzattii, Diospyros digyna,
Diospyros californica, Diospyros rosei, Diospyros rekoi and an unknown
species of Diospyros from Oaxaca, Mexico. Morphological comparisons are
made between D. conzattii, D. riojae, and the unknown species of
Diospyros from Oaxaca. D. riojae is cultivated in two markedly different
climates: arid and semi-arid localities west of the Sierra Gorda,
especially near Tecozautla, Hidalgo, and humid areas of the Huasteca
Region, east of the Sierra Gorda. Much lower levels of genetic diversity
were detected in western populations, where populations are larger, and
management intensity is the greatest. Neither the results of our genetic
analysis, nor our morphological analysis of recent collections of D.
riojae, D. conzattii, and an unknown species from Oaxaca, are consistent
with the most recent revision of Neotropical Diospy- ros. Taxon-level
divergence, rather than cultivar-level divergence, is suggested for D.
riojae and D. conzattii.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2013-14-1-r10">Comparative
analysis of tandem repeats from hundreds of species reveals unique
insights into centromere evolution</a> [<a
href="http://arxiv.org/abs/1209.4967">preprint</a>]<br />
Melters DP, Bradnam KR, Young HA, Telis N …[9 authors]… Tobias C,
<strong>Ross-Ibarra J</strong>, Korf IF, Chan SW-L.
</summary>
<p style="margin-left: 30px">
<p><strong>Background</strong><br />
Centromeres are essential for chromosome segregation, yet their DNA
sequences evolve rapidly. In most animals and plants that have been
studied, centromeres contain megabase-scale arrays of tandem repeats.
Despite their importance, very little is known about the degree to which
centromere tandem repeats share common properties between different
species across different phyla. We used bioinformatic methods to
identify high-copy tandem repeats from 282 species using publicly
available genomic sequence and our own data.</p>
<p><strong>Results</strong> Our methods are compatible with all current
sequencing technologies. Long Pacific Biosciences sequence reads allowed
us to find tandem repeat monomers up to 1,419 bp. We assumed that the
most abundant tandem repeat is the centromere DNA, which was true for
most species whose centromeres have been previously characterized,
suggesting this is a general property of genomes. High-copy centromere
tandem repeats were found in almost all animal and plant genomes, but
repeat monomers were highly variable in sequence composition and length.
Furthermore, phylogenetic analysis of sequence homology showed little
evidence of sequence conservation beyond approximately 50 million years
of divergence. We find that despite an overall lack of sequence
conservation, centromere tandem repeats from diverse species showed
similar modes of evolution.</p>
<strong>Conclusions</strong> While centromere position in most
eukaryotes is epigenetically determined, our results indicate that
tandem repeats are highly prevalent at centromeres of both animal and
plant genomes. This suggests a functional role for such repeats, perhaps
in promoting concerted evolution of centromere DNA across chromosomes.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://www.nature.com/articles/hdy20132">Diversity and
abundance of the Abnormal chromosome 10 meiotic drive complex in <em>Zea
mays</em></a> [<a
href="http://arxiv.org/abs/1209.5466">preprint</a>]<br />
Kanizay LB, <strong>Pyhäjärvi T</strong>, Lowry E, <strong>Hufford
MB</strong>, Peterson DG, <strong>Ross-Ibarra J</strong>, Dawe RK
</summary>
<p style="margin-left: 30px">
Maize Abnormal chromosome 10 (Ab10) contains a classic meiotic drive
system that exploits the asymmetry of meiosis to preferentially transmit
itself and other chromosomes containing specialized heterochromatic
regions called knobs. The structure and diversity of the Ab10 meiotic
drive haplotype is poorly understood. We developed a bacterial
artificial chromosome (BAC) library from an Ab10 line and used the data
to develop sequence-based markers, focusing on the proximal portion of
the haplotype that shows partial homology to normal chromosome 10. These
molecular and additional cytological data demonstrate that two
previously identified Ab10 variants (Ab10-I and Ab10-II) share a common
origin. Dominant PCR markers were used with fluorescence in situ
hybridization to assay 160 diverse teosinte and maize landrace
populations from across the Americas, resulting in the identification of
a previously unknown but prevalent form of Ab10 (Ab10-III). We find that
Ab10 occurs in at least 75% of teosinte populations at a mean frequency
of 15%. Ab10 was also found in 13% of the maize landraces, but does not
appear to be fixed in any wild or cultivated population. Quantitative
analyses suggest that the abundance and distribution of Ab10 is governed
by a complex combination of intrinsic fitness effects as well as
extrinsic environmental variability.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, et al. &quot;Genetic diversity, structure, and patterns of differentiation in the genus Vitis.&quot; Plant systematics and evolution 299.2 (2013): 317-330.

Zdunić, Goran, et al. &quot;Genetic diversity and differentiation within and between cultivated (Vitis vinifera L. ssp. sativa) and wild (Vitis vinifera L. ssp. sylvestris) grapes.&quot; Vitis 52.1 (2013): 29-32.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-10" class="section level3">
<h3>2012</h3>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/hufford2012.pdf">Teosinte as a
model system for population and ecological genomics</a><br />
<strong>Hufford MB</strong>, <strong>Bilinski P</strong>,
<strong>Pyhäjärvi T</strong>, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
As the cost of next-generation sequencing diminishes and genomic
resources improve, crop wild relatives are well positioned to make major
contributions to the field of ecological genomics via full-genome
resequencing and reference-assisted de novo assembly of genomes of
plants from natural populations. The wild relatives of maize,
collectively known as teosinte, are a more varied and representative
study system than many other model flowering plants. In this review of
the population and ecological genomics of the teosintes we highlight
recent advances in the study of maize domestication, introgres- sive
hybridization, and local adaptation, and discuss future prospects for
applying the genomic resources of maize to this intriguing group of
species. The maize/ teosinte study system is an excellent example of how
crops and their wild relatives can bridge the model/ non-model gap.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_24240aa930bf4728be83b4b57da55f17.pdf">Comparative
population genomics of maize domestication and improvement</a> [<a
href="http://figshare.com/articles/new_fileset/757736">data</a>]<br />
<strong>Hufford MB</strong>*, Xun X*, <strong>van Heerwaarden
J</strong>*, <strong>Pyhäjärvi T</strong>* …[17 authors]… Ware D,
Buckler ES, Yang S, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Domestication and plant breeding are ongoing 10,000-year- old
evolutionary experiments that have radically altered wild species to
meet human needs. Maize has undergone a particularly striking
transformation. Researchers have sought for decades to identify the
genes underlying maize evolution1,2, but these efforts have been limited
in scope. Here, we report a comprehensive assessment of the evolution of
modern maize based on the genome-wide resequencing of 75 wild, landrace
and improved maize lines3. We find evidence of recovery of diversity
after domestication, likely introgression from wild relatives, and
evidence for stronger selection during domestication than improvement.
We identify a number of genes with stronger signals of selection than
those previously shown to underlie major morphological changes4,5.
Finally, through transcriptome-wide analysis of gene expression, we find
evidence both consistent with removal of cis-acting variation during
maize domestication and improvement and suggestive of modern breeding
having increased dominance in expression while targeting highly
expressed genes.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/Munoz_Diez_R1-1.pdf">Using
nextgen sequencing to investigate genome size variation and transposable
element content</a><br />
Munoz Diez C, Vitte C, <strong>Ross-Ibarra J</strong>, Gaut BS,
Tenaillon MI
</summary>
<p style="margin-left: 30px">
Transposable element (TE) content explains a large part of Eukaryotic
genome size variation. TE content is determined by transposition,
removal and host responses, but the efficiency of these forces is
ultimately governed by genetic drift and natural selection. Contribution
of TE families to genome size variation has been recently quantified
using next generation sequencing (NGS) in two species pairs: Zea mays
ssp. mays and Zea luxurians, Arabidopsis lyrata and A. thaliana. In both
interspecific comparisons, genome-wide differences in TE content rather
than the proliferation of a small subset of TE families was observed. We
discuss three nonexclusive hypotheses to explain this pattern: selection
for genome shrinkage, differential efficiency of epigenetic control, and
a purely stochastic process of genome size evolution. Additional
genome-wide assessments are needed to assess the extent to which
selection shapes TE genomic content. To facilitate such studies, we
discuss the use of NGS in “orphan” species.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/vanheerwaarden2012.pdf">Historical
genomics of North American maize</a> [<a
href="http://figshare.com/articles/van_Heerwaarden_et_al_2012/757738">data</a>]<br />
<strong>van Heerwaarden J</strong>, <strong>Hufford MB</strong>,
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Since the advent of modern plant breeding in the 1930s, North American
maize has undergone a dramatic adaptation to high-in- put agriculture.
Despite the importance of genetic contributions to historical yield
increases, little is known about the underlying ge- nomic changes. Here
we use high-density SNP genotyping to char- acterize a set of North
American maize lines spanning the history of modern breeding. We provide
a unique analysis of genome- wide developments in genetic diversity,
ancestry, and selection. The genomic history of maize is marked by a
steady increase in genetic differentiation and linkage disequilibrium,
whereas allele frequencies in the total population have remained
relatively con- stant. These changes are associated with increasing
genetic sepa- ration of breeding pools and decreased diversity in the
ancestry of individual lines. We confirm that modern heterotic groups
are the product of ongoing divergence from a relatively homogeneous
landrace population, but show that differential landrace ancestry
remains evident. Using a recent association approach, we charac- terize
signals of directional selection throughout the genome, identifying a
number of candidate genes of potential agronomic relevance. However,
overall we find that selection has had limited impact on genome-wide
patterns of diversity and ancestry, with little evidence for individual
lines contributing disproportionately to the accumulation of favorable
alleles in today’s elite germ- plasm. Our data suggest breeding progress
has mainly involved selection and recombination of relatively common
alleles, contrib- uted by a representative but limited set of ancestral
lines.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/swansonwagner2012.pdf">Reshaping
of the maize transcriptome by domestication</a><br />
Swanson-Wagner R, Briskine R, Schaefer R, <strong>Hufford MB</strong>,
<strong>Ross-Ibarra J</strong>, Myers CL, Tiffin P, Springer NM
</summary>
<p style="margin-left: 30px">
Through domestication, humans have substantially altered the morphology
of Zea mays ssp. parviglumis (teosinte) into the cur- rently
recognizable maize. This system serves as a model for study- ing
adaptation, genome evolution, and the genetics and evolution of complex
traits. To examine how domestication has reshaped the transcriptome of
maize seedlings, we used expression profil- ing of 18,242 genes for 38
diverse maize genotypes and 24 teo- sinte genotypes. We detected
evidence for more than 600 genes having significantly different
expression levels in maize compared with teosinte. Moreover, more than
1,100 genes showed signifi- cantly altered coexpression profiles,
reflective of substantial rewir- ing of the transcriptome since
domestication. The genes with altered expression show a significant
enrichment for genes pre- viously identified through population genetic
analyses as likely targets of selection during maize domestication and
improvement; 46 genes previously identified as putative targets of
selection also exhibit altered expression levels and coexpression
relationships. We also identified 45 genes with altered, primarily
higher, expres- sion in inbred relative to outcrossed teosinte. These
genes are enriched for functions related to biotic stress and may
reflect responses to the effects of inbreeding. This study not only
docu- ments alterations in the maize transcriptome following domesti-
cation, identifying several genes that may have contributed to the
evolution of maize, but highlights the complementary information that
can be gained by combining gene expression with population genetic
analyses.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_86a609db506740ce96c549567c246b4c.pdf">Maize
HapMap2 identifies extant variation from a genome in flux</a><br />
Chia J-M, Song C, …[<strong>Hufford, MB</strong>, <strong>Pyhäjärvi
T</strong>, and 27 authors]… <strong>Ross-Ibarra, J</strong>, McMullen
MD, Buckler ES, Zhang G, Xu Y, Ware, D
</summary>
<p style="margin-left: 30px">
Whereas breeders have exploited diversity in maize for yield
improvements, there has been limited progress in using beneficial
alleles in undomesticated varieties. Characterizing standing variation
in this complex genome has been challenging, with only a small fraction
of it described to date. Using a population genetics scoring model, we
identified 55 million SNPs in 103 lines across pre-domestication and
domesticated Zea mays varieties, including a representative from the
sister genus Tripsacum. We find that structural variations are pervasive
in the Z. mays genome and are enriched at loci associated with important
traits. By investigating the drivers of genome size variation, we find
that the larger Tripsacum genome can be explained by transposable
element abundance rather than an allopolyploid origin. In contrast,
intraspecies genome size variation seems to be controlled by chromosomal
knob content. There is tremendous overlap in key gene content in maize
and Tripsacum, suggesting that adaptations from Tripsacum (for example,
perennialism and frost and drought tolerance) can likely be integrated
into maize.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.genetics.org/content/191/3/883.abstract">Megabase-scale
inversion polymorphism in the wild ancestor of maize</a> [<a
href="http://figshare.com/articles/Fang_et_al_2012/840519">data</a>]<br />
Fang Z, <strong>Pyhäjärvi T</strong>, Weber AL, Dawe RK, …[2 authors]…
<strong>Ross-Ibarra C</strong>, Doebley J, Morrell PL,
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We identify and study the evolutionary genetics of a 50Mb inversion in
natural populations of teosinte
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_5ca9c5a7f6174a5bb0c85e67dec67dcc.pdf">Genetic
architecture of maize kernel composition in the Nested Association
Mapping and Inbred Association panels</a><br />
Cook JP, McMullen MD, Holland JB, Tian F, Bradbury P,
<strong>Ross-Ibarra J</strong>, Buckler ES, Flint-Garcia SA
</summary>
<p style="margin-left: 30px">
The maize (Zea mays) kernel plays a critical role in feeding humans and
livestock around the world and in a wide array of industrial
applications. An understanding of the regulation of kernel starch,
protein, and oil is needed in order to manipulate composition to meet
future needs. We conducted joint-linkage quantitative trait locus
mapping and genome-wide association studies (GWAS) for kernel starch,
protein, and oil in the maize nested association mapping population,
composed of 25 recombinant inbred line families derived from diverse
inbred lines. Joint-linkage mapping revealed that the genetic
architecture of kernel composition traits is controlled by 21–26
quantitative trait loci. Numerous GWAS associations were detected,
including several oil and starch associations in acyl-CoA:diacylglycerol
acyltransferase1-2, a gene that regulates oil composition and quantity.
Results from nested association mapping were verified in a 282 inbred
association panel using both GWAS and candidate gene association
approaches. We identified many beneficial alleles that will be useful
for improving kernel starch, protein, and oil content.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_a2a75f7435ec4c569d4a6996e8b5451e.pdf">Crop
genomics: advances and applications</a><br />
Morrell PL, Buckler ES, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
The completion of reference genome sequences for many important crops
and the ability to perform high-throughput resequencing are providing
opportunities for improving our understanding of the history of plant
domestication and to accelerate crop improvement. Crop plant comparative
genomics is being transformed by these data and a new generation of
experimental and computational approaches. The future of crop
improvement will be centred on comparisons of individual plant genomes,
and some of the best opportunities may lie in using combinations of new
genetic mapping strategies and evolutionary analyses to direct and
optimize the discovery and use of genetic variation. Here we review such
strategies and insights that are emerging.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Tanja Pyhäjärvi</strong></p>
<pre><code>Krutovsky, Konstantin V., et al. &quot;Gene flow, spatial structure, local adaptation, and assisted migration in trees.&quot; Genomics of tree crops. Springer, New York, NY, 2012. 71-116.

Pyhäjärvi, Tanja, Esa Aalto, and Outi Savolainen. &quot;Time scales of divergence and speciation among natural populations and subspecies of Arabidopsis lyrata (Brassicaceae).&quot; American journal of botany 99.8 (2012): 1314-1322.</code></pre>
<p><strong>Matthew Hufford</strong></p>
<pre><code>Hufford, Matthew B., et al. &quot;Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight.&quot; PLoS One 7.11 (2012): e47659.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-11" class="section level3">
<h3>2011</h3>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_14630f30e97f4e04b8b5980632002f7a.pdf">Identification
of a functional transposon insertion in the maize domestication gene
<em>tb1</em></a> [<a
href="http://www.panzea.org/db/gateway?file_id=Studer_etal_2011_Nat_Genet_data">data</a>]<br />
Studer A, Zhao Q, <strong>Ross-Ibarra J</strong>, Doebley J
</summary>
<p style="margin-left: 30px">
Genetic diversity created by transposable elements is an important
source of functional variation upon which selection acts during
evolution1–6. Transposable elements are associated with adaptation to
temperate climates in Drosophila7, a SINE element is associated with the
domestication of small dog breeds from the gray wolf8 and there is
evidence that transposable elements were targets of selection during
human evolution9. Although the list of examples of transposable elements
associated with host gene function continues to grow, proof that
transposable elements are causative and not just correlated with
functional variation is limited. Here we show that a transposable
element (Hopscotch) inserted in a regulatory region of the maize
domestication gene, teosinte branched1 (tb1), acts as an enhancer of
gene expression and partially explains the increased apical dominance in
maize compared to its progenitor, teosinte. Molecular dating indicates
that the Hopscotch insertion predates maize domestication by at least
10,000 years, indicating that selection acted on standing variation
rather than new mutation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_4916e7f7acec4aee869da040d519e0be.pdf">Genetic
signals of origin, spread and introgression in a large sample of maize
landraces</a> [<a
href="http://figshare.com/articles/new_fileset/757735">data</a>]<br />
<strong>van Heerwaarden J</strong>, Doebley J, Briggs WH, Glaubitz JC,
Goodman MM, Sanchez Gonzalez JJ, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We resolve the conflict between genetic, ecological, and achaeological
data regarding the geographic location of maize domestication.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/hufford2011.pdf">Influence of
cryptic population structure on observed mating patterns in the wild
progenitor of maize</a> [<a
href="10.6084/m9.figshare.757734">data</a>]<br />
<strong>Hufford MB</strong>, Gepts P, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Indirect two-generation analysis of pollen flow has proven to be an
effective alternative to exhaustive paternity analysis in numerous plant
populations. In this investigation, the method is extended to an annual
wild maize species, Zea mays ssp. parviglumis (Poaceae). Our analysis of
mating system in parviglumis revealed high levels of outcrossing and
higher biparental inbreeding than typically observed in grasses. Pollen
dispersal analysis suggested low levels of long-distance dispersal.
Given previous evidence for intrapopulation genetic structure in
parviglumis populations, we explored the impact of cryptic population
structure on estimates of mating system and pollen flow. Subpopulations
inferred through spatially explicit Bayesian assignment showed mark-
edly different values for both mating system parameters and pollen flow
than the entire population. Finally, a novel method of pollen haplotype
assignment revealed nonrandom mating consistent with intrapopulation
structure. These results indicate parviglumis could be particularly
susceptible to habitat fragmentation currently occurring throughout
Mexico due to recent changes in land use.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://www.ncbi.nlm.nih.gov/pubmed/21296765">Genome size and
TE content as determined by high-throughput sequencing in maize and
<em>Zea luxurians</em></a><br />
Tenaillon MI, <strong>Hufford MB</strong>, Gaut BS, <strong>Ross-Ibarra
J</strong>
</summary>
<p style="margin-left: 30px">
The genome of maize (Zea mays ssp. mays) consists mostly of transposable
elements (TEs) and varies in size among lines. This variation extends to
other species in the genus Zea: although maize and Zea luxurians
diverged only ∼140,000 years ago, their genomes differ in size by ∼50%.
We used paired-end Illumina sequencing to evaluate the potential
contribution of TEs to the genome size difference between these two
species. We aligned the reads both to a filtered gene set and to an
exemplar database of unique repeats representing 1,514 TE families; ∼85%
of reads mapped against TE repeats in both species. The relative
contribution of TE families to the B73 genome was highly correlated with
previous estimates, suggesting that reliable estimates of TE content can
be obtained from short high-throughput sequencing reads, even at low
coverage. Because we used paired-end reads, we could assess whether a TE
was near a gene by determining if one paired read mapped to a TE and the
second read mapped to a gene. Using this method, Class 2 DNA elements
were found significantly more often in genic regions than Class 1 RNA
elements, but Class 1 elements were found more often near other TEs.
Overall, we found that both Class 1 and 2 TE families account for ∼70%
of the genome size difference between B73 and luxurians. Interestingly,
the relative abundance of TE families was conserved between species (r =
0.97), suggesting genome-wide control of TE content rather than
family-specific effects.
</p>
</details>
<p><br></p>
<details>
<summary>
<p><strong>Other papers by lab members</strong></p>
</summary>
<p style="margin-left: 30px">
<p><strong>Tanja Pyhäjärvi</strong></p>
<pre><code>Savolainen, Outi, et al. &quot;Adaptive potential of northernmost tree populations to climate change, with emphasis on Scots pine (Pinus sylvestris L.).&quot; Journal of Heredity 102.5 (2011): 526-536.

Pyhäjärvi, Tanja, Sonja T. Kujala, and Outi Savolainen. &quot;Revisiting protein heterozygosity in plants—nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris.&quot; Tree genetics &amp; genomes 7.2 (2011): 385-397.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-12" class="section level3">
<h3>2010</h3>
<details>
<summary>
<a href="http://www.genetics.org/content/185/3/969.abstract">Patterns of
population structure and environmental associations to aridity across
the range of loblolly pine (<em>Pinus taeda</em>)</a><br />
Eckert AJ, <strong>van Heerwaarden J</strong>, Wegrzyn JL, Nelson CD,
<strong>Ross-Ibarra J</strong>, Gonzalez-Martinez SC, and Neale DB
</summary>
<p style="margin-left: 30px">
Natural populations of forest trees exhibit striking phenotypic
adaptations to diverse environmental gradients, thereby making them
appealing subjects for the study of genes underlying ecologically
relevant phenotypes. Here, we use a genome-wide data set of single
nucleotide polymorphisms genotyped across 3059 functional genes to study
patterns of population structure and identify loci associated with
aridity across the natural range of loblolly pine (Pinus taeda L.).
Overall patterns of population structure, as inferred using principal
components and Bayesian cluster analyses, were consistent with three
genetic clusters likely resulting from expansions out of Pleistocene
refugia located in Mexico and Florida. A novel application of
association analysis, which removes the confounding effects of shared
ancestry on correlations between genetic and environmental variation,
identified five loci correlated with aridity. These loci were primarily
involved with abiotic stress response to temperature and drought. A
unique set of 24 loci was identified as FST outliers on the basis of the
genetic clusters identified previously and after accounting for
expansions out of Pleistocene refugia. These loci were involved with a
diversity of physiological processes. Identification of nonoverlapping
sets of loci highlights the fundamental differences implicit in the use
of either method and suggests a pluralistic, yet complementary, approach
to the identification of genes underlying ecologically relevant
phenotypes.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Fuchs-et-al-2010.pdf">Reproductive
biology of <strong>Macleania rupestris</strong> (Ericaceae): a
pollen-limited Neotropical cloud-forest species in Costa Rica</a><br />
Fuchs EJ, <strong>Ross-Ibarra J</strong>, Barrantes G
</summary>
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/Whitney-et-al-2010-402.pdf">A
role for nonadaptive processes in plant genome size evolution?</a><br />
Whitney KD, Baack EJ, Hamrick JL, Godt, MJW …[3 authors]… Goodwillie C,
Kalisz S, Leitch I, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Genome sizes vary widely among species, but comprehensive explanations
for the emergence of this variation have not been validated. Lynch and
Conery (2003) hypothesized that genome expansion is maladaptive, and
that lineages with small effective population size (Ne) evolve larger
genomes than those with large Ne as a consequence of the lowered
efficacy of natural selection in small populations. In addition, mating
systems likely affect genome size evolution via effects on both Ne and
the spread of transposable elements (TEs). We present a comparative
analysis of the effects of Ne and mating system on genome size evolution
in seed plants. The dataset includes 205 species with monoploid genome
size estimates (corrected for recent polyploidy) ranging from 2Cx = 0.3
to 65.9 pg. The raw data exhibited a strong positive relationship
between outcrossing and genome size, a negative relationship between Ne
and genome size, but no detectable Ne × outcrossing interaction. In
contrast, phylogenetically independent contrast analyses found only a
weak relationship between outcrossing and genome size and no
relationship between Ne and genome size. Thus, seed plants do not
support the Lynch and Conery mechanism of genome size evolution. Further
work is needed to disentangle contrasting effects of mating systems on
the efficacy of selection and TE transmission.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Van-Heerwaarden-et-al-2010-951.pdf">Fine
scale genetic structure in the wild ancestor of maize (<strong>Zea
mays</strong> ssp. <strong>parviglumis</strong>)</a><br />
<strong>van Heerwaarden J</strong>, <strong>Ross-Ibarra J</strong>,
Doebley J, Glaubitz JC, Sanchez Gonzalez J, Gaut BS, Eguiarte LE
</summary>
<p style="margin-left: 30px">
Analysis of fine scale genetic structure in continuous populations of
outcrossing plant species has traditionally been limited by the
availability of sufficient markers. We used a set of 468 SNPs to
characterize fine-scale genetic structure within and between two dense
stands of the wild ancestor of maize, teosinte (Zea mays ssp.
parviglumis). Our analyses confirmed that teosinte is highly outcrossing
and showed little population structure over short distances. We found
that the two populations were clearly genetically differen- tiated,
although the actual level of differentiation was low. Spatial
autocorrelation of relatedness was observed within both sites but was
somewhat stronger in one of the populations. Using principal component
analysis, we found evidence for significant local differentiation in the
population with stronger spatial autocorrelation. This differentiation
was associated with pronounced shifts in the first two principal
components along the field. These shifts corresponded to changes in
allele frequencies, potentially due to local topographical features.
There was little evidence for selection at individual loci as a
contributing factor to differentiation. Our results demonstrate that
significant local differentiation may, but need not, co-occur with
spatial autocorrelation of relatedness. The present study represents one
of the most detailed analyses of local genetic structure to date and
provides a benchmark for future studies dealing with fine scale patterns
of genetic diversity in natural plant populations.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000327">Widespread
gene conversion in centromere cores</a><br />
Shi J, Wolf S, Burke J, Presting G, <strong>Ross-Ibarra J</strong>, Dawe
RK
</summary>
<p style="margin-left: 30px">
Centromeres are the most dynamic regions of the genome, yet they are
typified by little or no crossing over, making it difficult to explain
the origin of this diversity. To address this question, we developed a
novel CENH3 ChIP display method that maps kinetochore footprints over
transposon-rich areas of centromere cores. A high level of polymorphism
made it possible to map a total of 238 within-centromere markers using
maize recombinant inbred lines. Over half of the markers were shown to
interact directly with kinetochores (CENH3) by chromatin
immunoprecipitation. Although classical crossing over is fully
suppressed across CENH3 domains, two gene conversion events (i.e.,
non-crossover marker exchanges) were identified in a mapping population.
A population genetic analysis of 53 diverse inbreds suggests that
historical gene conversion is widespread in maize centromeres, occurring
at a rate &gt;1×10−5/marker/generation. We conclude that gene conversion
accelerates centromere evolution by facilitating sequence exchange among
chromosomes. [<a
href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000326">plos
biology commentary</a>]
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://mbe.oxfordjournals.org/content/27/2/409.abstract">Indel-associated
mutation rate varies with mating system in flowering plants</a><br />
Hollister JD, <strong>Ross-Ibarra J</strong>, Gaut BS
</summary>
<p style="margin-left: 30px">
We show that mating system differences in heterozygosity mitigate the
effects of insertion and deletion polymorphisms on nucleotide diversity.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://pubmed.ncbi.nlm.nih.gov/19738636/">Genetic diversity in
a crop metapopulation</a><br />
<strong>van Heerwaarden J</strong>, van Eeuwijk FA, <strong>Ross-Ibarra
J</strong>
</summary>
<p style="margin-left: 30px">
The need to protect crop genetic resources has sparked a growing
interest in the genetic diversity maintained in traditional farming
systems worldwide. Although traditional seed management has been
proposed as an important determinant of genetic diversity and structure
in crops, no models exist that can adequately describe the genetic
effects of seed management. We present a metapopulation model that
accounts for several features unique to managed crop populations. Using
traditional maize agriculture as an example, we develop a
coalescence-based model of a crop metapopulation undergoing pollen and
seed flow as well as seed replacement. In contrast to metapopulation
work on natural systems, we model seed migration as episodic and
originating from a single source per population rather than as a
constant immigration from the entire metapopulation. We find that the
correlated origin of migrants leads to surprising results, including a
loss of invariance of within-deme diversity and a parabolic relationship
between F(ST) and migration quantity. In contrast, the effects of
migration frequency on diversity and structure are more similar to
classical predictions, suggesting that seed migration in managed crop
populations cannot be described by a single parameter. In addition to
migration, we investigate the effects of deme size and extinction rates
on genetic structure, and show that high levels of pollen migration may
mask the effects of seed management on structure. Our results highlight
the importance of analytically evaluating the effects of deviations from
classical metapopulation models, especially in systems for which data
are available to estimate specific model parameters.
</p>
</details>
<p><br></p>
<details>
<summary>
<strong>Other papers by lab members</strong>
</summary>
<p style="margin-left: 30px">
<p><strong>Matthew Hufford</strong></p>
<pre><code>Epanchin-Niell, Rebecca S., et al. &quot;Controlling invasive species in complex social landscapes.&quot; Frontiers in Ecology and the Environment 8.4 (2010): 210-216.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-13" class="section level3">
<h3>2009</h3>
<details>
<summary>
<a
href="http://media.wix.com/ugd/fe9228_352f4ee9b69a425589e55a4bac60bc5d.pdf">A
first-generation haplotype map of maize</a><br />
Gore MA, Chia JM, Elshire RJ, Sun Q …[4 authors]… Grills GS,
<strong>Ross-Ibarra J</strong>, Ware DH, Buckler ES
</summary>
<pre><code> &lt;p style=&quot;margin-left: 30px&quot;&gt;</code></pre>
Maize is an important crop species of high genetic diversity. We
identified and genotyped several million sequence polymorphisms among 27
diverse maize inbred lines and discovered that the genome was
characterized by highly divergent haplotypes and showed 10- to 30-fold
variation in recombination rates. Most chromosomes have pericentromeric
regions with highly suppressed recombination that appear to have
influenced the effectiveness of selection during maize inbred
development and may be a major component of heterosis. We found hundreds
of selective sweeps and highly differentiated regions that probably
contain loci that are key to geographic adaptation. This survey of
genetic diversity provides a foundation for uniting breeding efforts
across the world and for dissecting complex traits through genome-wide
association studies.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008346">A
pleistocene clone of Palmer’s Oak persisting in Southern
California</a><br />
<strong>May MR</strong>, <strong>Provance MC</strong>, Sanders AC,
Ellstrand NC, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
We discover a 13,000 year old oak clone in the hills of Southern
California
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.02984.x/epdf">Selection
on grain shattering genes and rates of rice domestication</a><br />
Zhang LB, Zhu Q, Wu ZQ, <strong>Ross-Ibarra J</strong>, Gaut BS, Ge S,
Sang T
</summary>
<p style="margin-left: 30px">
Molecular cloning of major quantitative trait loci (QTLs) responsible
for the reduc-tion of rice grain shattering, a hallmark of cereal
domestication, provided opportunitiesfor in-depth investigation of
domestication processes.• Here, we studied nucleotide variation at the
shattering loci, sh4 and qSH1, forcultivated rice, Oryza sativa ssp.
indica and Oryza sativa ssp. japonica, and the wildprogenitors, Oryza
nivara andOryza rufipogon.• The nonshattering sh4 allele was fixed in
all rice cultivars, with levels of sequencepolymorphism significantly
reduced in both indica and japonica cultivars relative tothe wild
progenitors. The sh4 phylogeny together with the neutrality tests and
coales-cent simulations suggested that sh4 had a single origin and was
fixed by artificialselection during the domestication of rice. Selection
on qSH1 was not detected inindica and remained unclear in japonica.•
Selection on sh4 could be strong enough to have driven its fixation in a
populationof cultivated rice within a period of c. 100 yr. The slow
fixation of the nonshatteringphenotype observed at the archeological
sites might be a result of relatively weakselection on mutations other
than sh4 in early rice cultivation. The fixation of sh4could have been
achieved later through strong selection for the optimal phenotype
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://www.genetics.org/content/181/4/1399.abstract">Historical
divergence and gene flow in the genus Zea</a><br />
<strong>Ross-Ibarra J</strong>, Tenaillon M, Gaut BS
</summary>
<p style="margin-left: 30px">
Gene flow plays a fundamental role in plant evolutionary history, yet
its role in population divergence—and ultimately speciation—remains
poorly understood. We investigated gene flow and the modalities of
divergence in the domesticate Zea mays ssp. mays and three wild Zea taxa
using sequence polymorphism data from 26 nuclear loci. We described
diversity across loci and assessed evidence for adaptive and purifying
selection at nonsynonymous sites. For each of three divergence events in
the history of these taxa, we used approximate Bayesian simulation to
estimate population sizes and divergence times and explicitly compare
among alternative models of divergence. Our estimates of divergence
times are surprisingly consistent with previous data from other markers
and suggest rapid diversification of lineages within Zea in the last
∼150,000 years. We found widespread evidence of historical gene flow,
including evidence for divergence in the face of gene flow. We speculate
that cultivated maize may serve as a bridge for gene flow among
otherwise allopatric wild taxa.
</p>
</details>
<p><br></p>
</div>
<div id="section-14" class="section level3">
<h3>2008</h3>
<details>
<summary>
<a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002411">Patterns of
polymorphism and demographic history in natural populations of
<em>Arabidopsis lyrata</em></a><br />
<strong>Ross-Ibarra J</strong>, Wright SI, Foxe JP, Kawabe A,
DeRose-Wilson L, Gos G, Charlesworth D, Gaut BS
</summary>
<p style="margin-left: 30px">
<p>Background</p>
<p>Many of the processes affecting genetic diversity act on local
populations. However, studies of plant nucleotide diversity have largely
ignored local sampling, making it difficult to infer the demographic
history of populations and to assess the importance of local adaptation.
Arabidopsis lyrata, a self-incompatible, perennial species with a
circumpolar distribution, is an excellent model system in which to study
the roles of demographic history and local adaptation in patterning
genetic variation.</p>
<p>Principal Findings</p>
<p>We studied nucleotide diversity in six natural populations of
Arabidopsis lyrata, using 77 loci sampled from 140 chromosomes. The six
populations were highly differentiated, with a median FST of 0.52, and
structure analysis revealed no evidence of admixed individuals. Average
within-population diversity varied among populations, with the highest
diversity found in a German population; this population harbors 3-fold
higher levels of silent diversity than worldwide samples of A. thaliana.
All A. lyrata populations also yielded positive values of Tajima’s D. We
estimated a demographic model for these populations, finding evidence of
population divergence over the past 19,000 to 47,000 years involving
non-equilibrium demographic events that reduced the effective size of
most populations. Finally, we used the inferred demographic model to
perform an initial test for local adaptation and identified several
genes, including the flowering time gene FCA and a disease resistance
locus, as candidates for local adaptation events.</p>
<p>Conclusions</p>
Our results underscore the importance of population-specific,
non-equilibrium demographic processes in patterning diversity within A.
lyrata. Moreover, our extensive dataset provides an important resource
for future molecular population genetic studies of local adaptation in
A. lyrata.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://www.pnas.org/content/105/37/13965.short">Demography and
weak selection drive patterns of TE diversity in natural populations of
<em>A. lyrata</em></a><br />
Lockton S, <strong>Ross-Ibarra J</strong>, Gaut BS
</summary>
<p style="margin-left: 30px">
We show that differences in demographic history among populations of
have changed the abundance and freqeuncy of transposable elements in the
<em>A lyrata</em> genome.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614810/">Multiple
domestications do not appear monophyletic</a><br />
<strong>Ross-Ibarra J</strong>, Gaut BS
</summary>
<p style="margin-left: 30px">
A rebuttal showing that genetic data can indeed successfully recover
multiple independent domestications.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra_Gaut_2008.pdf">Selection
on major components of angiosperm genomes</a><br />
Gaut BS, <strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Discusses some of the evolutionary forces responsible for shaping genome
size across plants.
</p>
</details>
<p><br></p>
</div>
<div id="section-15" class="section level3">
<h3>2007</h3>
<details>
<summary>
<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1876441/">Plant
domestication, a unique opportunity to identify the genetic basis of
adaptation</a><br />
<strong>Ross-Ibarra J</strong>, Morrell PL, Gaut BS
</summary>
<p style="margin-left: 30px">
Presents plant domestication as an ideal system in which to study the
genetics of adaptation.
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2006.01275.x/abstract">Genome
size and recombination in angiosperms: a second look</a><br />
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Despite dramatic differences in genome size – and thus space for
recombination to occur – previous workers found no correlation between
recombination rate and genome size in flowering plants. Here I
re-investigate these claims using phylogenetic comparative methods to
test a large data set of recombination data in angiosperms. I show that
genome size is significantly correlated with recombination rate across a
wide sampling of species and that change in genome size explains a
meaningful proportion (∼20%) of variation in recombination rate. I show
that the strength of this correlation is comparable with that of several
characters previously linked to evolutionary change in recombination
rate, but argue that consideration of processes of genome size change
likely make the observed correlation a conservative estimate. And
finally, although I find that recombination rate increases less than
proportionally to change in genome size, several mechanistic and
theoretical arguments suggest that this result is not unexpected.
</p>
</details>
<p><br></p>
</div>
<div id="section-16" class="section level3">
<h3>2001-2006</h3>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Wares-et-al-2006.pdf">Mitochondrial
DNA and population size</a><br />
Wares JP, Barber PH, <strong>Ross-Ibarra J</strong>, Sotka EE, Toonen RJ
</summary>
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2005-15881692.pdf">Quantitative
trait loci and the study of plant domestication</a><br />
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Plant domestication ranks as one of the most important developments in
human history, giving human populations the potential to harness
unprecedented quantities of the earth’s resources. But domestication has
also played a more subtle historical role as the foundation of the
modern study of evolution and adaptation. Until recently, however,
researchers interested in domestication were limited to studying
phenotypic changes or the genetics of simple Mendelian traits, when
often the characters of most interest – fruit size, yield, height,
flowering time, etc. – are quantitative in nature. The goals of this
paper are to review some of the recent work on the quantitative genetics
of plant domestication, identify some of the common trends found in this
literature, and offer some novel interpretations of the data that is
currently available.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2004.pdf">The
evolution of recombination under domestication: a test of two
hypotheses</a> [<a
href="http://rilab.ucdavis.edu/pubs/supp_data_rossibarra2004.tgz">data</a>]<br />
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
Here I test two different hypotheses about the how recombination should
relate to domestication: Does domestication itself select for increased
recombination (yes), or respecies with higher recombination rates more
likely to be domesticated (no).
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2003.pdf">Origen y
domesticación de la chaya (<em>Cnidoscolus aconitifolius</em> Mill IM
Johnst): La espinaca Maya</a><br />
<strong>Ross-Ibarra J</strong>
</summary>
<p style="margin-left: 30px">
I use genetic data to study the origin of a leafy shrub domesticated by
the Maya
</p>
</details>
<p><br></p>
<details>
<summary>
<a
href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra_Molina-Cruz-2002.pdf">The
ethnobotany of Chaya (<em>Cnidoscolus aconitifolius</em> ssp.
<em>aconitifolius</em> Breckon): A nutritious Maya vegetable</a><br />
<strong>Ross-Ibarra J</strong>, Molina-Cruz A
</summary>
<p style="margin-left: 30px">
Chaya (Cnidoscolusaconitifolius ssp. aconitifolius Breckon) is a
domesticatedleafy green vegetable of the Maya region of
Guatemala,Belize, southeast Mexico and the YucatdnPeninsula, and parts
of Honduras. Thoughrelativelyunknownoutsideof thisarea,
evidencesuggeststhatchayawas of significant importanceto ancientpeoples
of the Yucata’nPeninsulaandperhapselsewherewithintheMaya region.Herewe
reviewwhatlittleresearchhas beendoneon thisimpressiveplant,as wellas
recountour own ethnobotanicalinvestigationinto its use as a food plant
and medicine,and discuss its botany,nomenclature,and agriculturaluse.
Due to its ease of cultivation,potential productivity,and above all its
substantialnutritionalvalue, we propose chaya as a potential cropfor
areas outsideMesoamerica.
</p>
</details>
<p><br></p>
<details>
<summary>
<a href="https://pubmed.ncbi.nlm.nih.gov/11454621/">Implications of
mating patterns for conservation of the endangered plant <em>Eriogonum
ovalifolium</em> var. <em>vineum</em></a><br />
Neel MC, <strong>Ross-Ibarra J</strong>, Ellstrand NC
</summary>
<p style="margin-left: 30px">
Mating patterns have direct application to conservation because of their
influence on structuring genetic diversity within and among populations
and on maintaining that diversity over time. We measured population and
family outcrossing rates, biparental inbreeding, correlation of
outcrossed paternity, and inbreeding coefficients in six populations
from throughout the ecological range of the endangered plant Eriogonum
ovalifolium var. vineum using naturally pollinated families. The taxon
was primarily outcrossed: population outcrossing rates averaged 0.80 (SE
0.03) and family outcrossing rates averaged 0.88 (SE 0.03); neither rate
varied among populations. Five population rates were significantly
different from 1 while family rates differed from 1 in only one
population. We found high correlated outcrossed paternity and evidence
for biparental inbreeding in five populations each. As expected from the
predominantly outcrossed mating system, levels of diversity were high
and inbreeding coefficients among maternal individuals were low
(averaging -0.05, SE 0.12). Differences between inbreeding coefficients
of progeny (average 0.21, SE 0.06) and mothers indicated selection
against homozygous offspring. These results indicate that it is
important to maintain large populations to prevent increases in
inbreeding and to maintain pollinator communities to facilitate
outcrossing.
</p>
</details>
<p><br></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
