<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Publications</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<script>
  window.onload = choosePic;
  var myPix = new Array("images/image1.jpg","images/image2.jpg","images/image3.jpg","images/image4.jpg","images/image5.jpg","images/image6.jpg","images/image7.jpg","images/image8.jpg","images/image8.jpg","images/image10.jpg");
  function choosePic() {
     var randomNum = Math.floor(Math.random() * myPix.length);
     document.getElementById("myPicture").src = myPix[randomNum];
  }
</script>

<div class="navbar navbar-default  navbar-float-top" role="navigation">
    <div class="container" overflow="auto" align="left" style="width: inherit;">
      <img id="myPicture" alt="Brand" src="" align='center'>
    </div>

  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>

      </button>

      <a class="navbar-brand" href="index.html"></a>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav" align="center">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs/wiki">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
        <li><a href="press.html">Press</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Publications</h1>

</div>


<title>
<span class="math inline">\(if(title-prefix)\)</span><span class="math inline">\(title-prefix\)</span> - <span class="math inline">\(endif\)</span><span class="math inline">\(pagetitle\)</span>
</title>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<p><a href="http://scholar.google.com/citations?user=5SzRq1oAAAAJ"><img src="images/scholar.png" style="width: 100px;"></a></p>
<SCRIPT language="JavaScript" SRC='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></SCRIPT>
<p>Journal names have been intentionally excluded. Lab member names are in bold. Links are provided to the published article and when possible an open-access version (pre- or post-print) of the paper. Click on a triangle for the article abstract.</p>
<p><br></p>
<div id="preprints" class="section level3">
<h3>Preprints</h3>
<br> <br>
<details>
<summary> Domestication reshaped the genetic basis of inbreeding depression in a maize landrace compared to its wild relative, teosinte [<a href="https://www.biorxiv.org/content/10.1101/2021.09.01.458502v1">preprint</a>]<br />
Holland JB, Samayoa LF, Olukolu BA, Yang CJ, …[7 authors]… <strong>Yang J</strong>, <strong>Ross-Ibarra J</strong>, Buckler ES, Doebley JF. </summary>
<p style="margin-left: 30px">
Inbreeding depression is the reduction in fitness and vigor resulting from mating of close relatives observed in many plant and animal species. The extent to which the genetic load of mutations contributing to inbreeding depression is due to rare large-effect variation versus potentially more common variants with very small individual effects is unknown and may be affected by population history. We compared the effects of outcrossing and self-fertilization on 18 traits in a landrace population of maize, which underwent a population bottleneck during domestication, and a neighboring population of its wild relative teosinte. Inbreeding depression was greater in maize than teosinte for 15 of 18 traits, congruent with the greater segregating genetic load predicted from sequence data in the maize population. For many traits - and more commonly in maize - genetic variation among self-fertilized families was less than expected based on additive and dominance variance estimated in outcrossed families, suggesting that a negative covariance between additive and homozygous dominance effects limits the variation available to selection under partial inbreeding. We identified quantitative trait loci (QTL) representing large-effect rare variants carried by only a single parent, which were more important in teosinte than maize. Teosinte also carried more putative juvenile-acting lethal variants identified by segregation distortion. These results suggest a mixture of mostly polygenic, small-effect recessive variation underlying inbreeding depression, with an additional contribution from rare larger-effect variants that was more important in teosinte but depleted in maize following to the domestication bottleneck. Purging associated with the maize domestication bottleneck may have selected against large effect variants, but polygenic load is harder to purge and segregating mutational burden increased in maize compared to teosinte.
</p>
</details>
<br>
<details>
<summary> Controlling for variable transposition rate with an age-adjusted site frequency spectrum [<a href="https://www.biorxiv.org/content/10.1101/2021.08.16.456262v1">preprint</a>]<br />
<strong>Horvath R</strong>, <strong>Menon M</strong>, Stitzer MC, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
Recognition of the important role of transposable elements (TEs) in eukaryotic genomes quickly led to a burgeoning literature modeling and estimating the effects of selection on TEs. Much of the empirical work on selection has focused on analyzing the site frequency spectrum (SFS) of TEs. But TEs differ from standard evolutionary models in a number of ways that can impact the power and interpretation of the SFS. For example, rather than mutating under a clock-like model, transposition often occurs in bursts which can inflate particular frequency categories compared to expectations under a standard neutral model. If a TE burst has been recent, the excess of low frequency polymorphisms can mimic the effect of purifying selection. Here, we investigate how transposition bursts affect the frequency distribution of TEs and the correlation between age and allele frequency. Using information on the TE age distribution, we propose an age-adjusted site frequency spectrum to compare TEs and neutral polymorphisms to more effectively evaluate whether TEs are under selective constraints. We show that our approach can minimize instances of false inference of selective constraint, but also allows for a correct identification of even weak selection affecting TEs which experienced a transposition burst and is robust to at least simple demographic changes. The results presented here will help researchers working on TEs to more reliably identify the effects of selection on TEs without having to rely on the assumption of a constant transposition rate.
</p>
</details>
<br>
<details>
<summary> Analysis of genotype by environment interactions in a maize mapping population [<a href="https://www.biorxiv.org/content/10.1101/2021.07.21.453280v1">preprint</a>]<br />
<strong>Hudson AI</strong>, <strong>Odell SG</strong>, Dubreuil P, Tixier M-H, Praud S, Runcie DE, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
Genotype by environment interactions are a significant challenge for crop breeding as well as being important for understanding the genetic basis of environmental adaptation. In this study, we analyzed genotype by environment interaction in a maize multi-parent advanced generation intercross population grown across five environments. We found that genotype by environment interactions contributed as much as genotypic effects to the variation in some agronomically important traits. In order to understand how genetic correlations between traits change across environments, we estimated the genetic variance-covariance matrix in each environment. Changes in genetic covariances between traits across environments were common, even among traits that show low genotype by environment variance. We also performed a genome-wide association study to identify markers associated with genotype by environment interactions but found only a small number of significantly associated markers, possibly due to the highly polygenic nature of genotype by environment interactions in this population.
</p>
</details>
<br>
<details>
<summary> Modeling Allelic Diversity of Multi-parent Mapping Populations Affects Detection of Quantitative Trait Loci [<a href="https://www.biorxiv.org/content/10.1101/2021.07.14.452335v1">preprint</a>]<br />
<strong>Odell SG</strong>, <strong>Hudson AI</strong>, Praud S, Dubreuil P, Tixier M-H, <strong>Ross-Ibarra J</strong>, Runcie DE </summary>
<p style="margin-left: 30px">
The search for quantitative trait loci (QTL) that explain complex traits such as yield and flowering time has been ongoing in all crops. Methods such as bi-parental QTL mapping and genome-wide association studies (GWAS) each have their own advantages and limitations. Multi-parent advanced generation intercross (MAGIC) populations contain more recombination events and genetic diversity than bi-parental mapping populations and reduce the confounding effect of population structure that is an issue in association mapping populations. Here we discuss the results of using a MAGIC population of doubled haploid (DH) maize lines created from 16 diverse founders to perform QTL mapping. We compare three models that assume bi-allelic, founder, and ancestral haplotype allelic states for QTL. The three methods have different power to detect QTL for a variety of agronomic traits. Although the founder approach finds the most QTL, there are also QTL unique to each method, suggesting that each model has advantages for traits with different genetic architectures. A closer look at a well-characterized flowering time QTL, qDTA8, which contains vgt1, suggests a potential epistatic interaction and highlights the strengths and weaknesses of each method. Overall, our results reinforce the importance of considering different approaches to analyzing genotypic datasets, and show the limitations of binary SNP data for identifying multi-allelic QTL.
</p>
</details>
<br>
<details>
<summary> Strengthened mutualistic adaptation between teosinte and its rhizosphere biota in cold climates [<a href="https://www.biorxiv.org/content/10.1101/2021.04.20.440703v1">preprint</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Gasca-Pineda J, Baxter, I, Eguiarte LE, <strong>Ross-Ibarra J</strong>, Strauss SY </summary>
<p style="margin-left: 30px">
<p>While abiotic environments consistently shape local adaptation, the strength of local adaptation to biotic interactions may vary more. One theory, COCO (CO-evolutionary Outcomes across Conditionality), predicts it may be strongest where species experience greater stress, because stress increases fitness impacts of species interactions. For example, in plant interactions with rhizosphere biota, positive outcomes increase with stress from low soil fertility, drought and cold.</p>
<p>To investigate the influence of abiotic stress gradients on adaptation between plants and rhizosphere biota, we used a greenhouse common garden experiment recombining teosinte, Zea mays ssp. mexicana (wild relative of maize), and rhizosphere biota, collected across a stress gradient (elevational variation in temperature, precipitation, and nutrients).</p>
<p>We found stronger local adaptation between teosinte and rhizosphere biota from colder, more stressful sites, as expected by COCO. However, biota from less stressful, warmer sites provided greater average benefits across teosinte populations. Links between plant traits and 20-element profiles of plant leaves explained fitness variation, persisted in the field, were influenced by both plants and biota, and largely reflected patterns of local adaptation.</p>
In sum, we uncovered greater local adaptation to biotic interactions in colder sites, and that both plants and rhizosphere biota affect the expression of plant phenotypes.
</p>
</details>
<br>
<details>
<summary>Portrait of a genus: the genetic diversity of Zea [<a href="https://www.biorxiv.org/content/10.1101/2021.04.07.438828v1?ct=">preprint</a>]<br />
Chen L, Luo J, Minliang Jin, <strong>Yang N</strong> …[21 authors]… Fernie AR, Warburton ML, <strong>Ross-Ibarra J</strong>, Yan J. </summary>
<p style="margin-left: 30px">
Maize is a globally valuable commodity and one of the most extensively studied genetic model organisms. However, we know surprisingly little about the extent and potential utility of the genetic variation found in the wild relatives of maize. Here, we characterize a high-density genomic variation map from ~700 genomes encompassing maize and all wild taxa of the genus Zea, identifying over 65 million single nucleotide polymorphisms (SNPs), 8 million Insertion/Deletion (InDel) polymorphisms, and over one thousand novel inversions. The variation map reveals evidence of selection within taxa displaying novel adaptations such as perenniality and regrowth. We focus in detail on evidence of convergent adaptation in highland teosinte and temperate maize. This study not only indicates the key role of hormone related pathways in highland adaptation and flowering time related pathways in high latitude adaptation, but also identifies significant overlap in the genes underlying adaptations to both environments. To show how this data can identify useful genetic variants, we generated and characterized novel mutant alleles for two flowering time candidate genes. This work provides the most extensive sampling to date of the genetic diversity inherent in the genus Zea, resolving questions on evolution and identifying adaptive variants for direct use in modern breeding.
</p>
</details>
<br>
<details>
<summary>Selective sorting of ancestral introgression in maize and teosinte along an elevational cline [<a href="https://www.biorxiv.org/content/10.1101/2021.03.05.434040v1">preprint</a>]<br />
Calfee E, <strong>Gates DJ</strong>, <strong>Lorant A</strong>, <strong>Perkins MT</strong>, Coop G, <strong>Ross-Ibarra J</strong>. </summary>
<p style="margin-left: 30px">
While often deleterious, hybridization can also be a key source of genetic variation and pre-adapted haplotypes, enabling rapid evolution and niche expansion. Here we evaluate these opposing selection forces on introgressed ancestry between maize (Zea mays ssp. mays), and its wild teosinte relative. Introgression from ecologically diverse teosinte may have facilitated maize’s global range expansion, in particular to challenging high elevation regions (&gt; 1500 m). We generated low-coverage genome sequencing data for 348 maize and mexicana individuals to evaluate patterns of introgression in 14 sympatric population pairs, spanning the elevational range of Zea mays ssp. mexicana, a teosinte endemic to the mountains of Mexico. While recent hybrids are commonly observed in sympatric populations and mexicana demonstrates fine-scale local adaptation, we find that the majority of mexicana ancestry tracts introgressed &gt;1000 generations ago. This mexicana ancestry seems to have maintained much of its diversity and likely came from a common ancestral source, rather than contemporary sympatric populations, resulting in relatively low Fst between mexicana ancestry tracts sampled from geographically distant maize populations. Introgressed mexicana ancestry is reduced in lower-recombination rate quintiles of the genome and around domestication genes, consistent with pervasive selection against introgression. However, we also find mexicana ancestry increases across the sampled elevational gradient and that high introgression peaks are most commonly shared among high-elevation maize populations, consistent with introgression from mexicana facilitating adaptation to the highland environment. In the other direction, we find patterns consistent with adaptive and clinal introgression of maize ancestry into sympatric mexicana at many loci across the genome, suggesting that maize also contributes to adaptation in mexicana, especially at the lower end of its elevational range. In sympatric maize, in addition to high introgression regions we find many genomic regions where selection for local adaptation maintains steep gradients in introgressed mexicana ancestry across elevation, including at least two inversions: the well-characterized Inv4m and a new 3 Mb inversion Inv9f surrounding the macrohairless1 locus on chromosome 9. The bulk of our ancestry selection outliers show no signals of sweeps or local sourcing from sympatric populations and so likely represent ancestral introgression sorted by selection, resulting in correlated but distinct outcomes of introgression in different contemporary maize landrace populations.
</p>
</details>
<br>
<details>
<p><summary>The genomic signature of wild-to-crop introgression during the domestiation of scarlet runner bean (<em>Phaseolus coccineus L.</em>) [<a href="https://www.biorxiv.org/content/10.1101/2021.02.03.429668v1">preprint</a>]<br />
Guerra-Garcia A, Rojas-Barrera IC, <strong>Ross-Ibarra J</strong>, Papa R, Piñero D.</p>
</summary>
<p style="margin-left: 30px">
The scarlet runner bean is an open-pollinated legume from the highlands of Mesoamerica that is cultivated in small-scale agriculture for its dry seeds and immature pods. Demographic bottlenecks associated with domestication might reduce genetic diversity and facilitate the accumulation of deleterious mutations. Conversely, introgression from wild relatives could be a source of variation. Using Genotyping by Sequencing data (79,286 SNVs) from 237 cultivated and wild samples, we evaluated the demographic history of traditional varieties from different regions of Mexico and looked for evidence of introgression between sympatric wild and cultivated populations. Traditional varieties have high levels of diversity, even though there is evidence of a severe initial genetic bottleneck, followed by a population expansion. Introgression from wild to domesticated populations was detected, but not in the opposite direction. This asymmetric introgression might contribute to the recovery of genetic variation and it has occurred at different times: constantly in the center of Mexico; recently in the North West; and anciently in the South. Several factors are acting together to increase and maintain genetic diversity in P. coccineus cultivars, such as demographic expansion and introgression. Wild relatives represent a valuable genetic resource and have played a key role in scarlet runner bean evolution via introgression into traditional varieties.
</p>
</details>
<br>
<details>
<p><summary>Teosinte introgression modulates phosphatidylcholine levels and induces early maize flowering time [<a href="https://www.biorxiv.org/content/10.1101/2021.01.25.426574v1">preprint</a>]<br />
Rodríguez-Zapata F, Barnes AC, Blöcher-Juárez KA, <strong>Gates DJ</strong> …[16 authors]… <strong>Ross-Ibarra J</strong>, Hufford M, Sawers RJH, Rellán-Álvarez R.</p>
</summary>
<p style="margin-left: 30px">
After domestication from lowland teosinte parviglumis (Zea mays ssp. parviglumis) in the warm Mexican southwest, maize (Zea mays ssp. mays) colonized the highlands of Mexico and South America. In the highlands, maize was exposed to lower temperatures that imposed strong selection on flowering time. Previous work in maize and other has linked variation in phospholipid metabolism to low temperature stress as well as changes in flowering time. Here, we combined linkage mapping analysis with genome scans to identify High PhosphatidylCholine 1 (HPC1), a gene which encodes a phospholipase A1 enzyme, as a major driver of phospholipid variation in highland maize. Common garden experiments demonstrated strong genotype-by-environment interactions associated with variation at HPC1, with the highland HPC1 allele leading to higher fitness in highlands, possibly by hastening flowering. The HPC1 variant we identified in highland maize results in impaired function of the encoded protein due to a polymorphism in a highly conserved sequence. A meta-analysis indicated a strong association between the identity of the amino acid at this position in a prokaryotic protein harboring this conserved sequence and optimal growth temperature of the organism. Mutagenesis of HPC1 via genome editing validated its role in regulating phospholipid metabolism. Finally, we show that the highland HPC1 allele entered cultivated maize by introgression from the wild highland teosinte Zea mays ssp. mexicana and has been maintained in maize breeding lines from Northern US, Canada and Europe. Thus, HPC1 introgressed from teosinte mexicana underlies a large metabolic QTL that modulates phosphatidylcholine levels and has an adaptive effect at least in part via induction of early flowering time.
</p>
</details>
<br>
<details>
<summary>Conflict over fertilization underlies the transient evolution of reinforcement. [<a href="https://www.biorxiv.org/content/10.1101/2020.11.10.377481v1">preprint</a>]<br />
<strong>Rushworth C</strong>, Wardlaw AW, <strong>Ross-Ibarra J</strong>, Brandvain YB. </summary>
<p style="margin-left: 30px">
When two populations or species hybridize, their offspring often experience reductions in fitness relative to either parental population. The production of low fitness hybrids may be prevented by the evolution of increased prezygotic isolation; a process known as reinforcement. Theoretical challenges to the evolution of reinforcement are generally cast as a coordination problem — e.g., linkage disequilibrium between trait and preference loci is difficult to maintain in the face of recombination. However, the evolution of reinforcement also poses a potential conflict between mates. For example, the opportunity costs to hybridization may differ between the sexes or species. This is particularly likely for postmating prezygotic isolation, as the ability to fertilize both conspecific and heterospecific eggs is beneficial to male gametes, but heterospecific mating may incur a cost for female gametes. Motivated by this problem, we develop a population genetic model of interspecific conflict over reinforcement, inspired by `gametophytic factors’, which act as postmating prezygotic barriers among Zea mays subspecies. We demonstrate that this conflict results in the transient evolution of reinforcement –– after female preference for a conspecific gamete trait rises to high frequency, male traits adaptively introgress into the other population. Ultimately the male gamete trait fixes in both species, and prezygotic isolation returns to pre-reinforcement levels. We interpret geographic patterns of isolation among Z. mays subspecies in light of these findings, and suggest when and how this conflict can be mediated. Our results suggest that sexual conflict may pose an understudied obstacle to the evolution of reinforcement via postmating prezygotic isolation.
</p>
</details>
<br>
<details>
<summary>Single gene resolution of locally adaptive genetic variation in Mexican maize [<a href="https://www.biorxiv.org/content/10.1101/706739v1">preprint</a>]<br />
<strong>Gates DJ</strong>, Runcie D, Janzen GM, Romero Navarro A …[4 authors]… Buckler ES, Hearne S, Hufford MB, <strong>Ross-Ibarra J</strong><br />
</summary>
<p style="margin-left: 30px">
We present a high-resolution genome-wide association analysis to identify loci exhibiting adaptive patterns in a large panel of more than 4500 traditional maize landraces representing the breadth of genetic diversity of maize in Mexico. We evaluate associations between genotype and plant performance in 13 common gardens across a range of environments, identifying hundreds of candidate genes underlying genotype by environment interaction. We further identify genetic associations with environment across Mexico and show that such loci are associated with variation in yield and flowering time in our field trials and predict performance in independent drought trials. Our results indicate that the variation necessary to adapt crops to changing climate exists in traditional landraces that have been subject to ongoing environmental adaptation and can be identified by both phenotypic and environmental association.
</p>
</details>
<p><br></p>
</div>
<div id="section" class="section level3">
<h3>2021</h3>
<br>
<details>
<summary>The Genomic Ecosystem of Transposable Elements in Maize [<a href="https://www.biorxiv.org/content/10.1101/559922v1">preprint</a>][<a href="https://github.com/mcstitzer/maize_genomic_ecosystem">github</a>][<a href="https://mcstitzer.shinyapps.io/maize_te_families/">interactive</a>]<br />
<strong>Stitzer MC</strong>, Anderson SN, Springer NM, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
Transposable elements (TEs) constitute the majority of flowering plant DNA, reflecting their tremendous success in subverting, avoiding, and surviving the defenses of their host genomes to ensure their selfish replication. More than 85% of the sequence of the maize genome can be ascribed to past transposition, providing a major contribution to the structure of the genome. Evidence from individual loci has informed our understanding of how transposition has shaped the genome, and a number of individual TE insertions have been causally linked to dramatic phenotypic changes. But genome-wide analyses in maize and other taxa have frequently represented TEs as a relatively homogeneous class of fragmentary relics of past transposition, obscuring their evolutionary history and interaction with their host genome. Using an updated annotation of structurally intact TEs in the maize reference genome, we investigate the family-level ecological and evolutionary dynamics of TEs in maize. Integrating a variety of data, from descriptors of individual TEs like coding capacity, expression, and methylation, as well as similar features of the sequence they inserted into, we model the relationship between these attributes of the genomic environment and the survival of TE copies and families. Our analyses reveal a diversity of ecological strategies of TE families, each representing the evolution of a distinct ecological niche allowing survival of the TE family. In contrast to the wholesale relegation of all TEs to a single category of junk DNA, these differences generate a rich ecology of the genome, suggesting families of TEs that coexist in time and space compete and cooperate with each other. We conclude that while the impact of transposition is highly family- and context-dependent, a family-level understanding of the ecology of TEs in the genome can refine our ability to predict the role of TEs in generating genetic and phenotypic diversity.
</p>
</details>
<br>
<details>
<summary><a href="https://science.sciencemag.org/content/373/6555/655.abstract"><em>De novo</em> assembly, annotation, and comparative analysis of 26 diverse maize genomes</a>. [<a href="https://www.biorxiv.org/content/10.1101/2021.01.14.426684v1">preprint</a>]<br />
Hufford MB, Seetharam AS, …[40 authors including <strong>Hudson AI</strong> and <strong>Tittes S</strong>]… <strong>Ross-Ibarra J</strong>, Yu J, Gent JI, Hirsch CN, Ware D, Dawe RK. </summary>
<p style="margin-left: 30px">
We report de novo genome assemblies, transcriptomes, annotations, and methylomes for the 26 inbreds that serve as the founders for the maize nested association mapping population. The data indicate that the number of pan-genes exceeds 103,000 and that the ancient tetraploid character of maize continues to degrade by fractionation to the present day. Excellent contiguity over repeat arrays and complete annotation of centromeres further reveal the locations and internal structures of major cytological landmarks. We show that combining structural variation with SNPs can improve the power of quantitative mapping studies. Finally, we document variation at the level of DNA methylation, and demonstrate that unmethylated regions are enriched for cis-regulatory elements that overlap QTL and contribute to changes in gene expression.
</p>
</details>
<br>
<details>
<summary><a href="https://genome.cshlp.org/content/early/2021/05/27/gr.266528.120.abstract#aff-4">Conserved noncoding sequences provide insights into regulatory sequence and loss of gene expression in maize</a>. [<a href="https://www.biorxiv.org/content/10.1101/2020.07.11.192575v2">preprint</a>]<br />
Song B, Buckler ES, Wang H, Wu Y, …[4 authors including <strong>D Gates</strong>]… Bradbury PJ, <strong>Ross-Ibarra J</strong>, Hufford MB, Romay MC.<br />
</summary>
<p style="margin-left: 30px">
DNA sequencing technology has advanced so quickly, identifying key functional regions using evolutionary approaches is required to understand how those genomes work. This research develops a sensitive sequence alignment approach to identify functional constrained non-coding sequences in the Andropogoneae tribe. The grass tribe Andropogoneae contains several crop species descended from a common ancestor ~18 million years ago. Despite broadly similar phenotypes, they have tremendous genomic diversity with a broad range of ploidy levels and transposons. These features make Andropogoneae a powerful system for studying conserved non-coding sequence (CNS), here we used it to understand the function of CNS in maize. We find that 86% of CNS comprise known genomic elements e.g., cis-regulatory elements, chromosome interactions, introns, several transposable element superfamilies, and are linked to genomic regions related to DNA replication initiation, DNA methylation and histone modification. In maize, we show that CNSs regulate gene expression and variants in CNS are associated with phenotypic variance, and rare CNS absence contributes to loss of gene expression. Furthermore, we find the evolution of CNS is associated with the functional diversification of duplicated genes in the context of the maize subgenomes. Our results provide a quantitative understanding of constrained non-coding elements and identify functional non-coding variation in maize.
</p>
</details>
<br>
<details>
<summary><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab119/6255756">Molecular Parallelism Underlies Convergent Highland Adaptation of Maize Landraces</a> [<a href="https://www.biorxiv.org/content/10.1101/2020.07.31.227629v1">preprint</a>]<br />
<strong>Wang L</strong>, <strong>Josephs EB</strong>, Lee KM, Roverts LM, Rellán-Álvarez R, <strong>Ross-Ibarra J</strong>, Hufford MB.<br />
</summary>
<p style="margin-left: 30px">
Convergent phenotypic evolution provides some of the strongest evidence for adaptation. However, the extent to which recurrent phenotypic adaptation has arisen via parallelism at the molecular level remains unresolved, as does the evolutionary origin of alleles underlying such adaptation. Here, we investigate genetic mechanisms of convergent highland adaptation in maize landrace populations and evaluate the genetic sources of recurrently selected alleles. Population branch excess statistics reveal strong evidence of parallel adaptation at the level of individual SNPs, genes and pathways in four independent highland maize populations, even though most SNPs show unique patterns of local adaptation. The majority of selected SNPs originated via migration from a single population, most likely in the Mesoamerican highlands. Polygenic adaptation analyses of quantitative traits reveal that alleles affecting flowering time are significantly associated with elevation, indicating the flowering time pathway was targeted by highland adaptation. In addition, repeatedly selected genes were significantly enriched in the flowering time pathway, indicating their significance in adapting to highland conditions. Overall, our study system represents a promising model to study convergent evolution in plants with potential applications to crop adaptation across environmental gradients.
</p>
</details>
<br>
<details>
<summary><a href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyab061/6237897?searchresult=1">Gene body methylation is under selection in <em>Arabidopsis thaliana</em>.</a> [<a href="https://www.biorxiv.org/content/10.1101/2020.09.04.283333v1">preprint</a>]<br />
Muyle A, <strong>Ross-Ibarra J</strong>, Seymour DK, Gaut BS. </summary>
<p style="margin-left: 30px">
In plants, mammals and insects, some genes are DNA methylated in the CG dinucleotide context, a phenomenon called gene body methylation. It has long been controversial whether this phenomenon has any functional role. Here, we took advantage of the availability of 868 leaf methylomes in Arabidopsis thaliana to characterize the population frequency of gene body methylation at the gene level. We used two outgroups to infer the ancestral methylation state of orthologs. Using the A. thaliana gene body methylation site frequency spectrum and a population genetics model specifically designed for epigenetic data, we find that genes with ancestral gene body methylation are under significant selection to remain methylated. Conversely, ancestrally unmethylated genes are under selection to remain unmethylated. The estimated selection coefficients are small, on the order of magnitude of selection acting on codon usage. We also find that A. thaliana is losing gene body methylation more often than gaining it, which could be due to a recent reduction in the efficacy of selection after a switch to selfing. We investigated the potential function of gene body methylation through its link with gene expression level. We show that, within genes with polymorphic methylation states in A. thaliana wild populations, the gene body methylated state is consistently associated with the highest gene expression level. Our work suggests that gene body methylation has a small effect on fitness, but substantial enough for natural selection to act on it.
</p>
</details>
<br>
<details>
<summary><a href="https://www.nature.com/articles/s41477-020-00834-5.epdf?sharing_token=_mN9oSJzgPbS9Ldsr01s8dRgN0jAjWel9jnR3ZoTv0OIqw-nLs7dqtihevHgjRdBgKyJmnWQSkGOUEc4hulR2jnTLjP4fqtKbSUSzA7iWbyA0E0YEvXdfVUO_1_dmi2uNdJtrPRcKz9O_jp58VwFcP02Yhzr7I3YNxgg8Oit1Xk%3D">Comparative evolutionary analysis and prediction of deleterious mutation patterns between sorghum and maize</a> [<a href="https://doi.org/10.1101/777623">preprint</a>][<a href="https://github.com/GoreLab/Sorghum-HapMap">github</a>]<br />
Lozano R, Gazave E, dos Santos JPR, Stetter MG …[4 authors]… Mockler T, Buckler ES, <strong>Ross-Ibarra J</strong>, Gore MA </summary>
<p style="margin-left: 30px">
Sorghum and maize share a close evolutionary history that can be explored through comparative genomics. To perform a large-scale comparison of the genomic variation between these two species, we analyzed 13M variants identified from whole genome resequencing (WGS) of 468 sorghum lines together with 25M variants previously identified in 1,218 maize lines. Deleterious mutations in both species were prevalent in pericentromeric regions, enriched in non-syntenic genes, and present at low allele frequencies. A comparison of deleterious burden between sorghum and maize revealed that sorghum, in contrast to maize, departed from the “domestication cost” hypothesis that predicts a higher deleterious burden among domesticates compared to wild lines. Additionally, sorghum and maize population genetic summary statistics were used to predict a gene deleterious index with an accuracy higher than 0.5. This research represents a key step towards understanding the evolutionary dynamics of deleterious variants in sorghum and provides a comparative genomics framework to start prioritizing them for removal through genome editing and breeding.
</p>
</details>
<p><br></p>
</div>
<div id="section-1" class="section level3">
<h3>2020</h3>
<br>
<details>
<summary><a href="https://www.nature.com/articles/s41467-020-19333-4">Adaptive evolution of DNA methylation reshaped gene regulation in maize</a> [<a href="https://www.biorxiv.org/content/10.1101/2020.03.13.991117v1">preprint</a>] [<a href="https://github.com/jyanglab/msfs_teo">github</a>]<br />
Xu G, Lyu J, Li Q, Liu H, Wang D, Zhang M, Springer NM, <strong>Ross-Ibarra J</strong>, Yang J </summary>
<p style="margin-left: 30px">
DNA methylation is a ubiquitous chromatin feature – in maize, more than 25% of cytosines in the genome are methylated. Recently, major progress has been made in describing the molecular mechanisms driving methylation, yet variation and evolution of the methylation landscape during maize domestication remain largely unknown. Here we leveraged whole-genome sequencing (WGS) and whole-genome bisulfite sequencing (WGBS) on populations of modern maize, landrace, and teosinte (Zea mays ssp. parviglumis) to investigate the adaptive and phenotypic consequences of methylation variations in maize. By using a novel estimation approach, we inferred the methylome site frequency spectrum (mSFS) to estimate forward and backward methylation mutation rates and selection coefficients. We only found weak evidence for direct selections on methylations in any context, but thousands of differentially methylated regions (DMRs) were identified in population-wide that are correlated with recent selections. Further investigation revealed that DMRs are enriched in 5’ untranslated regions, and that maize hypomethylated DMRs likely helped rewire distal gene regulation. For two trait-associated DMRs, vgt1-DMR and tb1-DMR, our HiChIP data indicated that the interactive loops between DMRs and respective downstream genes were present in B73, a modern maize line, but absent in teosinte. And functional analyses suggested that these DMRs likely served as cis-acting elements that modulated gene regulation after domestication. Our results enable a better understanding of the evolutionary forces acting on patterns of DNA methylation and suggest a role of methylation variation in adaptive evolution.
</p>
</details>
<br>
<details>
<summary><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008791">The genetic architecture of the maize progenitor, teosinte, and how it was altered during maize domestication</a><br />
Chen Q, Fernando Samayoa L, Yang CJ, …[5 authors]… <strong>Lorant A</strong>, Buckler ES, <strong>Ross-Ibarra J</strong>, Holland JB, Doebley JF </summary>
<p style="margin-left: 30px">
The genetics of domestication has been extensively studied ever since the rediscovery of Mendel’s law of inheritance and much has been learned about the genetic control of trait differences between crops and their ancestors. Here, we ask how domestication has altered genetic architecture by comparing the genetic architecture of 18 domestication traits in maize and its ancestor teosinte using matched populations. We observed a strongly reduced number of QTL for domestication traits in maize relative to teosinte, which is consistent with the previously reported depletion of additive variance by selection during domestication. We also observed more dominance in maize than teosinte, likely a consequence of selective removal of additive variants. We observed that large effect QTL have low minor allele frequency (MAF) in both maize and teosinte. Regions of the genome that are strongly differentiated between teosinte and maize (high F ST ) explain less quantitative variation in maize than teosinte, suggesting that, in these regions, allelic variants were brought to (or near) fixation during domestication. We also observed that genomic regions of high recombination explain a disproportionately large proportion of heritable variance both before and after domestication. Finally, we observed that about 75% of the additive variance in both teosinte and maize is “missing” in the sense that it cannot be ascribed to detectable QTL and only 25% of variance maps to specific QTL. This latter result suggests that morphological evolution during domestication is largely attributable to very large numbers of QTL of very small effect.
</p>
</details>
<br>
<details>
<summary><a href="https://www.g3journal.org/content/ggg/early/2020/05/28/g3.120.401196.full.pdf">Loss of diversity and accumulation of genetic load in doubled-haploid lines from European maize landraces</a> [<a href="https://www.biorxiv.org/content/10.1101/817791v1">preprint</a>][<a href="https://github.com/LZeitler/eurodh-scripts/blob/master/README.org">github</a>]<br />
<strong>Zeitler L</strong>, <strong>Ross-Ibarra J</strong>, <strong>Stetter MGS</strong> </summary>
<p style="margin-left: 30px">
Maize landraces are well adapted to their local environments and present valuable sources of genetic diversity for breeding and conservation. But the maintenance of open-pollinated landraces in  programs is challenging, as regeneration of seed can often lead to inbreeding depression and the loss of diversity due to genetic drift. Recent reports suggest that the production of doubled-haploid (DH) lines from landraces may serve as a convenient means to preserve useful genetic diversity in a homozygous form that is immediately useful for modern breeding. The production of doubled-haploid (DH) lines presents an extreme case of inbreeding which results in instantaneous homozygosity genome-wide. Here we analyzed the effect of DH production on genetic diversity, using genome-wide SNP data from hundreds of individuals of five European landraces and their related DH lines. In contrast to previous findings, we observe a dramatic loss of diversity at both the haplotype level and that of individual SNPs. We identify thousands of SNPs that exhibit allele frequency differences larger than expected under models of neutral genetic drift, and document patterns of heterozygosity and polymorphism at conserved sites that suggest an important role for deleterious recessive load in determining diversity differences between landrace and DH populations. Although we were unable to uncover more details about the mode of selection, we conclude that while landrace DH lines may be a valuable tool for the introduction of variation into maize breeding programs they come at the cost of decreased genetic diversity and increased genetic load.
</p>
</details>
<br>
<details>
<summary><a href="https://drive.google.com/file/d/1UxPN_Yf2HQ0OOJdqbP537Bwxlm2N1EkL/view?usp=sharing">Genome-wide selection and genetic improvement during modern maize breeding</a><br />
Wang B, Lin Z, Li X, Zhao Y …[17 authors]… Hufford MB, <strong>Ross-Ibarra J</strong>, He H, Wang H. </summary>
<p style="margin-left: 30px">
Modern crop breeding has made a profound impact on food production to sustain population growth during the past century; however, systematic analyses of the genomic basis underlying the dramatic increase in crop yields during modern breeding remain limited in scope. Here we report an analysis of the genomic and phenotypic changes associated with modern maize breeding across a chronological sampling of 350 elite inbred lines representing multiple eras of both Chinese and U.S. germplasm. We document a number of convergent phenotypic changes in both countries. Using genome-wide association and selection scan methods, we identify 160 loci underlying adaptive agronomic phenotypes and more than 1,800 genomic regions representing the targets of selection during modern breeding. We functionally validate two candidate genes underpinning variation in ear height and tassel branch number using CRISPR knockout mutants. In sum, this work demonstrates the utility of breeding-era approach for identifying breeding signatures and lays the foundation for future genomics-enabled maize breeding.&quot;
</p>
</details>
<br>
<details>
<summary><a href="https://www.sciencedirect.com/science/article/pii/S1369526620300339">Evolutionary insights into plant breeding</a> [<a href="https://agrixiv.org/akdt8">preprint</a>]<br />
<strong>Turner-Hissong SD</strong>, Mabry ME, Beissinger TM, <strong>Ross-Ibarra J</strong>, Pires JC </summary>
<p style="margin-left: 30px">
Crop domestication is a fascinating area of study, as evidencedby a multitude of recent reviews. Coupled with the increasing availability of genomic and phenomic resources in numerous crop species, insights from evolutionary biology will enable a deeper understanding of the genetic architecture and short-term evolution of complex traits, which can be used to inform selection strategies. Future advances in crop improvement will rely on the integration of population genetics with plant breeding methodology, and the development of community resources to support research in a variety of crop life histories and reproductive strategies. We highlight recent advances in the role of selective sweeps and demographic history in shaping genetic architecture, how these breakthroughs can inform selection strategies, and the application of precision gene editing to leverage these connections.
</p>
</details>
<br>
<details>
<summary><a href="http://library.oapen.org/bitstream/handle/20.500.12657/23339/1006816.pdf?sequence=1#page=292">Genomics of long- and short- term adaptation in maize and teosinte</a> [<a href="https://peerj.com/manuscripts/31241/">preprint</a>]<br />
<strong>Lorant A</strong>, <strong>Ross-Ibarra J</strong>, Tenaillon M<br />
</summary>
<p style="margin-left: 30px">
We review studies on short and long term adaptation, both natural and artificial, in maize and teosinte.
</p>
</details>
<br>
<details>
<summary><a href="https://www.genetics.org/content/214/4/1019">The temporal dynamics of background selection in non-equilibrium populations</a> [<a href="https://www.biorxiv.org/content/10.1101/618389v1">preprint</a>][<a href="https://github.com/RILAB/BGS_sims/">github</a>]<br />
Torres R, <strong>Stetter MG</strong>, Hernandez R, <strong>Ross-Ibarra J</strong><br />
</summary>
<p style="margin-left: 30px">
We use simulations to show that the effects of nonequilibrium population dynamics on linked selection differ qualitatively from predictions under equilibrium models, calling into question inferences about the efficacy of selection made from summary statistics.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Catherine Rushworth</strong></p>
<pre><code>Rushworth, C.A., R.S. Baucom, B.K. Blackman, M. Neiman, M.E. Orive, A. Sethuraman, J. Ware, and D.R. Matute. 2021. Who are we now? A demographic assessment of the evolution societies. In press, Evolution. Early View: https://onlinelibrary.wiley.com/doi/10.1111/evo.14168.  

Rushworth, C.A., and T. Mitchell-Olds. 2021. The evolution of sex is tempered by costly hybridization in Boechera (rock cress). Journal of Heredity Early View: https://doi.org/10.1093/jhered/esaa041.  

Rushworth, C.A., Y. J. Brandvain, and T. Mitchell-Olds. 2020. Identifying the fitness consequences of sex in complex natural environments. Evolution Letters 4-6: 516-529. https://dx.doi.org/10.1002/evl3.194  

Jordon-Thaden, I., J. Beck, C.A. Rushworth, M.D. Windham, N. Diaz, J. Cantley, C. Martine, and C.J. Rothfels. 2020. RADseq for any lab: a basic two-enzyme protocol performs well in herbarium and silica-dried tissues across four genera. Applications in Plant Sciences 8(4): e11344.  </code></pre>
<p><strong>Silas Tittes</strong></p>
<pre><code>Tittes, Silas. &quot;rdmc: an open source R package implementing convergent adaptation models of Lee and Coop (2017).&quot; G3: Genes, Genomes, Genetics 10.9 (2020): 3041-3046.  

O&#39;Hara, N. B., Franks, S. J., Kane, N. C., Tittes, S., &amp; Rest, J. S. (2020). Evolution of pathogen response genes associated with increased disease susceptibility during adaptation to an extreme drought in a Brassica rapa plant population.  

Wooliver, R., Tittes, S. B., &amp; Sheth, S. N. (2020). A resurrection study reveals limited evolution of thermal performance in response to recent climate change across the geographic range of the scarlet monkeyflower. Evolution, 74(8), 1699-1710.  

Comeault, A. A., Wang, J., Tittes, S., Isbell, K., Ingley, S., Hurlbert, A. H., &amp; Matute, D. R. (2020). Genetic diversity and thermal performance in invasive and native populations of African fig flies. Molecular biology and evolution, 37(7), 1893-1906.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-2" class="section level3">
<h3>2019</h3>
<br>
<details>
<summary><a href="http://www.g3journal.org/cgi/pmidlookup?view=long&amp;pmid=31506319">Dynamic patterns of transcript abundance of transposable element families in maize</a> [<a href="https://www.biorxiv.org/content/10.1101/668558v1">preprint</a>]<br />
Anderson SN, <strong>Stitzer MC</strong>, Zhou P, <strong>Ross-Ibarra J</strong>, Hirsch CD, Springer NM </summary>
<p style="margin-left: 30px">
We develop an approach to estiamte family-level expression in transposable elements. While a relatively small proportion of TE families are transcribed, expression is highly dynamic with most families exhibiting tissue-specific expression. We also assay expression among different maize genotypes and use a mapping population to identify what proportion of copies in a family contribute to expression.&quot;
</p>
</details>
<br>
<details>
<summary><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/tpj.14489">Transposable elements contribute to dynamic genome content in maize</a> [<a href="https://www.biorxiv.org/content/10.1101/547398v1?rss=1">preprint</a>]<br />
Anderson SN, <strong>Stitzer MC</strong>, Brohammer AB, Peng Zhou P, Noshay JM, Hirsch CD, <strong>Ross-Ibarra J</strong>, Hirsch CN, Springer NM </summary>
<p style="margin-left: 30px">
We identify hundreds of thousands of variable transposable elements among a small set of maize genome assemblies. We find evidence of a large number of genes impacted by TEs, including TEs in genes and genes in TEs, and evidence of recent transposition as well.&quot;&quot;
</p>
</details>
<br>
<details>
<summary><a href="https://www.genetics.org/content/213/1/143">Hybrid decay: a transgenerational epigenetic decline in vigor and viability triggered in backcross populations of teosinte with maize</a> [<a href="https://www.biorxiv.org/content/10.1101/588715v1">preprint</a>]<br />
Wei X, Anderson SN, Wang X, …[<strong>Bilinski P</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong> and 4 other authors]… Chen X, Springer NM, Doebley JF </summary>
<p style="margin-left: 30px">
We describe a phenomenon we name ‘hybrid decay’ triggered by backcrossing between domesticated maize and a specific teosinte population, resulting in genome instability, activation of transposable elements, and altered epigenetics .&quot;
</p>
</details>
<br>
<details>
<summary><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/evo.13818">Adaptive phenotypic divergence in teosinte differs across biotic contexts</a> [<a href="https://www.biorxiv.org/content/10.1101/382770v2">preprint</a>] [<a href="https://onlinelibrary.wiley.com/doi/full/10.1111/evo.13857">perspective</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Strauss SY, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that rhizosphere biota impact adaptive divergence of phenotypic traits in teosinte along an environmental cline. This work suggests an important role for biotic interactions in determining the outcome of local adaptation.
</p>
</details>
<p><br></p>
<details>
<summary><a href="https://peerj.com/articles/6815/">Characterization of introgression from the teosinte <em>Zea mays</em> ssp. <em>mexicana</em> to Mexican highland maize</a> [<a href="https://peerj.com/preprints/27489/">preprint</a>]<br />
Gonzalez-Segovia E, Pérez-Limon S, C Cíntora-Martínez C, Guerrero-Zavala A, Jansen G, Hufford MB, <strong>Ross-Ibarra J</strong>, Ruairidh J H Sawers </summary>
<p style="margin-left: 30px">
We look for introgression from teosinte in the genomes of two highland landraces and use QTL mapping to test for functional relevance of introgressed regions.
</p>
</details>
<br>
<details>
<summary><a href="https://www.genetics.org/content/211/3/989">Detecting adaptive differentiation in structured populations with genomic data and common gardens</a> [<a href="https://www.biorxiv.org/content/early/2018/07/13/368506">preprint</a>] [<a href="https://github.com/emjosephs/qpc-maize">github</a>]<br />
<strong>Josephs EB</strong>, Berg JJ, <strong>Ross-Ibarra J</strong>, Coop G </summary>
<p style="margin-left: 30px">
We develop methods to detect polygenic adaptation on phenotypes from common gardens and GWAS panels.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Taylor Perkins</strong></p>
<pre><code>Perkins, M. Taylor, Tetyana Zhebentyayeva, Paul H. Sisco, and J. Hill Craddock. “Genome-wide sequence-based genotyping supports a nonhybrid origin of Castanea alabamensis.” bioRxiv (2019): 680371.   

Craddock, J. Hill, and M. Taylor Perkins. “Chestnut (Castanea spp. Miller) Breeding.” In Advances in Plant Breeding Strategies, vol. 4. (J.M. Al-Khayri et al., eds.)(in press), 55 pages. Springer Nature Switzerland AG, Basel.  </code></pre>
<p><strong>Michelle Stitzer</strong></p>
<pre><code>Roessler, Kyria, Aline Muyle, Concepcion M. Diez, Garren RJ Gaut, Alexandros Bousios, Michelle C. Stitzer, Danelle K. Seymour, John F. Doebley, Qingpo Liu, and Brandon S. Gaut. &quot;The genome-wide dynamics of purging during selfing in maize.&quot; Nature plants (2019): 1-11.  

Noshay, Jaclyn M., Sarah N. Anderson, Peng Zhou, Lexiang Ji, William Ricci, Zefu Lu, Michelle C. Stitzer et al. &quot;Monitoring the interplay between transposable element families and DNA methylation in maize.&quot; PLoS genetics 15, no. 9 (2019): e1008291.     

Marcela K. Tello-Ruiz, Cristina F. Marco, Fei-Man Hsu, Rajdeep S. Khangura, Pengfei Qiao, Sirjan Sapkota, Michelle C. Stitzer, Rachael Wasikowski, Hao Wu, Junpeng Zhan, Kapeel Chougule, Lindsay C. Barone, Cornel Ghiban, Demitri Muna, Andrew C. Olson, Liya C. Wang, Doreen C. Ware, David A. Micklos bioRxiv 654848; doi: https://doi.org/10.1101/654848  

Stitzer, Michelle C., and Philipp Brand. &quot;Digest: Hybrid incompatibilities and introgression in wild monkeyflowers.&quot; Evolution 72, no. 11 (2018): 2565-2566.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-3" class="section level3">
<h3>2018</h3>
<details>
<summary><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007794">Genetic architecture and selective sweeps after polygenic adaptation to distant trait optima</a> [<a href="https://www.biorxiv.org/content/early/2018/05/03/313247">preprint</a>] [<a href="https://mgstetter.shinyapps.io/quantgensimAPP/">interactive</a>]<br />
<strong>Stetter MG</strong>, Thornton K, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We use simulations to study the population genetics of selection on a quantitative trait and learn which parameters drive quantitative trait evollution.
</p>
</details>
<br>
<details>
<summary><a href="https://www.journals.uchicago.edu/doi/10.1086/700118">Evolutionary responses to conditionality in species interactions across environmental gradients</a> [<a href="http://biorxiv.org/content/early/2017/03/01/031195">preprint</a>][<a href="https://www.authorea.com/inst/14743?articles_format=list&amp;page=5">review</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, <strong>Ross-Ibarra J</strong>, Strauss SY </summary>
<p style="margin-left: 30px">
We present the CoCoA hypothesis which describes how patterns of biotic coadpation change across varying abiotic environments
</p>
</details>
<br>
<details>
<summary><a href="https://books.google.com/books?id=ZZl7DwAAQBAJ&amp;pg=PA318&amp;lpg=PA318&amp;dq=maize+genome+snodgrass&amp;source=bl&amp;ots=jD5IvHVlpG&amp;sig=JtkrFv2iBqp4Noq5ud81Y0aIQrU&amp;hl=en&amp;sa=X&amp;ved=2ahUKEwigrdeIkf3eAhXnwlQKHQqgBEEQ6AEwAXoECAMQAQ#v=onepage&amp;q=maize%20genome%20snodgrass&amp;f=false">Evolution and Adaptation in the Maize Genome</a><br />
Manchanda N, Snodgrass SJ, <strong>Ross-Ibarra J</strong>, Hufford MB<br />
</summary>
<p style="margin-left: 30px">
We review progress thus far in genomic research of maize domestication and adaptation. We discuss the insights genomics has shed on our understanding of these processes and conclude with a future outlook for how genomics might be further applied.
</p>
</details>
<br>
<details>
<summary><a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.15350">Maize domestication and gene interaction</a> [<a href="https://peerj.com/preprints/26502/">preprint</a>]<br />
<strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We review genetic work looking at the role of dominance, epistasis, and pleiotropy during maize domestication.
</p>
</details>
<br>
<details>
<summary><a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">A novel maize kinesin causes neocentromere activity and meiotic drive, altering inheritance patterns across the genome</a> [<a href="http://sci-hub.tw/http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">pdf</a>] [<a href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;cad=rja&amp;uact=8&amp;ved=0ahUKEwiIk5W14eHSAhVO92MKHbseACgQtwIIHDAA&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DbcCxsVPwOE8&amp;usg=AFQjCNGJ2xxs5xSbskrcavIoJ0D8ve3fjA&amp;sig2=gmS2FYQxQlrQJRw8PGPDFw">YouTube</a>]<br />
Dawe RK, Lowry EG, Gent JI, <strong>Stitzer MC</strong>, Higgins DM, <strong>Ross-Ibarra J</strong>, Wallace J, Kanizay LB … [7 authors]</summary>
<p style="margin-left: 30px">
We identify a small gene family Kindr as the causal locus for meiotic drive in maize.
</p>
</details>
<br>
<details>
<summary><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162">Parallel altitudinal clines reveal adaptive evolution of genome size in <em>Zea mays</em></a> [<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007249">perspective</a>] [<a href="http://www.biorxiv.org/content/early/2017/07/13/134528">preprint</a>] [<a href="https://github.com/paulbilinski/GenomeSizeAnalysis">github</a>] [<a href="https://www.slideshare.net/jrossibarra/parallel-altitudinal-clines-reveal-adaptive-evolution-of-genome-size-in-zea-mays">slides</a>]<br />
<strong>Bilinski P</strong>, Albert P, Berg JJ, Birchler JA, Grote M, <strong>Lorant A</strong>, <strong>Quezada J</strong>, Swarts K, <strong>Yang J</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that genome size can be analyzed as a quantitative trait and that selection on flowering time in maize has likely driven adaptive changes in genome size along multiple altitudinal clines.
</p>
</details>
<br>
<details>
<summary><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajb2.1002">Adaptation in plant genomes: bigger is different</a> [<a href="https://www.biorxiv.org/content/early/2017/10/02/196501">preprint</a>][<a href="https://github.com/RILAB/AJB_MutationalTargetSize_GenomeSize">github</a>][<a href="https://www.youtube.com/watch?v=aXLuWruOmO4_">YouTube</a>][<a href="https://www.slideshare.net/jrossibarra/adaptation-in-plant-genomes-bigger-is-different">slides</a>]<br />
<strong>Mei W</strong>, <strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We develop the functional space hypothesis in which we posit that adaptation in large genomes uses more noncoding variation and is more likely to proceed via soft sweeps.
</p>
</details>
<br>
<details>
<summary><a href="https://www.sciencedirect.com/science/article/pii/S1462901117310602">Harnessing cross-border resources to confront climate change</a><br />
Aburto-Oropeza O …[<strong>Ross-Ibarra J</strong> and 85 authors in alphabetical order]… Taylor JE </summary>
<p style="margin-left: 30px">
Discusses the challenges of changing climates pose for society and the environment along the Mexican-US border.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Wenbin Mei</strong> and <strong>Michelle Stitzer</strong></p>
<pre><code>Springer, Nathan M., et al. &quot;The maize W22 genome provides a foundation for functional genomics and transposon biology.&quot; Nature genetics (2018): 1.</code></pre>
<p><strong>Markus Stetter</strong></p>
<pre><code>Joshi, Dinesh C., et al. &quot;From zero to hero: the past, present and future of grain amaranth breeding.&quot; Theoretical and Applied Genetics (2018): 1-17.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B. &quot;Determining the evolutionary forces shaping G× E.&quot; New Phytologist (2018).

Uzunović, Jasmina, et al. &quot;Transposable elements are important contributors to standing variation in gene expression in Capsella grandiflora.&quot; bioRxiv (2018): 289173.</code></pre>
<p><strong>Anne Lorant</strong></p>
<pre><code>Kremling, Karl AG, et al. &quot;Dysregulation of expression correlates with rare-allele burden and fitness loss in maize.&quot; Nature 555.7697 (2018): 520.</code></pre>
<p><strong>Daniel Gates</strong></p>
<pre><code>Gates, Daniel J., Diana Pilson, and Stacey D. Smith. &quot;Filtering of target sequence capture individuals facilitates species tree construction in the plant subtribe Iochrominae (Solanaceae).&quot; Molecular phylogenetics and evolution 123 (2018): 26-34.

Gates, Daniel J., et al. &quot;A novel R3 MYB transcriptional repressor associated with the loss of floral pigmentation in Iochroma.&quot; New Phytologist 217.3 (2018): 1346-1356.  </code></pre>
<p><strong>Dianne Velasco</strong></p>
<pre><code>Riaz, Summaira, et al. &quot;Genetic diversity analysis of cultivated and wild grapevine (Vitis vinifera L.) accessions around the Mediterranean basin and Central Asia.&quot; BMC plant biology 18.1 (2018): 137.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-4" class="section level3">
<h3>2017</h3>
<details>
<summary><a href="https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix134/4782225?searchresult=1">Construction of the third generation <em>Zea mays</em> haplotype map</a> [<a href="http://biorxiv.org/content/early/2015/09/16/026963">preprint</a>] [<a href="http://gigadb.org/dataset/100339">sequence data</a>] [<a href="https://www.panzea.org/genotypes">genotype data</a>] [<a href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=23">alignment data</a>] [<a href="https://bitbucket.org/bukowski1/maize_hapmap3_code">code</a>]<br />
Bukowski R …[16 authors]… <strong>Ross-Ibarra J</strong>, <strong>Lorant A</strong>, <strong>Buffalo V</strong>, Romay MC, Buckler ES, Ware D, Lai J, Sun Q, Xu Y </summary>
<p style="margin-left: 30px">
We present a SNP dataset of 80M variants from a sample of 1200 maize and teosinte genomes.
</p>
</details>
<br>
<details>
<summary><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1346-4">The interplay of demography and selection during maize domestication and diffusion</a> [<a href="http://www.google.com/url?q=http%3A%2F%2Fbiorxiv.org%2Fcontent%2Fearly%2F2017%2F03%2F07%2F114579&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHr4cDzAxS_7BXVZOfyuIWzXiHr0w">preprint</a>][<a href="https://github.com/HuffordLab/Wang_et_al._Demography">github</a>][<a href="https://www.youtube.com/watch?v=KgIUGHEZ2nM">YouTube</a>]<br />
Wang L, <strong>Beissinger TM</strong>, <strong>Lorant A</strong>, <strong>Ross-Ibarra C</strong>, <strong>Ross-Ibarra J</strong>, Hufford MB.</summary>
<p style="margin-left: 30px">
We show how population bottlenecks and gene flow have shaped patterns of genetic load across the maize genome and among populatons.
</p>
</details>
<br>
<details>
<summary><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007019">Incomplete dominance of deleterious alleles contribute substantially to trait variation and heterosis in maize</a> [<a href="http://biorxiv.org/content/early/2016/11/07/086132">preprint</a>] [<a href="https://github.com/yangjl/GERP-diallel">github</a>]<br />
<strong>Yang J</strong>, <strong>Mezmouk S</strong>, Baumgarten A, Buckler ES, Guill KE, McMullen MD, Mumm RH, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We combine phenotypic data from crosses between multiple maize inbred lines and genome-wide characterization of deleterious alleles to show that a simple model of incomplete dominance may help explain hybrid vigor.
</p>
</details>
<br>
<details>
<summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184202">The potential role of genetic assimilation during maize domestication</a> [<a href="http://biorxiv.org/content/early/2017/02/03/105940">preprint</a>]<br />
<strong>Lorant A</strong>, Pedersen, S, Holst I, Hufford MB, Winter K, Piperno D, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Gene expression data from maize and teosinte grown in mid-Holocene environments suggests that for some genes plastic changes in expression may have preceded genetic changes during domestication.
</p>
</details>
<br>
<details>
<summary><a href="https://peerj.com/articles/3737/">Allele specific expression analysis identifies regulatory variation associated with stress-related genes in the Mexican highland maize landrace Palomero Toluqueño</a> [<a href="http://biorxiv.org/content/early/2017/06/20/152397">preprint</a>]<br />
Aguilar-Rangel MR, Chávez Montes RA, Gonzalez-Segovia E, <strong>Ross-Ibarra J</strong>, Simpson JK, Sawers RJH</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://science.sciencemag.org/content/357/6350/512.full">Genomic estimation of complex traits reveals ancient maize adaptation to temperate North America</a><br />
Swarts K, Gutaker RM …[<strong>Ross-Ibarra J</strong> and 14 authors in alphabetical order]… Krause J, Matson RG, Weigel D, Buckler ES, Burbano HA</summary>
<p style="margin-left: 30px">
SUMMARY <a href="http://www.nature.com/news/ancient-genomes-show-how-maize-adapted-to-life-at-high-altitudes-1.22403">Nature News</a> [<a href="https://phys.org/news/2017-08-precision-corn-climate.html">Phys.org</a>] [<a href="https://www.sciencedaily.com/releases/2017/08/170803141056.htm">Science Daily</a>]
</p>
</details>
<br>
<details>
<summary><a href="http://www.sciencedirect.com/science/article/pii/S0960982217307856">How to make a domesticate</a> [<a href="https://peerj.com/preprints/2902/">preprint</a>]<br />
<strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Mei W</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
A perspective that seeks to predict what factors influence the success of domestication, how many genes contributed to the process, where these genes originated and the implications for de novo domestication.
</p>
</details>
<br>
<details>
<summary><a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature22971.html">Improved maize reference genome with single-molecule technologies</a> [<a href="http://biorxiv.org/content/early/2016/12/19/079004">preprint</a>] [<a href="https://github.com/mcstitzer/maize_v4_TE_annotation">github</a>]<br />
Jiao Y, Peluso P, Shi J, Liang T, <strong>Stitzer MC</strong>, …[18 authors]… <strong>Ross-Ibarra J</strong>, Dawe K, Hastie A, Rank DR, Ware D</summary>
<p style="margin-left: 30px">
We present a high-quality long-read reference genome of the maize inbred B73 and the first detailed annotation of retroelements in any grass species. [<a href="https://news.science360.gov/obj/story/2caf076f-1759-40c2-9c56-62e3fbbdf44b/tracking-down-jumping-genes-maize">NSF research news</a>] [<a href="https://t.co/3MkC6eRDyD">UCD press release</a>]
</p>
</details>
<br>
<details>
<summary><a href="https://academic.oup.com/mbe/article/34/8/1825/3742530/Gene-Fractionation-and-Function-in-the-Ancient?guestAccessKey=391f8f04-5fd6-4bc7-bac0-0057fb3102ab">Gene fractionation and function in the ancient subgenomes of maize</a> [<a href="http://biorxiv.org/content/early/2016/12/19/095547">preprint</a>]<br />
<strong>Renny-Byfield S</strong>, Rodgers-Melnick E, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We show that the two subgenomes of maize differentially contribute to functional variation in phenotypes.
</p>
</details>
<br>
<details>
<summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177896">Genomic abundance is not predictive of tandem repeat localization in grass genomes</a> [<a href="https://peerj.com/preprints/2314/">preprint</a>] [<a href="https://github.com/paulbilinski/Github_centrepeat">github</a>]<br />
<strong>Bilinski P</strong>, Han Y, <strong>Hufford MB</strong>, <strong>Lorant A</strong>, Zhang P, Estep MC, Jiang J, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We show that short read sequence data allows discovery of novel tandem repeats in grass species but, unlike many other species, the most abundant repeat is frequently not centromeric.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, et al. &quot;Genetic and ecological insights into glacial refugia of walnut (Juglans regia L.).&quot; PloS one 12.10 (2017): e0185974.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B., et al. &quot;The relationship between selection, network connectivity, and regulatory variation within a population of Capsella grandiflora.&quot; Genome biology and evolution 9.4 (2017): 1099-1109.

Josephs, Emily B., John R. Stinchcombe, and Stephen I. Wright. &quot;What can genome‐wide association studies tell us about the evolutionary forces maintaining genetic variation for quantitative traits?.&quot; New Phytologist 214.1 (2017): 21-33.</code></pre>
<p><strong>Wenbin Mei</strong></p>
<pre><code>Mei, Wenbin, et al. &quot;A comprehensive analysis of alternative splicing in paleopolyploid maize.&quot; Frontiers in plant science 8 (2017): 694.

Liu, Xiaoxian, et al. &quot;Detecting alternatively spliced transcript isoforms from single‐molecule long‐read sequences without a reference genome.&quot; Molecular ecology resources 17.6 (2017): 1243-1256.

Gault, Christine M., et al. &quot;Aberrant splicing in maize rough endosperm3 reveals a conserved role for U12 splicing in eukaryotic multicellular development.&quot; Proceedings of the National Academy of Sciences (2017): 201616173.

Feng, Guanqiao, et al. &quot;Evolution of the 3R-MYB gene family in plants.&quot; Genome biology and evolution 9.4 (2017): 1013-1029.

Mei, Wenbin, et al. &quot;Evolutionarily conserved alternative splicing across monocots.&quot; Genetics (2017): genetics-300189.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-5" class="section level3">
<h3>2016</h3>
<details>
<summary><a href="http://www.cell.com/current-biology/abstract/S0960-9822(16)31120-4">Genome sequence of a 5310-year-old maize cob provides insights into the early stages of maize domestication</a><br />
Ramos-Madrigal J, Smith BD, Moreno-Mayar JV, Gopalakrishnan S, <strong>Ross-Ibarra J</strong>, Gilbert MTP, Wales N</summary>
<p style="margin-left: 30px">
Genome sequence of a 5,000 year old maize cob reveals it was partially domesticated and helps identify the timing of adaptation during domestication.
</p>
</details>
<br>
<details>
<summary><a href="http://www.g3journal.org/content/6/12/3985.short?rss=1&amp;ssource=mfr">Evolutionary genomics of peach and almond domestication</a> [<a href="http://biorxiv.org/content/early/2016/09/27/060160">preprint</a>]<br />
<strong>Velasco D</strong>, <strong>Hough J</strong>, Aradhya M, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Resequencing data from a number of peach and almond varieties show differences in population size and mating system, but little evidence for convergent selection during domestication.
</p>
</details>
<br>
<details>
<summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full">ANGSD-wrapper: utilities for analyzing next generation sequencing data</a> [<a href="https://github.com/mojaveazure/angsd-wrapper">github</a>] [<a href="https://peerj.com/preprints/1472/">preprint</a>]<br />
<strong>Durvasula A</strong>, Hoffman PJ, <strong>Kent TV</strong>, Liu C, Kono TJY, Morrell PL, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We provide a set of scripts and configuration files to facilitate easy popgen analysis of short-read sequencing data.
</p>
</details>
<br>
<details>
<summary><a href="http://rdcu.be/kHEB">Recent demography drives changes in linked selection across the maize genome</a> [<a href="https://github.com/timbeissinger/Maize-Teo-Scripts/">github</a>] [<a href="http://biorxiv.org/content/early/2015/11/13/031666">preprint</a>]<br />
<strong>Beissinger TM</strong>, Wang L, <strong>Crosby K</strong>, <strong>Durvasula A</strong>, Hufford MB, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We find little evidence for widespread hard sweeps in maize or teosinte and show that differences in demography during maize domestication and subsequent expansion have have changed the impacts of linked selection on surrounding diversity in the maize genome.
</p>
</details>
<br>
<details>
<summary><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy201610a.html">Maize diversity associated with social origin and environmental variation in southern Mexico</a> [<a href="https://peerj.com/preprints/1192/">preprint</a>]<br />
Orozco-Ramírez Q, Santacruz-Varela A, <strong>Ross-Ibarra J</strong>, Brush S </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://journal.frontiersin.org/article/10.3389/fpls.2016.00308/full">High quality maize centromere 10 sequence reveals evidence of frequent recombination events</a><br />
Wolfgruber TK, Nakashima MM …[5 authors]… <strong>Bilinski P</strong>, Dawe RK, <strong>Ross-Ibarra J</strong>, Birchler JA, Presting G </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Migicovsky, Zoë, et al. &quot;Genomic ancestry estimation quantifies use of wild species in grape breeding.&quot; BMC genomics 17.1 (2016): 478.</code></pre>
<p><strong>Emily Josephs</strong></p>
<pre><code>Josephs, Emily B., and Stephen I. Wright. &quot;On the Trail of Linked Selection.&quot; PLoS genetics 12.8 (2016): e1006240.</code></pre>
<p><strong>Tim Beissinger</strong></p>
<pre><code>Morota, Gota, Timothy M. Beissinger, and Francisco Peñagaricano. &quot;MeSH-informed enrichment analysis and MeSH-guided semantic similarity among functional terms and gene products in chicken.&quot; G3: Genes, Genomes, Genetics (2016): g3-116.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-6" class="section level3">
<h3>2015</h3>
<details>
<summary><a href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=2&amp;cad=rja&amp;uact=8&amp;ved=0CC0QFjABahUKEwiY8ezQoa3IAhWRL4gKHYbZCYU&amp;url=http%3A%2F%2Fwww.genetics.org%2Fcontent%2Fearly%2F2015%2F09%2F18%2Fgenetics.115.182410.abstract&amp;usg=AFQjCNF6HUbfJ3ij4yqX75WFIP7h0VqD8g&amp;sig2=7Gw1t6U1cc7imLVZEUWlHQ">The genomic impacts of drift and selection for hybrid performance in maize</a> [<a href="http://arxiv.org/abs/1307.7313">preprint</a>]<br />
Gerke JP, Edwards JW, Guill KE, <strong>Ross-Ibarra J</strong>, McMullen MD </summary>
<p style="margin-left: 30px">
We show how modern hybrid breeding approahces impact diversity across the genome. Using the Iowa Stiff Stalk synthetic population as an example, we track diversity across 16 generations of selection and compare with simulations. We show the vast majority of the changes in diversity observed can be explained by drift, but that fixation of large haplotypes, likley including considerable linked load, leads to selection for complementation in the recirpical populations. This work provided some of the first population genetic evidence supporting the complenmentaton model of hybrid vigor.
</p>
</details>
<br>
<details>
<summary><a href="https://www.nature.com/articles/ng.3422">Seed filling in domesticated maize and rice depends on SWEET-mediated hexose transport</a> [<a href="http://sci-hub.tw/https://www.nature.com/articles/ng.3422">pdf</a>]<br />
Sosso D, Luo D, Li Q-B, Sasse J, <strong>Yang J</strong> …[6 authors]… <strong>Ross-Ibarra J</strong>, Yang B, Frommer WB </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.genetics.org/content/200/4/1297">Independent molecular basis of convergent highland adaptation in maize</a> [<a href="http://biorxiv.org/content/early/2015/01/09/013607">preprint</a>] [<a href="https://github.com/rossibarra/hilo_paper">github</a>] [<a href="https://dx.doi.org/10.6084/m9.figshare.746990.v1">data</a>]<br />
<strong>Takuno S</strong>, Ralph P, Swarts K, Elshire RJ, Glaubitz JC, Buckler ES, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We find that maize adaptation to the highlands in Mexico and South America is largely independent and has made considerable use of standing genetic variation.
</p>
</details>
<br>
<details>
<summary><a href="https://peerj.com/articles/900/">Natural variation in teosinte at the domestication locus teosinte branched1 (tb1)</a> [<a href="https://peerj.com/preprints/685/">preprint</a>]<br />
<strong>Vann LE</strong>, <strong>Kono TJ</strong>, <strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>. </summary>
<p style="margin-left: 30px">
We find that variation in the domestication locus tb1 does not explain a substantial proportion of phenotypic variation for tillering in natural populations of teosinte.
</p>
</details>
<!--  [[github](https://github.com/lvann/tb1)] -->
<br>
<details>
<summary><a href="http://elifesciences.org/content/4/e05861">Genetic, evolutionary and plant breeding insights from the domestication of maize</a><br />
Hake S, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rdcu.be/kHD8">The origin and evolution of maize in the American Southwest</a> [<a href="http://biorxiv.org/content/early/2015/01/11/013540">preprint</a>] [<a href="ftp://ftp_20150114_27609:F+jYHhrE6t2e@ftp.dna.ku.dk">data</a>]<br />
Fonseca RR, Smith B, Wales N, Cappellini E …[12 authors]… Hufford MB, Albrechtsen A, <strong>Ross-Ibarra J</strong>, Gilbert MT </summary>
<p style="margin-left: 30px">
We use sequence data from ancient maize cobs to reveal the timing and geography of maize introduction into the Southwest US. One of the <a href="https://www.nature.com/collections/dnshkjhpxx/content/most-read">most read papers</a> from the last four years of the journal! <a href="http://news.ucdavis.edu/search/news_detail.lasso?id=11124">UC Davis Press Release</a> <a href="http://www.scienceworldreport.com/articles/21122/20150112/origins-maize-dna-tests-reveal-two-paths.htm">Science World Report</a> <a href="http://www.archaeology.org/news/2879-150108-southwest-corn-dna">Archaeology</a> <a href="http://www.agenciasinc.es/Noticias/El-maiz-entro-en-el-suroeste-de-EE-UU-hace-4.000-anos-a-traves-de-la-montana">SiNC</a>, <a href="http://news.sciencemag.org/archaeology/2015/01/how-corn-became-corn">Science magazine</a> <a href="http://www.nature.com/articles/nplants20149">Nature Plants commentary</a> <a href="http://www.the-scientist.com/?articles.view/articleNo/41865/title/Corn-Chronicle/">The Scientist</a>
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/dyer2015.pdf">Reply to Brush <em>et al.</em>: Wake-up call for crop conservation science</a><br />
Dyer GA, Lopez-Feldman A, Yunez-Naude A, Taylor JE, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004915">Transposable Elements Contribute to Activation of Maize Genes in Response to Abiotic Stress</a> [<a href="http://biorxiv.org/content/early/2014/08/15/008052">preprint</a>]<br />
Makarevitch I, Waters M, West P, <strong>Stitzer M</strong>, <strong>Ross-Ibarra J</strong>, Springer NM</summary>
<p style="margin-left: 30px">
We show that transposable elements can act as regulatory sequences that up- or down-regulate genes in maize in response to abiotic stress.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, J. E. Preece, and Dianne Velasco. &quot;Multivariate analysis of molecular and morphological diversity in fig (Ficus carica L.).&quot; V International Symposium on Fig 1173. 2015.</code></pre>
<p><strong>Paul Bilinski</strong></p>
<pre><code>Leung, Wilson, et al. &quot;Drosophila Muller F elements maintain a distinct set of genomic properties over 40 million years of evolution.&quot; G3: Genes, Genomes, Genetics 5.5 (2015): 719-740.</code></pre>
<p><strong>Anna O’Brien</strong></p>
<pre><code>Persson, Tomas, et al. &quot;Candidatus Frankia datiscae Dg1, the actinobacterial microsymbiont of Datisca glomerata, expresses the canonical nod genes nodABC in symbiosis with its host plant.&quot; PLoS One 10.5 (2015): e0127630.</code></pre>
<p><strong>Shohei Takuno</strong></p>
<pre><code>Le, Tu N., et al. &quot;Epigenetic regulation of intragenic transposable elements impacts gene transcription in Arabidopsis thaliana.&quot; Nucleic acids research 43.8 (2015): 3911-3921.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-7" class="section level3">
<h3>2014</h3>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/tiffin2014.pdf">Advances and limits of using population genetics to understand local adaptation</a> [<a href="https://peerj.com/preprints/488/">preprint</a>] [<a href="https://www.cell.com/trends/ecology-evolution/fulltext/S0169-5347(17)30188-X">correction</a>]<br />
Tiffin P, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We discuss some of the outstanding questions and key challenges facing the use of population genetics to study local adaptation.
</p>
</details>
<br>
<details>
<summary> <a href="http://rilab.ucdavis.edu/pdfs/Bilinski2014.pdf">Diversity and evolution of centromere repeats in the maize genome</a> [<a href="http://biorxiv.org/content/early/2014/05/12/005058">preprint</a>]<br />
<strong>Bilinski P</strong>, <strong>Distor KD</strong>, <strong>Gutierez-Lopez J</strong>, <strong>Mendoza Mendoza G</strong>, Shi J, Dawe K, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.g3journal.org/content/4/1/163.full">The pattern and distribution of deleterious mutations in maize</a> [<a href="http://arxiv.org/abs/1308.0380">preprint</a>] [<a href="https://github.com/RILAB/siftmappR">github</a>]<br />
<strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that deleterious mutations are abundant in modern maize breeding material and that genes with deleterious mutations are likely important contributors to phenotypic variation.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
</p>
</details>
<p><br></p>
</div>
<div id="section-8" class="section level3">
<h3>2013</h3>
<details>
<summary><a href="http://www.pnas.org/content/110/48/19639.abstract">Comprehensive analysis of imprinted genes in maize reveals allelic variation and limited conservation with other species</a> [<a href="http://arxiv.org/abs/1307.7678">preprint</a>] [<a href="https://github.com/paulbilinski/Waters_etal_SupplementalTableAnalysis">github</a>]<br />
Waters AJ, <strong>Bilinski P</strong>, Eichten SR, Vaugh MW, <strong>Ross-Ibarra J</strong>, Gehring M, Springer NM </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://gbe.oxfordjournals.org/content/5/9/1594">Complex patterns of local adaptation in teosinte</a> [<a href="http://datadryad.org/resource/doi:10.5061/dryad.8m648/1">data</a>] [<a href="http://arxiv.org/abs/1208.0634">preprint</a>]<br />
<strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003604">From many, one: genetic control of prolificacy during maize domestication</a> [<a href="http://figshare.com/articles/Wills_et_al_2013/775358">data</a>] [<a href="http://arxiv.org/abs/1303.0882">preprint</a>]<br />
Wills DM, Whipple C, <strong>Takuno S</strong>, Kursel LE, Shannon LM, <strong>Ross-Ibarra J</strong>, Doebley JF. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/McCouch2013.pdf">Agriculture: Feeding the future</a><br />
McCouch S, Baute GJ, Bradeen J, Bramel P …[36 authors]… Ward J, Waugh R, Wenzl P, Zamir. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003477">The genomic signature of crop-wild introgression in maize</a> [<a href="http://figshare.com/articles/Hufford_et_al_2013/766356">data</a>] [<a href="http://arxiv.org/abs/1208.3894">preprint</a>]<br />
<strong>Hufford MB</strong>, Lubinsky P, <strong>Pyhäjärvi T</strong>, <strong>Devengenzo MT</strong>, Ellstrand NC, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/provance2013.pdf">Population genetics and ethnobotany of cultivated <em>Diospyros riojae</em> (Ebenaceae), an endangered fruit crop from Mexico</a> [<a href="http://figshare.com/articles/Population_Genetics_and_Ethnobotany_of_Cultivated_Diospyros_riojae_G_mez_Pompa_Ebenaceae_an_Endangered_Fruit_Crop_from_Mexico/105841">preprint</a>]<br />
<strong>Provance MC</strong>, Garcia Ruiz I, <strong>Thommes C</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://genomebiology.com/2013/14/1/R10">Patterns of centromere tandem repeat evolution in 282 animal and plant genomes</a> [<a href="http://arxiv.org/abs/1209.4967">preprint</a>]<br />
Melters DP, Bradnam KR, Young HA, Telis N …[9 authors]… Tobias C, <strong>Ross-Ibarra J</strong>, Korf IF, Chan SW-L. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy20132a.html">Diversity and abundance of the Abnormal chromosome 10 meiotic drive complex in <em>Zea mays</em></a> [<a href="http://arxiv.org/abs/1209.5466">preprint</a>]<br />
Kanizay LB, <strong>Pyhäjärvi T</strong>, Lowry E, <strong>Hufford MB</strong>, Peterson DG, <strong>Ross-Ibarra J</strong>, Dawe RK </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Dianne Velasco</strong></p>
<pre><code>Aradhya, Mallikarjuna, et al. &quot;Genetic diversity, structure, and patterns of differentiation in the genus Vitis.&quot; Plant systematics and evolution 299.2 (2013): 317-330.

Zdunić, Goran, et al. &quot;Genetic diversity and differentiation within and between cultivated (Vitis vinifera L. ssp. sativa) and wild (Vitis vinifera L. ssp. sylvestris) grapes.&quot; Vitis 52.1 (2013): 29-32.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-9" class="section level3">
<h3>2012</h3>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/hufford2012.pdf">Teosinte as a model system for population and ecological genomics</a><br />
<strong>Hufford MB</strong>, <strong>Bilinski P</strong>, <strong>Pyhäjärvi T</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_24240aa930bf4728be83b4b57da55f17.pdf">Comparative population genomics of maize domestication and improvement</a> [<a href="http://figshare.com/articles/new_fileset/757736">data</a>]<br />
<strong>Hufford MB</strong>, Xun X, <strong>van Heerwaarden J</strong>, <strong>Pyhäjärvi T</strong> …[17 authors]… Ware D, Buckler ES, Yang S, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Munoz_Diez_R1-1.pdf">Using nextgen sequencing to investigate genome size variation and transposable element content</a><br />
Munoz Diez C, Vitte C, <strong>Ross-Ibarra J</strong>, Gaut BS, Tenaillon MI </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/vanheerwaarden2012.pdf">Historical genomics of North American maize</a> [<a href="http://figshare.com/articles/van_Heerwaarden_et_al_2012/757738">data</a>]<br />
<strong>van Heerwaarden J</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/swansonwagner2012.pdf">Reshaping of the maize transcriptome by domestication</a><br />
Swanson-Wagner R, Briskine R, Schaefer R, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>, Myers CL, Tiffin P, Springer NM </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_86a609db506740ce96c549567c246b4c.pdf">Maize HapMap2 identifies extant variation from a genome in flux</a><br />
Chia J-M, Song C, …[<strong>Hufford, MB</strong>, <strong>Pyhäjärvi T</strong>, and 27 authors]… <strong>Ross-Ibarra, J</strong>, McMullen MD, Buckler ES, Zhang G, Xu Y, Ware, D </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.genetics.org/content/191/3/883.abstract">Megabase-scale inversion polymorphism in the wild ancestor of maize</a> [<a href="http://figshare.com/articles/Fang_et_al_2012/840519">data</a>]<br />
Fang Z, <strong>Pyhäjärvi T</strong>, Weber AL, Dawe RK, …[2 authors]… <strong>Ross-Ibarra C</strong>, Doebley J, Morrell PL, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We identify and study the evolutionary genetics of a 50Mb inversion in natural populations of teosinte
</p>
</details>
<br>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_5ca9c5a7f6174a5bb0c85e67dec67dcc.pdf">Genetic architecture of maize kernel composition in the Nested Association Mapping and Inbred Association panels</a><br />
Cook JP, McMullen MD, Holland JB, Tian F, Bradbury P, <strong>Ross-Ibarra J</strong>, Buckler ES, Flint-Garcia SA </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary> <a href="http://media.wix.com/ugd/fe9228_a2a75f7435ec4c569d4a6996e8b5451e.pdf">Crop genomics: advances and applications</a><br />
Morrell PL, Buckler ES, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Tanja Pyhäjärvi</strong></p>
<pre><code>Krutovsky, Konstantin V., et al. &quot;Gene flow, spatial structure, local adaptation, and assisted migration in trees.&quot; Genomics of tree crops. Springer, New York, NY, 2012. 71-116.

Pyhäjärvi, Tanja, Esa Aalto, and Outi Savolainen. &quot;Time scales of divergence and speciation among natural populations and subspecies of Arabidopsis lyrata (Brassicaceae).&quot; American journal of botany 99.8 (2012): 1314-1322.</code></pre>
<p><strong>Matthew Hufford</strong></p>
<pre><code>Hufford, Matthew B., et al. &quot;Inferences from the historical distribution of wild and domesticated maize provide ecological and evolutionary insight.&quot; PLoS One 7.11 (2012): e47659.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-10" class="section level3">
<h3>2011</h3>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_14630f30e97f4e04b8b5980632002f7a.pdf">Identification of a functional transposon insertion in the maize domestication gene <em>tb1</em></a> [<a href="http://www.panzea.org/db/gateway?file_id=Studer_etal_2011_Nat_Genet_data">data</a>]<br />
Studer A, Zhao Q, <strong>Ross-Ibarra J</strong>, Doebley J </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_4916e7f7acec4aee869da040d519e0be.pdf">Genetic signals of origin, spread and introgression in a large sample of maize landraces</a> [<a href="http://figshare.com/articles/new_fileset/757735">data</a>]<br />
<strong>van Heerwaarden J</strong>, Doebley J, Briggs WH, Glaubitz JC, Goodman MM, Sanchez Gonzalez JJ, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We resolve the conflict between genetic, ecological, and achaeological data regarding the geographic location of maize domestication.
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/hufford2011.pdf">Influence of cryptic population structure on observed mating patterns in the wild progenitor of maize</a> [<a href="10.6084/m9.figshare.757734">data</a>]<br />
<strong>Hufford MB</strong>, Gepts P, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="https://www.ncbi.nlm.nih.gov/pubmed/21296765">Genome size and TE content as determined by high-throughput sequencing in maize and <em>Zea luxurians</em></a><br />
Tenaillon MI, <strong>Hufford MB</strong>, Gaut BS, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Tanja Pyhäjärvi</strong></p>
<pre><code>Savolainen, Outi, et al. &quot;Adaptive potential of northernmost tree populations to climate change, with emphasis on Scots pine (Pinus sylvestris L.).&quot; Journal of Heredity 102.5 (2011): 526-536.

Pyhäjärvi, Tanja, Sonja T. Kujala, and Outi Savolainen. &quot;Revisiting protein heterozygosity in plants—nucleotide diversity in allozyme coding genes of conifer Pinus sylvestris.&quot; Tree genetics &amp; genomes 7.2 (2011): 385-397.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-11" class="section level3">
<h3>2010</h3>
<details>
<summary><a href="http://www.genetics.org/content/185/3/969.abstract">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em>)</a><br />
Eckert AJ, <strong>van Heerwaarden J</strong>, Wegrzyn JL, Nelson CD, <strong>Ross-Ibarra J</strong>, Gonzalez-Martinez SC, and Neale DB </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary> <a href="http://rilab.ucdavis.edu/pdfs/Fuchs-et-al-2010.pdf">Reproductive biology of <strong>Macleania rupestris</strong> (Ericaceae): a pollen-limited Neotropical cloud-forest species in Costa Rica</a><br />
Fuchs EJ, <strong>Ross-Ibarra J</strong>, Barrantes G </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Whitney-et-al-2010-402.pdf">A role for nonadaptive processes in plant genome size evolution?</a><br />
Whitney KD, Baack EJ, Hamrick JL, Godt, MJW …[3 authors]… Goodwillie C, Kalisz S, Leitch I, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Van-Heerwaarden-et-al-2010-951.pdf">Fine scale genetic structure in the wild ancestor of maize (<strong>Zea mays</strong> ssp. <strong>parviglumis</strong>)</a><br />
<strong>van Heerwaarden J</strong>, <strong>Ross-Ibarra J</strong>, Doebley J, Glaubitz JC, Sanchez Gonzalez J, Gaut BS, Eguiarte LE</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000327">Widespread gene conversion in centromere cores</a><br />
Shi J, Wolf S, Burke J, Presting G, <strong>Ross-Ibarra J</strong>, Dawe RK </summary>
<p style="margin-left: 30px">
[<a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000326">plos biology commentary</a>]
</p>
</details>
<br>
<details>
<summary><a href="http://mbe.oxfordjournals.org/content/27/2/409.abstract">Indel-associated mutation rate varies with mating system in flowering plants</a><br />
Hollister JD, <strong>Ross-Ibarra J</strong>, Gaut BS </summary>
<p style="margin-left: 30px">
We show that mating system differences in heterozygosity mitigate the effects of insertion and deletion polymorphisms on nucleotide diversity.
</p>
</details>
<br>
<details>
<summary> <a href="https://pubmed.ncbi.nlm.nih.gov/19738636/">Genetic diversity in a crop metapopulation</a><br />
<strong>van Heerwaarden J</strong>, van Eeuwijk FA, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
The need to protect crop genetic resources has sparked a growing interest in the genetic diversity maintained in traditional farming systems worldwide. Although traditional seed management has been proposed as an important determinant of genetic diversity and structure in crops, no models exist that can adequately describe the genetic effects of seed management. We present a metapopulation model that accounts for several features unique to managed crop populations. Using traditional maize agriculture as an example, we develop a coalescence-based model of a crop metapopulation undergoing pollen and seed flow as well as seed replacement. In contrast to metapopulation work on natural systems, we model seed migration as episodic and originating from a single source per population rather than as a constant immigration from the entire metapopulation. We find that the correlated origin of migrants leads to surprising results, including a loss of invariance of within-deme diversity and a parabolic relationship between F(ST) and migration quantity. In contrast, the effects of migration frequency on diversity and structure are more similar to classical predictions, suggesting that seed migration in managed crop populations cannot be described by a single parameter. In addition to migration, we investigate the effects of deme size and extinction rates on genetic structure, and show that high levels of pollen migration may mask the effects of seed management on structure. Our results highlight the importance of analytically evaluating the effects of deviations from classical metapopulation models, especially in systems for which data are available to estimate specific model parameters.
</p>
</details>
<br>
<details>
<summary><strong>Other papers by lab members</strong></summary>
<p style="margin-left: 30px">
<p><strong>Matthew Hufford</strong></p>
<pre><code>Epanchin-Niell, Rebecca S., et al. &quot;Controlling invasive species in complex social landscapes.&quot; Frontiers in Ecology and the Environment 8.4 (2010): 210-216.</code></pre>
</p>
</details>
<p><br></p>
</div>
<div id="section-12" class="section level3">
<h3>2009</h3>
<details>
<summary><a href="http://media.wix.com/ugd/fe9228_352f4ee9b69a425589e55a4bac60bc5d.pdf">A first-generation haplotype map of maize</a><br />
Gore MA, Chia JM, Elshire RJ, Sun Q …[4 authors]… Grills GS, <strong>Ross-Ibarra J</strong>, Ware DH, Buckler ES </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008346">A pleistocene clone of Palmer’s Oak persisting in Southern California</a><br />
<strong>May MR</strong>, <strong>Provance MC</strong>, Sanders AC, Ellstrand NC, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We discover a 13,000 year old oak clone in the hills of Southern California
</p>
</details>
<br>
<details>
<summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.02984.x/epdf">Selection on grain shattering genes and rates of rice domestication</a><br />
Zhang LB, Zhu Q, Wu ZQ, <strong>Ross-Ibarra J</strong>, Gaut BS, Ge S, Sang T </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.genetics.org/content/181/4/1399.abstract">Historical divergence and gene flow in the genus Zea</a><br />
<strong>Ross-Ibarra J</strong>, Tenaillon M, Gaut BS </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<p><br></p>
</div>
<div id="section-13" class="section level3">
<h3>2008</h3>
<details>
<summary><a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002411">Patterns of polymorphism and demographic history in natural populations of <em>Arabidopsis lyrata</em></a><br />
<strong>Ross-Ibarra J</strong>, Wright SI, Foxe JP, Kawabe A, DeRose-Wilson L, Gos G, Charlesworth D, Gaut BS </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.pnas.org/content/105/37/13965.short">Demography and weak selection drive patterns of TE diversity in natural populations of <em>A. lyrata</em></a><br />
Lockton S, <strong>Ross-Ibarra J</strong>, Gaut BS</summary>
<p style="margin-left: 30px">
We show that differences in demographic history among populations of have changed the abundance and freqeuncy of transposable elements in the <em>A lyrata</em> genome.
</p>
</details>
<br>
<details>
<summary><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614810/">Multiple domestications do not appear monophyletic</a><br />
<strong>Ross-Ibarra J</strong>, Gaut BS</summary>
<p style="margin-left: 30px">
A rebuttal showing that genetic data can indeed successfully recover multiple independent domestications.
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra_Gaut_2008.pdf">Selection on major components of angiosperm genomes</a><br />
Gaut BS, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Discusses some of the evolutionary forces responsible for shaping genome size across plants.
</p>
</details>
<p><br></p>
</div>
<div id="section-14" class="section level3">
<h3>2007</h3>
<details>
<summary><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1876441/">Plant domestication, a unique opportunity to identify the genetic basis of adaptation</a><br />
<strong>Ross-Ibarra J</strong>, Morrell PL, Gaut BS</summary>
<p style="margin-left: 30px">
Presents plant domestication as an ideal system in which to study the genetics of adaptation.
</p>
</details>
<br>
<details>
<summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2006.01275.x/abstract">Genome size and recombination in angiosperms: a second look</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<p><br></p>
</div>
<div id="section-15" class="section level3">
<h3>2001-2006</h3>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Wares-et-al-2006.pdf">Mitochondrial DNA and population size</a><br />
Wares JP, Barber PH, <strong>Ross-Ibarra J</strong>, Sotka EE, Toonen RJ </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2005-15881692.pdf">Quantitative trait loci and the study of plant domestication</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2004.pdf">The evolution of recombination under domestication: a test of two hypotheses</a> [<a href="http://rilab.ucdavis.edu/pubs/supp_data_rossibarra2004.tgz">data</a>]<br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
Here I test two different hypotheses about the how recombination should relate to domestication: Does domestication itself select for increased recombination (yes), or respecies with higher recombination rates more likely to be domesticated (no).
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra-2003.pdf">Origen y domesticación de la chaya (<em>Cnidoscolus aconitifolius</em> Mill IM Johnst): La espinaca Maya</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
I use genetic data to study the origin of a leafy shrub domesticated by the Maya
</p>
</details>
<br>
<details>
<summary><a href="http://rilab.ucdavis.edu/pdfs/Ross-Ibarra_Molina-Cruz-2002.pdf">The ethnobotany of Chaya (<em>Cnidoscolus aconitifolius</em> ssp. <em>aconitifolius</em> Breckon): A nutritious Maya vegetable</a><br />
<strong>Ross-Ibarra J</strong>, Molina-Cruz A </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
<br>
<details>
<summary><a href="http://www.amjbot.org/content/88/7/1214.abstract">Implications of mating patterns for conservation of the endangered plant <em>Eriogonum ovalifolium</em> var. <em>vineum</em></a><br />
Neel MC, <strong>Ross-Ibarra J</strong>, Ellstrand NC</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
</details>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
