<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->





<script>
  window.onload = choosePic;
  var myPix = new Array("images/image1.jpg","images/image2.jpg","images/image3.jpg","images/image4.jpg","images/image5.jpg","images/image6.jpg","images/image7.jpg","images/image8.jpg","images/image8.jpg","images/image10.jpg");
  function choosePic() {
     var randomNum = Math.floor(Math.random() * myPix.length);
     document.getElementById("myPicture").src = myPix[randomNum];
  }
</script>

<div class="navbar navbar-default  navbar-float-top" role="navigation">
    <div class="container" overflow="auto" align="left" style="width: inherit;">
      <img id="myPicture" alt="Brand" src="" align='center'>
    </div>

  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <img alt="Brand" src="images/cornkey_icon.png" align='left'>
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>

      </button>

      <a class="navbar-brand" href="index.html">Ross-Ibarra Lab</a>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav" align="center">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs/wiki">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
        <li><a href="press.html">Press</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<div id="publications" class="section level2">
<h2>Publications <a href="http://scholar.google.com/citations?user=5SzRq1oAAAAJ"><img src="images/scholar.png" style="width: 50px;"></a></h2>
<SCRIPT language="JavaScript" SRC='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></SCRIPT>
<p>Journal names have been intentionally excluded. Lab member names are in bold. Links are provided to the published article and when possible an open-access version (pre- or post-print) of the paper. Hover over article links for a brief synopsis of the paper.</p>
<p>Click on a triangle for a brief summary of each paper.</p>
<div id="preprints" class="section level3">
<h3>Preprints</h3>
<!-- <details> -->
<!--     <summary>Variable adaptation between teosinte and soil biota across climate -->
<!--  [[preprint](https://www.biorxiv.org/content/early/2018/08/01/382770)]     -->
<!-- **O'Brien AM**, Sawers RJH, Baxter I, **Ross-Ibarra J**, Strauss SY -->
<!-- </summary> -->
<!--     <p style="margin-left: 30px">    -->
<!-- </p></details>  -->
<details> <summary>Adaptive phenotypic divergence in teosinte differs across biotic contexts [<a href="https://www.biorxiv.org/content/early/2018/08/01/382770">preprint</a>)] <strong>O’Brien AM</strong>, Sawers RJH, Strauss SY, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that rhizosphere biota impact adaptive divergence of phenotypic traits in teosinte along an environmental cline. This work suggests an important role for biotic interactions in determining the outcome of local adaptation.
</p>
<p></details></p>
<details> <summary>Detecting adaptive differentiation in structured populations with genomic data and common gardens [<a href="https://www.biorxiv.org/content/early/2018/07/13/368506">preprint</a>] [<a href="https://github.com/emjosephs/qpc-maize">github</a>]<br />
<strong>Josephs EB</strong>, Berg JJ, <strong>Ross-Ibarra J</strong>, Coop G </summary>
<p style="margin-left: 30px">
We develop methods to detect polygenic adaptation on phenotypes from common gardens and GWAS panels.
</p>
<p></details></p>
<details> <summary>Genetic architecture and selective sweeps after polygenic adaptation to distant trait optima [<a href="https://www.biorxiv.org/content/early/2018/05/03/313247">preprint</a>] [<a href="https://mgstetter.shinyapps.io/quantgensimAPP/">Shiny app</a>]<br />
<strong>Stetter MG</strong>, Thornton K, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We use simulations to study the population genetics of selection on a quantitative trait and learn which parameters drive quantitative trait evollution.
</p>
<p></details></p>
</div>
<div id="section" class="section level3">
<h3>2018</h3>
<details> <summary>Evolutionary responses to conditionality in species interactions across environmental gradients [<a href="http://biorxiv.org/content/early/2017/03/01/031195">preprint</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, <strong>Ross-Ibarra J</strong>, Strauss SY </summary>
<p style="margin-left: 30px">
We present the CoCoA hypothesis which describes how patterns of biotic coadpation change across varying abiotic environments
</p>
<p></details></p>
<details> <summary>Maize domestication and gene interaction [<a href="https://peerj.com/preprints/26502/">preprint</a>]<br />
<strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We review genetic work looking at the role of dominance, epistasis, and pleiotropy during maize domestication.
</p>
<p></details></p>
<details> <summary><a href="http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">A novel maize kinesin causes neocentromere activity and meiotic drive, altering inheritance patterns across the genome</a> [<a href="http://sci-hub.hk/http://www.cell.com/cell/fulltext/S0092-8674(18)30289-7">pdf</a>] [<a href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=1&amp;cad=rja&amp;uact=8&amp;ved=0ahUKEwiIk5W14eHSAhVO92MKHbseACgQtwIIHDAA&amp;url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DbcCxsVPwOE8&amp;usg=AFQjCNGJ2xxs5xSbskrcavIoJ0D8ve3fjA&amp;sig2=gmS2FYQxQlrQJRw8PGPDFw">YouTube</a>]<br />
Dawe RK, Lowry EG, Gent JI, <strong>Stitzer MC</strong>, Higgins DM, <strong>Ross-Ibarra J</strong>, Wallace J, Kanizay LB … [7 authors]</summary>
<p style="margin-left: 30px">
We identify a small gene family Kindr as the causal locus for meiotic drive in maize.
</p>
<p></details></p>
<details> <summary><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162">Parallel altitudinal clines reveal adaptive evolution of genome size in <em>Zea mays</em></a> [<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007249">perspective</a>] [<a href="http://www.biorxiv.org/content/early/2017/07/13/134528">preprint</a>] [<a href="https://github.com/paulbilinski/GenomeSizeAnalysis">github</a>] [<a href="https://www.slideshare.net/jrossibarra/parallel-altitudinal-clines-reveal-adaptive-evolution-of-genome-size-in-zea-mays">slides</a>]<br />
<strong>Bilinski P</strong>, Albert P, Berg JJ, Birchler JA, Grote M, <strong>Lorant A</strong>, <strong>Quezada J</strong>, Swarts K, <strong>Yang J</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that genome size can be analyzed as a quantitative trait and that selection on flowering time in maize has likely driven adaptive changes in genome size along multiple altitudinal clines.
</p>
<p></details></p>
<details> <summary><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajb2.1002">Adaptation in plant genomes: bigger is different</a> [<a href="https://www.biorxiv.org/content/early/2017/10/02/196501">preprint</a>][<a href="https://github.com/RILAB/AJB_MutationalTargetSize_GenomeSize">github</a>][<a href="https://www.youtube.com/watch?v=aXLuWruOmO4_">YouTube</a>][<a href="https://www.slideshare.net/jrossibarra/adaptation-in-plant-genomes-bigger-is-different">slides</a>]<br />
<strong>Mei W</strong>, <strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We develop the functional space hypothesis in which we posit that adaptation in large genomes uses more noncoding variation and is more likely to proceed via soft sweeps.
</p>
<p></details></p>
<details> <summary><a href="https://www.sciencedirect.com/science/article/pii/S1462901117310602">Harnessing cross-border resources to confront climate change</a><br />
Aburto-Oropeza O …[<strong>Ross-Ibarra J</strong> and 85 authors in alphabetical order]… Taylor JE </summary>
<p style="margin-left: 30px">
Discusses the challenges of changing climates pose for society and the environment along the Mexican-US border.
</p>
<p></details></p>
</div>
<div id="section-1" class="section level3">
<h3>2017</h3>
<details> <summary><a href="https://academic.oup.com/gigascience/advance-article/doi/10.1093/gigascience/gix134/4782225?searchresult=1">Construction of the third generation <em>Zea mays</em> haplotype map</a> [<a href="http://biorxiv.org/content/early/2015/09/16/026963">preprint</a>] [<a href="http://gigadb.org/dataset/100339">sequence data</a>] [<a href="https://www.panzea.org/genotypes">genotype data</a>] [<a href="http://cbsusrv04.tc.cornell.edu/users/panzea/download.aspx?filegroupid=23">alignment data</a>] [<a href="https://bitbucket.org/bukowski1/maize_hapmap3_code">code</a>]<br />
Bukowski R …[16 authors]… <strong>Ross-Ibarra J</strong>, <strong>Lorant A</strong>, <strong>Buffalo V</strong>, Romay MC, Buckler ES, Ware D, Lai J, Sun Q, Xu Y </summary>
<p style="margin-left: 30px">
We present a SNP dataset of 80M variants from a sample of 1200 maize and teosinte genomes.
</p>
<p></details></p>
<details> <summary><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1346-4">The interplay of demography and selection during maize domestication and diffusion</a> [<a href="http://www.google.com/url?q=http%3A%2F%2Fbiorxiv.org%2Fcontent%2Fearly%2F2017%2F03%2F07%2F114579&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHr4cDzAxS_7BXVZOfyuIWzXiHr0w">preprint</a>][<a href="https://github.com/HuffordLab/Wang_et_al._Demography">github</a>][<a href="https://www.youtube.com/watch?v=KgIUGHEZ2nM">YouTube</a>]<br />
Wang L, <strong>Beissinger TM</strong>, <strong>Lorant A</strong>, <strong>Ross-Ibarra C</strong>, <strong>Ross-Ibarra J</strong>, Hufford MB.</summary>
<p style="margin-left: 30px">
We show how population bottlenecks and gene flow have shaped patterns of genetic load across the maize genome and among populatons.
</p>
<p></details></p>
<details> <summary><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007019">Incomplete dominance of deleterious alleles contribute substantially to trait variation and heterosis in maize</a> [<a href="http://biorxiv.org/content/early/2016/11/07/086132">preprint</a>] [<a href="https://github.com/yangjl/GERP-diallel">github</a>]<br />
<strong>Yang J</strong>, <strong>Mezmouk S</strong>, Baumgarten A, Buckler ES, Guill KE, McMullen MD, Mumm RH, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We combine phenotypic data from crosses between multiple maize inbred lines and genome-wide characterization of deleterious alleles to show that a simple model of incomplete dominance may help explain hybrid vigor.
</p>
<p></details></p>
<details> <summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184202">The potential role of genetic assimilation during maize domestication</a> [<a href="http://biorxiv.org/content/early/2017/02/03/105940">preprint</a>]<br />
<strong>Lorant A</strong>, Pedersen, S, Holst I, Hufford MB, Winter K, Piperno D, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Gene expression data from maize and teosinte grown in mid-Holocene environments suggests that for some genes plastic changes in expression may have preceded genetic changes during domestication.
</p>
<p></details></p>
<details> <summary><a href="https://peerj.com/articles/3737/">Allele specific expression analysis identifies regulatory variation associated with stress-related genes in the Mexican highland maize landrace Palomero Toluqueño</a> [<a href="http://biorxiv.org/content/early/2017/06/20/152397">preprint</a>]<br />
Aguilar-Rangel MR, Chávez Montes RA, Gonzalez-Segovia E, <strong>Ross-Ibarra J</strong>, Simpson JK, Sawers RJH</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://science.sciencemag.org/content/357/6350/512.full">Genomic estimation of complex traits reveals ancient maize adaptation to temperate North America</a><br />
Swarts K, Gutaker RM …[<strong>Ross-Ibarra J</strong> and 14 authors in alphabetical order]… Krause J, Matson RG, Weigel D, Buckler ES, Burbano HA</summary>
<p style="margin-left: 30px">
SUMMARY <a href="http://www.nature.com/news/ancient-genomes-show-how-maize-adapted-to-life-at-high-altitudes-1.22403">Nature News</a> <a href="https://phys.org/news/2017-08-precision-corn-climate.html">Phys.org</a> <a href="https://www.sciencedaily.com/releases/2017/08/170803141056.htm">Science Daily</a>
</p>
<p></details></p>
<details> <summary><a href="http://www.sciencedirect.com/science/article/pii/S0960982217307856">How to make a domesticate</a> [<a href="https://peerj.com/preprints/2902/">preprint</a>]<br />
<strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Mei W</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
A perspective that seeks to predict what factors influence the success of domestication, how many genes contributed to the process, where these genes originated and the implications for de novo domestication.
</p>
<p></details></p>
<details> <summary><a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature22971.html">Improved maize reference genome with single-molecule technologies</a> [<a href="http://biorxiv.org/content/early/2016/12/19/079004">preprint</a>] [<a href="https://github.com/mcstitzer/maize_v4_TE_annotation">github</a>]<br />
Jiao Y, Peluso P, Shi J, Liang T, <strong>Stitzer MC</strong>, …[18 authors]… <strong>Ross-Ibarra J</strong>, Dawe K, Hastie A, Rank DR, Ware D</summary>
<p style="margin-left: 30px">
We present a high-quality long-read reference genome of the maize inbred B73 and the first detailed annotation of retroelements in any grass species. [<a href="https://news.science360.gov/obj/story/2caf076f-1759-40c2-9c56-62e3fbbdf44b/tracking-down-jumping-genes-maize">NSF research news</a>] [<a href="https://t.co/3MkC6eRDyD">UCD press release</a>]
</p>
<p></details></p>
<details> <summary><a href="https://academic.oup.com/mbe/article/34/8/1825/3742530/Gene-Fractionation-and-Function-in-the-Ancient?guestAccessKey=391f8f04-5fd6-4bc7-bac0-0057fb3102ab">Gene fractionation and function in the ancient subgenomes of maize</a> [<a href="http://biorxiv.org/content/early/2016/12/19/095547">preprint</a>]<br />
<strong>Renny-Byfield S</strong>, Rodgers-Melnick E, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We show that the two subgenomes of maize differentially contribute to functional variation in phenotypes.
</p>
<p></details></p>
<details> <summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177896">Genomic abundance is not predictive of tandem repeat localization in grass genomes</a> [<a href="https://peerj.com/preprints/2314/">preprint</a>] [<a href="https://github.com/paulbilinski/Github_centrepeat">github</a>]<br />
<strong>Bilinski P</strong>, Han Y, <strong>Hufford MB</strong>, <strong>Lorant A</strong>, Zhang P, Estep MC, Jiang J, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We show that short read sequence data allows discovery of novel tandem repeats in grass species but, unlike many other species, the most abundant repeat is frequently not centromeric.
</p>
<p></details></p>
</div>
<div id="section-2" class="section level3">
<h3>2016</h3>
<details> <summary><a href="http://www.cell.com/current-biology/abstract/S0960-9822(16)31120-4">Genome sequence of a 5310-year-old maize cob provides insights into the early stages of maize domestication</a><br />
Ramos-Madrigal J, Smith BD, Moreno-Mayar JV, Gopalakrishnan S, <strong>Ross-Ibarra J</strong>, Gilbert MTP, Wales N</summary>
<p style="margin-left: 30px">
Genome sequence of a 5,000 year old maize cob reveals it was partially domesticated and helps identify the timing of adaptation during domestication.
</p>
<p></details></p>
<details> <summary><a href="http://www.g3journal.org/content/6/12/3985.short?rss=1&amp;ssource=mfr">Evolutionary genomics of peach and almond domestication</a> [<a href="http://biorxiv.org/content/early/2016/09/27/060160">preprint</a>]<br />
<strong>Velasco D</strong>, <strong>Hough J</strong>, Aradhya M, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Resequencing data from a number of peach and almond varieties show differences in population size and mating system, but little evidence for convergent selection during domestication.
</p>
<p></details></p>
<details> <summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12578/full">ANGSD-wrapper: utilities for analyzing next generation sequencing data</a> [<a href="https://github.com/mojaveazure/angsd-wrapper">github</a>] [<a href="https://peerj.com/preprints/1472/">preprint</a>]<br />
<strong>Durvasula A</strong>, Hoffman PJ, <strong>Kent TV</strong>, Liu C, Kono TJY, Morrell PL, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We provide a set of scripts and configuration files to facilitate easy popgen analysis of short-read sequencing data.
</p>
<p></details></p>
<details> <summary><a href="http://rdcu.be/kHEB">Recent demography drives changes in linked selection across the maize genome</a> [<a href="https://github.com/timbeissinger/Maize-Teo-Scripts/">github</a>] [<a href="http://biorxiv.org/content/early/2015/11/13/031666">preprint</a>]<br />
<strong>Beissinger TM</strong>, Wang L, <strong>Crosby K</strong>, <strong>Durvasula A</strong>, Hufford MB, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We find little evidence for widespread hard sweeps in maize or teosinte and show that differences in demography during maize domestication and subsequent expansion have have changed the impacts of linked selection on surrounding diversity in the maize genome.
</p>
<p></details></p>
<details> <summary><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy201610a.html">Maize diversity associated with social origin and environmental variation in southern Mexico</a> [<a href="https://peerj.com/preprints/1192/">preprint</a>]<br />
Orozco-Ramírez Q, Santacruz-Varela A, <strong>Ross-Ibarra J</strong>, Brush S </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://journal.frontiersin.org/article/10.3389/fpls.2016.00308/full">High quality maize centromere 10 sequence reveals evidence of frequent recombination events</a><br />
Wolfgruber TK, Nakashima MM …(5 authors)… <strong>Bilinski P</strong>, Dawe RK, <strong>Ross-Ibarra J</strong>, Birchler JA, Presting G </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-3" class="section level3">
<h3>2015</h3>
<details> <summary><a href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=2&amp;cad=rja&amp;uact=8&amp;ved=0CC0QFjABahUKEwiY8ezQoa3IAhWRL4gKHYbZCYU&amp;url=http%3A%2F%2Fwww.genetics.org%2Fcontent%2Fearly%2F2015%2F09%2F18%2Fgenetics.115.182410.abstract&amp;usg=AFQjCNF6HUbfJ3ij4yqX75WFIP7h0VqD8g&amp;sig2=7Gw1t6U1cc7imLVZEUWlHQ">The genomic impacts of drift and selection for hybrid performance in maize</a> [<a href="http://arxiv.org/abs/1307.7313">preprint</a>]<br />
Gerke JP, Edwards JW, Guill KE, <strong>Ross-Ibarra J</strong>, McMullen MD </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.nature.com.sci-hub.bz/ng/journal/v47/n12/full/ng.3422.html">Seed filling in domesticated maize and rice depends on SWEET-mediated hexose transport</a><br />
Sosso D, Luo D, Li Q-B, Sasse J, <strong>Yang J</strong> …[6 authors]… <strong>Ross-Ibarra J</strong>, Yang B, Frommer WB </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.genetics.org/content/200/4/1297">Independent molecular basis of convergent highland adaptation in maize</a> [<a href="http://biorxiv.org/content/early/2015/01/09/013607">preprint</a>] [<a href="https://github.com/rossibarra/hilo_paper">github</a>] [<a href="https://dx.doi.org/10.6084/m9.figshare.746990.v1">data</a>]<br />
<strong>Takuno S</strong>, Ralph P, Swarts K, Elshire RJ, Glaubitz JC, Buckler ES, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We find that maize adaptation to the highlands in Mexico and South America is largely independent and has made considerable use of standing genetic variation.
</p>
<p></details></p>
<details> <summary><a href="https://peerj.com/articles/900/">Natural variation in teosinte at the domestication locus teosinte branched1 (tb1)</a> [<a href="https://peerj.com/preprints/685/">preprint</a>]<br />
<strong>Vann LE</strong>, <strong>Kono TJ</strong>, <strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>. </summary>
<p style="margin-left: 30px">
We find that variation in the domestication locus tb1 does not explain a substantial proportion of phenotypic variation for tillering in natural populations of teosinte.
</p>
<p></details> <!--  [[github](https://github.com/lvann/tb1)] --></p>
<details> <summary><a href="http://elifesciences.org/content/4/e05861">Genetic, evolutionary and plant breeding insights from the domestication of maize</a><br />
Hake S, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://rdcu.be/kHD8">The origin and evolution of maize in the American Southwest</a> [<a href="http://biorxiv.org/content/early/2015/01/11/013540">preprint</a>] [<a href="ftp://ftp_20150114_27609:F+jYHhrE6t2e@ftp.dna.ku.dk">data</a>]<br />
Fonseca RR, Smith B, Wales N, Cappellini E …[12 authors]… Hufford MB, Albrechtsen A, <strong>Ross-Ibarra J</strong>, Gilbert MT </summary>
<p style="margin-left: 30px">
We use sequence data from ancient maize cobs to reveal the timing and geography of maize introduction into the Southwest US. One of the <a href="https://www.nature.com/collections/dnshkjhpxx/content/most-read">most read papers</a> from the last four years of the journal! <a href="http://news.ucdavis.edu/search/news_detail.lasso?id=11124">UC Davis Press Release</a> <a href="http://www.scienceworldreport.com/articles/21122/20150112/origins-maize-dna-tests-reveal-two-paths.htm">Science World Report</a> <a href="http://www.archaeology.org/news/2879-150108-southwest-corn-dna">Archaeology</a> <a href="http://www.agenciasinc.es/Noticias/El-maiz-entro-en-el-suroeste-de-EE-UU-hace-4.000-anos-a-traves-de-la-montana">SiNC</a>, <a href="http://news.sciencemag.org/archaeology/2015/01/how-corn-became-corn">Science magazine</a> <a href="http://www.nature.com/articles/nplants20149">Nature Plants commentary</a> <a href="http://www.the-scientist.com/?articles.view/articleNo/41865/title/Corn-Chronicle/">The Scientist</a>
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/dyer2015.pdf">Reply to Brush <em>et al.</em>: Wake-up call for crop conservation science</a><br />
Dyer GA, Lopez-Feldman A, Yunez-Naude A, Taylor JE, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004915">Transposable Elements Contribute to Activation of Maize Genes in Response to Abiotic Stress</a> [<a href="http://biorxiv.org/content/early/2014/08/15/008052">preprint</a>]<br />
Makarevitch I, Waters M, West P, <strong>Stitzer M</strong>, <strong>Ross-Ibarra J</strong>, Springer NM</summary>
<p style="margin-left: 30px">
We show that transposable elements can act as regulatory sequences that up- or down-regulate genes in maize in response to abiotic stress.
</p>
<p></details></p>
</div>
<div id="section-4" class="section level3">
<h3>2014</h3>
<details> <summary><a href="http://www.rilab.org/pdfs/tiffin2014.pdf">Advances and limits of using population genetics to understand local adaptation</a> [<a href="https://peerj.com/preprints/488/">preprint</a>]<br />
Tiffin P, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We discuss some of the outstanding questions and key challenges facing the use of population genetics to study local adaptation.
</p>
<p></details></p>
<details> <summary> <a href="http://rilab.org/pdfs/Bilinski2014.pdf">Diversity and evolution of centromere repeats in the maize genome</a> [<a href="http://biorxiv.org/content/early/2014/05/12/005058">preprint</a>]<br />
<strong>Bilinski P</strong>, <strong>Distor KD</strong>, <strong>Gutierez-Lopez J</strong>, <strong>Mendoza Mendoza G</strong>, Shi J, Dawe K, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.g3journal.org/content/4/1/163.full">The pattern and distribution of deleterious mutations in maize</a> [<a href="http://arxiv.org/abs/1308.0380">preprint</a>] [<a href="https://github.com/RILAB/siftmappR">github</a>]<br />
<strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We show that deleterious mutations are abundant in modern maize breeding material and that genes with deleterious mutations are likely important contributors to phenotypic variation.
</p>
<p></details></p>
</div>
<div id="section-5" class="section level3">
<h3>2013</h3>
<details> <summary><a href="http://www.pnas.org/content/110/48/19639.abstract">Comprehensive analysis of imprinted genes in maize reveals allelic variation and limited conservation with other species</a> [<a href="http://arxiv.org/abs/1307.7678">preprint</a>] [<a href="https://github.com/paulbilinski/Waters_etal_SupplementalTableAnalysis">github</a>]<br />
Waters AJ, <strong>Bilinski P</strong>, Eichten SR, Vaugh MW, <strong>Ross-Ibarra J</strong>, Gehring M, Springer NM </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://gbe.oxfordjournals.org/content/5/9/1594">Complex patterns of local adaptation in teosinte</a> [<a href="http://datadryad.org/resource/doi:10.5061/dryad.8m648/1">data</a>] [<a href="http://arxiv.org/abs/1208.0634">preprint</a>]<br />
<strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003604">From many, one: genetic control of prolificacy during maize domestication</a> [<a href="http://figshare.com/articles/Wills_et_al_2013/775358">data</a>] [<a href="http://arxiv.org/abs/1303.0882">preprint</a>]<br />
Wills DM, Whipple C, <strong>Takuno S</strong>, Kursel LE, Shannon LM, <strong>Ross-Ibarra J</strong>, Doebley JF. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/McCouch2013.pdf">Agriculture: Feeding the future</a><br />
McCouch S, Baute GJ, Bradeen J, Bramel P …[36 authors]… Ward J, Waugh R, Wenzl P, Zamir. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003477">The genomic signature of crop-wild introgression in maize</a> [<a href="http://figshare.com/articles/Hufford_et_al_2013/766356">data</a>] [<a href="http://arxiv.org/abs/1208.3894">preprint</a>]<br />
<strong>Hufford MB</strong>, Lubinsky P, <strong>Pyhäjärvi T</strong>, <strong>Devengenzo MT</strong>, Ellstrand NC, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/provance2013.pdf">Population genetics and ethnobotany of cultivated <em>Diospyros riojae</em> (Ebenaceae), an endangered fruit crop from Mexico</a> [<a href="http://figshare.com/articles/Population_Genetics_and_Ethnobotany_of_Cultivated_Diospyros_riojae_G_mez_Pompa_Ebenaceae_an_Endangered_Fruit_Crop_from_Mexico/105841">preprint</a>]<br />
<strong>Provance MC</strong>, Garcia Ruiz I, <strong>Thommes C</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://genomebiology.com/2013/14/1/R10">Patterns of centromere tandem repeat evolution in 282 animal and plant genomes</a> [<a href="http://arxiv.org/abs/1209.4967">preprint</a>]<br />
Melters DP, Bradnam KR, Young HA, Telis N …[9 authors]… Tobias C, <strong>Ross-Ibarra J</strong>, Korf IF, Chan SW-L. </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy20132a.html">Diversity and abundance of the Abnormal chromosome 10 meiotic drive complex in <em>Zea mays</em></a> [<a href="http://arxiv.org/abs/1209.5466">preprint</a>]<br />
Kanizay LB, <strong>Pyhäjärvi T</strong>, Lowry E, <strong>Hufford MB</strong>, Peterson DG, <strong>Ross-Ibarra J</strong>, Dawe RK </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-6" class="section level3">
<h3>2012</h3>
<details> <summary><a href="http://www.rilab.org/pdfs/hufford2012.pdf">Teosinte as a model system for population and ecological genomics</a><br />
<strong>Hufford MB</strong>, <strong>Bilinski P</strong>, <strong>Pyhäjärvi T</strong>, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_24240aa930bf4728be83b4b57da55f17.pdf">Comparative population genomics of maize domestication and improvement</a> [<a href="http://figshare.com/articles/new_fileset/757736">data</a>]<br />
<strong>Hufford MB</strong>, Xun X, <strong>van Heerwaarden J</strong>, <strong>Pyhäjärvi T</strong> …[17 authors]… Ware D, Buckler ES, Yang S, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Munoz_Diez_R1-1.pdf">Using nextgen sequencing to investigate genome size variation and transposable element content</a><br />
Munoz Diez C, Vitte C, <strong>Ross-Ibarra J</strong>, Gaut BS, Tenaillon MI </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/vanheerwaarden2012.pdf">Historical genomics of North American maize</a> [<a href="http://figshare.com/articles/van_Heerwaarden_et_al_2012/757738">data</a>]<br />
<strong>van Heerwaarden J</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/swansonwagner2012.pdf">Reshaping of the maize transcriptome by domestication</a><br />
Swanson-Wagner R, Briskine R, Schaefer R, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>, Myers CL, Tiffin P, Springer NM </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_86a609db506740ce96c549567c246b4c.pdf">Maize HapMap2 identifies extant variation from a genome in flux</a><br />
Chia J-M, Song C, …[<strong>Hufford, MB</strong>, <strong>Pyhäjärvi T</strong>, and 27 authors]… <strong>Ross-Ibarra, J</strong>, McMullen MD, Buckler ES, Zhang G, Xu Y, Ware, D </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.genetics.org/content/191/3/883.abstract">Megabase-scale inversion polymorphism in the wild ancestor of maize</a> [<a href="http://figshare.com/articles/Fang_et_al_2012/840519">data</a>]<br />
Fang Z, <strong>Pyhäjärvi T</strong>, Weber AL, Dawe RK, …[2 authors]… <strong>Ross-Ibarra C</strong>, Doebley J, Morrell PL, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
We identify and study the evolutionary genetics of a 50Mb inversion in natural populations of teosinte
</p>
<p></details></p>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_5ca9c5a7f6174a5bb0c85e67dec67dcc.pdf">Genetic architecture of maize kernel composition in the Nested Association Mapping and Inbred Association panels</a><br />
Cook JP, McMullen MD, Holland JB, Tian F, Bradbury P, <strong>Ross-Ibarra J</strong>, Buckler ES, Flint-Garcia SA </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary> <a href="http://media.wix.com/ugd/fe9228_a2a75f7435ec4c569d4a6996e8b5451e.pdf">Crop genomics: advances and applications</a><br />
Morrell PL, Buckler ES, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-7" class="section level3">
<h3>2011</h3>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_14630f30e97f4e04b8b5980632002f7a.pdf">Identification of a functional transposon insertion in the maize domestication gene <em>tb1</em></a> [<a href="http://www.panzea.org/db/gateway?file_id=Studer_etal_2011_Nat_Genet_data">data</a>]<br />
Studer A, Zhao Q, <strong>Ross-Ibarra J</strong>, Doebley J </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_4916e7f7acec4aee869da040d519e0be.pdf">Genetic signals of origin, spread and introgression in a large sample of maize landraces</a> [<a href="http://figshare.com/articles/new_fileset/757735">data</a>]<br />
<strong>van Heerwaarden J</strong>, Doebley J, Briggs WH, Glaubitz JC, Goodman MM, Sanchez Gonzalez JJ, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We resolve the conflict between genetic, ecological, and achaeological data regarding the geographic location of maize domestication.
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/hufford2011.pdf">Influence of cryptic population structure on observed mating patterns in the wild progenitor of maize</a> [<a href="10.6084/m9.figshare.757734">data</a>]<br />
<strong>Hufford MB</strong>, Gepts P, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="https://www.ncbi.nlm.nih.gov/pubmed/21296765">Genome size and TE content as determined by high-throughput sequencing in maize and <em>Zea luxurians</em></a><br />
Tenaillon MI, <strong>Hufford MB</strong>, Gaut BS, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-8" class="section level3">
<h3>2010</h3>
<details> <summary><a href="http://www.genetics.org/content/185/3/969.abstract">Patterns of population structure and environmental associations to aridity across the range of loblolly pine (<em>Pinus taeda</em>)</a><br />
Eckert AJ, <strong>van Heerwaarden J</strong>, Wegrzyn JL, Nelson CD, <strong>Ross-Ibarra J</strong>, Gonzalez-Martinez SC, and Neale DB </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary> <a href="http://www.rilab.org/pdfs/Fuchs-et-al-2010.pdf">Reproductive biology of <strong>Macleania rupestris</strong> (Ericaceae): a pollen-limited Neotropical cloud-forest species in Costa Rica</a><br />
Fuchs EJ, <strong>Ross-Ibarra J</strong>, Barrantes G </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Whitney-et-al-2010-402.pdf">A role for nonadaptive processes in plant genome size evolution?</a><br />
Whitney KD, Baack EJ, Hamrick JL, Godt, MJW …[3 authors]… Goodwillie C, Kalisz S, Leitch I, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Van-Heerwaarden-et-al-2010-951.pdf">Fine scale genetic structure in the wild ancestor of maize (<strong>Zea mays</strong> ssp. <strong>parviglumis</strong>)</a><br />
<strong>van Heerwaarden J</strong>, <strong>Ross-Ibarra J</strong>, Doebley J, Glaubitz JC, Sanchez Gonzalez J, Gaut BS, Eguiarte LE</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000327">Widespread gene conversion in centromere cores</a><br />
Shi J, Wolf S, Burke J, Presting G, <strong>Ross-Ibarra J</strong>, Dawe RK </summary>
<p style="margin-left: 30px">
[<a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000326">plos biology commentary</a>]
</p>
<p></details></p>
<details> <summary><a href="http://mbe.oxfordjournals.org/content/27/2/409.abstract">Indel-associated mutation rate varies with mating system in flowering plants</a><br />
Hollister JD, <strong>Ross-Ibarra J</strong>, Gaut BS </summary>
<p style="margin-left: 30px">
We show that mating system differences in heterozygosity mitigate the effects of insertion and deletion polymorphisms on nucleotide diversity.
</p>
<p></details></p>
<details> <summary> <a href="http://www.nature.com/hdy/journal/v104/n1/full/hdy2009110a.html">Genetic diversity in a crop metapopulation</a><br />
<strong>van Heerwaarden J</strong>, van Eeuwijk FA, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We develop a coalescent model to predict genetic structure and diversity of a crop plant in an agronomic ecosystem.
</p>
<p></details></p>
</div>
<div id="section-9" class="section level3">
<h3>2009</h3>
<details> <summary><a href="http://media.wix.com/ugd/fe9228_352f4ee9b69a425589e55a4bac60bc5d.pdf">A first-generation haplotype map of maize</a><br />
Gore MA, Chia JM, Elshire RJ, Sun Q …[4 authors]… Grills GS, <strong>Ross-Ibarra J</strong>, Ware DH, Buckler ES </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0008346">A pleistocene clone of Palmer’s Oak persisting in Southern California</a> <strong>May MR</strong>, <strong>Provance MC</strong>, Sanders AC, Ellstrand NC, <strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
We discover a 13,000 year old oak clone in the hilss of Southern California
</p>
<p></details></p>
<details> <summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1469-8137.2009.02984.x/epdf">Selection on grain shattering genes and rates of rice domestication</a><br />
Zhang LB, Zhu Q, Wu ZQ, <strong>Ross-Ibarra J</strong>, Gaut BS, Ge S, Sang T </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.genetics.org/content/181/4/1399.abstract">Historical divergence and gene flow in the genus Zea</a><br />
<strong>Ross-Ibarra J</strong>, Tenaillon M, Gaut BS </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-10" class="section level3">
<h3>2008</h3>
<details> <summary><a href="http://dx.doi.org/10.1371%2Fjournal.pone.0002411">Patterns of polymorphism and demographic history in natural populations of <em>Arabidopsis lyrata</em></a><br />
<strong>Ross-Ibarra J</strong>, Wright SI, Foxe JP, Kawabe A, DeRose-Wilson L, Gos G, Charlesworth D, Gaut BS </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.pnas.org/content/105/37/13965.short">Demography and weak selection drive patterns of TE diversity in natural populations of <em>A. lyrata</em></a><br />
Lockton S, <strong>Ross-Ibarra J</strong>, Gaut BS</summary>
<p style="margin-left: 30px">
We show that differences in demographic history among populations of have changed the abundance and freqeuncy of transposable elements in the <em>A lyrata</em> genome.
</p>
<p></details></p>
<details> <summary><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614810/">Multiple domestications do not appear monophyletic</a><br />
<strong>Ross-Ibarra J</strong>, Gaut BS</summary>
<p style="margin-left: 30px">
A rebuttal showing that genetic data can indeed successfully recover multiple independent domestications.
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Ross-Ibarra_Gaut_2008.pdf">Selection on major components of angiosperm genomes</a><br />
Gaut BS, <strong>Ross-Ibarra J</strong></summary>
<p style="margin-left: 30px">
Discusses some of the evolutionary forces responsible for shaping genome size across plants.
</p>
<p></details></p>
</div>
<div id="section-11" class="section level3">
<h3>2007</h3>
<details> <summary><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1876441/">Plant domestication, a unique opportunity to identify the genetic basis of adaptation</a><br />
<strong>Ross-Ibarra J</strong>, Morrell PL, Gaut BS</summary>
<p style="margin-left: 30px">
Presents plant domestication as an ideal system in which to study the genetics of adaptation.
</p>
<p></details></p>
<details> <summary><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2006.01275.x/abstract">Genome size and recombination in angiosperms: a second look</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
<div id="section-12" class="section level3">
<h3>2001-2006</h3>
<details> <summary><a href="http://www.rilab.org/pdfs/Wares-et-al-2006.pdf">Mitochondrial DNA and population size</a><br />
Wares JP, Barber PH, <strong>Ross-Ibarra J</strong>, Sotka EE, Toonen RJ </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Ross-Ibarra-2005-15881692.pdf">Quantitative trait loci and the study of plant domestication</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Ross-Ibarra-2004.pdf">The evolution of recombination under domestication: a test of two hypotheses</a> [<a href="http://www.rilab.org/pubs/supp_data_rossibarra2004.tgz">data</a>]<br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
Here I test two different hypotheses about the how recombination should relate to domestication: Does domestication itself select for increased recombination (yes), or respecies with higher recombination rates more likely to be domesticated (no).
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Ross-Ibarra-2003.pdf">Origen y domesticación de la chaya (<em>Cnidoscolus aconitifolius</em> Mill IM Johnst): La espinaca Maya</a><br />
<strong>Ross-Ibarra J</strong> </summary>
<p style="margin-left: 30px">
I use genetic data to study the origin of a leafy shrub domesticated by the Maya
</p>
<p></details></p>
<details> <summary><a href="http://www.rilab.org/pdfs/Ross-Ibarra_Molina-Cruz-2002.pdf">The ethnobotany of Chaya (<em>Cnidoscolus aconitifolius</em> ssp. <em>aconitifolius</em> Breckon): A nutritious Maya vegetable</a><br />
<strong>Ross-Ibarra J</strong>, Molina-Cruz A </summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
<details> <summary><a href="http://www.amjbot.org/content/88/7/1214.abstract">Implications of mating patterns for conservation of the endangered plant <em>Eriogonum ovalifolium</em> var. <em>vineum</em></a><br />
Neel MC, <strong>Ross-Ibarra J</strong>, Ellstrand NC</summary>
<p style="margin-left: 30px">
SUMMARY
</p>
<p></details></p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
