<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->





<script>
  window.onload = choosePic;
  var myPix = new Array("images/image1.jpg","images/image2.jpg","images/image3.jpg","images/image4.jpg","images/image5.jpg","images/image6.jpg","images/image7.jpg","images/image8.jpg","images/image8.jpg","images/image10.jpg");
  function choosePic() {
     var randomNum = Math.floor(Math.random() * myPix.length);
     document.getElementById("myPicture").src = myPix[randomNum];
  }
</script>

<div class="navbar navbar-default  navbar-float-top" role="navigation">
    <div class="container" overflow="auto" align="left" style="width: inherit;">
      <img id="myPicture" alt="Brand" src="" align='center'>
    </div>

  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <img alt="Brand" src="images/cornkey_icon.png" align='left'>
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>

      </button>

      <a class="navbar-brand" href="index.html">Ross-Ibarra Lab</a>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav" align="center">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs/wiki">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
        <li><a href="press.html">Press</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<div id="research" class="section level2">
<h2>Research</h2>
<hr>
<p><a name="adaptation"></a> <img src="images/teosinte.jpg" /></p>
</div>
<div id="genetics-of-adaptation" class="section level2">
<h2><strong>Genetics of adaptation</strong></h2>
<p>Maize spread rapidly after domestication, adapting to a wide range of environments. Today maize is grown across a broader geographic breadth than any of the world’s other staple crops, from sea level to altitudes of <span class="math inline">\(&gt;4,000\)</span>m and from deserts to near-flooded conditions. The wild relatives of maize have also adapted to environments varying widely in elevation, temperature, and moisture availability. The lab is working on a number of projects using maize and its wild relatives to understand the genetic basis of adaptation. Projects in this area are currently funded by multiple grants from the NSF PGRP (<a href="https://nsf.gov/awardsearch/showAward?AWD_ID=1546719&amp;HistoricalAwards=false">here</a> and <a href="https://nsf.gov/awardsearch/showAward?AWD_ID=1822330&amp;HistoricalAwards=false">here</a>)</p>
<div id="selected-recent-publications" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p>Adaptive phenotypic divergence in teosinte differs across biotic contexts [<a href="https://www.biorxiv.org/content/early/2018/08/01/382770">preprint</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Strauss SY, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="http://www.genetics.org/content/early/2019/01/24/genetics.118.301786">Detecting adaptive differentiation in structured populations with genomic data and common gardens</a> [<a href="https://www.biorxiv.org/content/early/2018/07/13/368506">preprint</a>] [<a href="https://github.com/emjosephs/qpc-maize">github</a>]<br />
<strong>Josephs EB</strong>, Berg JJ, <strong>Ross-Ibarra J</strong>, Coop G</p></li>
<li><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007794">Genetic architecture and selective sweeps after polygenic adaptation to distant trait optima</a> [<a href="https://www.biorxiv.org/content/early/2018/05/03/313247">preprint</a>] [<a href="https://mgstetter.shinyapps.io/quantgensimAPP/">Shiny app</a>]<br />
<strong>Stetter MG</strong>, Thornton K, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://www.journals.uchicago.edu/doi/10.1086/700118">Evolutionary responses to conditionality in species interactions across environmental gradients</a> [<a href="http://biorxiv.org/content/early/2017/03/01/031195">preprint</a>][<a href="https://www.authorea.com/inst/14743?articles_format=list&amp;page=5">review</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, <strong>Ross-Ibarra J</strong>, Strauss SY</p></li>
<li><p><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162">Parallel altitudinal clines reveal adaptive evolution of genome size in <em>Zea mays</em></a> [<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007249">perspective</a>] [<a href="http://www.biorxiv.org/content/early/2017/07/13/134528">preprint</a>] [<a href="https://github.com/paulbilinski/GenomeSizeAnalysis">github</a>] [<a href="https://www.slideshare.net/jrossibarra/parallel-altitudinal-clines-reveal-adaptive-evolution-of-genome-size-in-zea-mays">slides</a>]<br />
<strong>Bilinski P</strong>, Albert P, Berg JJ, Birchler JA, Grote M, <strong>Lorant A</strong>, <strong>Quezada J</strong>, Swarts K, <strong>Yang J</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajb2.1002">Adaptation in plant genomes: bigger is different</a> [<a href="https://www.biorxiv.org/content/early/2017/10/02/196501">preprint</a>][<a href="https://github.com/RILAB/AJB_MutationalTargetSize_GenomeSize">github</a>][<a href="https://www.youtube.com/watch?v=aXLuWruOmO4_">YouTube</a>][<a href="https://www.slideshare.net/jrossibarra/adaptation-in-plant-genomes-bigger-is-different">slides</a>]<br />
<strong>Mei W</strong>, <strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1346-4">The interplay of demography and selection during maize domestication and diffusion</a> [<a href="http://www.google.com/url?q=http%3A%2F%2Fbiorxiv.org%2Fcontent%2Fearly%2F2017%2F03%2F07%2F114579&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHr4cDzAxS_7BXVZOfyuIWzXiHr0w">preprint</a>][<a href="https://github.com/HuffordLab/Wang_et_al._Demography">github</a>][<a href="https://www.youtube.com/watch?v=KgIUGHEZ2nM">YouTube</a>]<br />
Wang L, <strong>Beissinger TM</strong>, <strong>Lorant A</strong>, <strong>Ross-Ibarra C</strong>, <strong>Ross-Ibarra J</strong>, Hufford MB.</p></li>
<li><p><a href="http://rdcu.be/kHEB">Recent demography drives changes in linked selection across the maize genome</a> [<a href="https://github.com/timbeissinger/Maize-Teo-Scripts/">github</a>] [<a href="http://biorxiv.org/content/early/2015/11/13/031666">preprint</a>]<br />
<strong>Beissinger TM</strong>, Wang L, <strong>Crosby K</strong>, <strong>Durvasula A</strong>, Hufford MB, <strong>Ross-Ibarra J</strong></p></li>
</ul>
<hr>
<p><a name="experimental"></a> <img src="images/corn.jpg" /></p>
</div>
</div>
<div id="experimental-evolution" class="section level2">
<h2><strong>Experimental Evolution</strong></h2>
<p>Plant domestication and modern breeding represent examples of experimentally evolved populations. Studying these populations provides an opportunity to understand not only the genetic basis of evolutionary change but also how the processes of evolution interact to shape modern genetic and phenotypic diversity. Projects in this area are currently funded by the <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1753632&amp;HistoricalAwards=false">NSF DEB</a>.</p>
<div id="selected-recent-publications-1" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007794">Genetic architecture and selective sweeps after polygenic adaptation to distant trait optima</a> [<a href="https://www.biorxiv.org/content/early/2018/05/03/313247">preprint</a>] [<a href="https://mgstetter.shinyapps.io/quantgensimAPP/">Shiny app</a>]<br />
<strong>Stetter MG</strong>, Thornton K, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://nph.onlinelibrary.wiley.com/doi/abs/10.1111/nph.15350">Maize domestication and gene interaction</a> <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007019">Incomplete dominance of deleterious alleles contribute substantially to trait variation and heterosis in maize</a> [<a href="http://biorxiv.org/content/early/2016/11/07/086132">preprint</a>] [<a href="https://github.com/yangjl/GERP-diallel">github</a>]<br />
<strong>Yang J</strong>, <strong>Mezmouk S</strong>, Baumgarten A, Buckler ES, Guill KE, McMullen MD, Mumm RH, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0184202">The potential role of genetic assimilation during maize domestication</a> [<a href="http://biorxiv.org/content/early/2017/02/03/105940">preprint</a>]<br />
<strong>Lorant A</strong>, Pedersen, S, Holst I, Hufford MB, Winter K, Piperno D, <strong>Ross-Ibarra J</strong></p></li>
</ul>
<hr>
<p><a name="csomes"></a> <img src="images/csomes.jpg" /></p>
</div>
</div>
<div id="genome-evolution" class="section level2">
<h2><strong>Genome Evolution</strong></h2>
<p>In addition to discerning the genetic basis of phenotypic evolution, we are interested in understanding the processes that shape evolution of the genome itself. From copy number variation and inversions to the evolution of recombination rate and coevolution between transposable elements and their hosts, the diversity and evolutionary lability of genomes offers a lot to explore. This work is currently funded by the <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1238014">NSF PGRP</a></p>
<div id="selected-recent-publications-2" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p>The Genomic Ecosystem of Transposable Elements in Maize [<a href="https://www.biorxiv.org/content/10.1101/559922v1">preprint</a>] <strong>Stitzer MC</strong>, Anderson SN, Springer NM, <strong>Ross-Ibarra J</strong></p></li>
<li><p>Transposable elements contribute to dynamic genome content in maize [<a href="https://www.biorxiv.org/content/10.1101/547398v1?rss=1">preprint</a>]<br />
Anderson SN, <strong>Stitzer MC</strong>, Brohammer AB, Peng Zhou P, Noshay JM, Hirsch CD, <strong>Ross-Ibarra J</strong>, Hirsch CN, Springer NM</p></li>
<li><p><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007162">Parallel altitudinal clines reveal adaptive evolution of genome size in <em>Zea mays</em></a> [<a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007249">perspective</a>] [<a href="http://www.biorxiv.org/content/early/2017/07/13/134528">preprint</a>] [<a href="https://github.com/paulbilinski/GenomeSizeAnalysis">github</a>] [<a href="https://www.slideshare.net/jrossibarra/parallel-altitudinal-clines-reveal-adaptive-evolution-of-genome-size-in-zea-mays">slides</a>]<br />
<strong>Bilinski P</strong>, Albert P, Berg JJ, Birchler JA, Grote M, <strong>Lorant A</strong>, <strong>Quezada J</strong>, Swarts K, <strong>Yang J</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajb2.1002">Adaptation in plant genomes: bigger is different</a> [<a href="https://www.biorxiv.org/content/early/2017/10/02/196501">preprint</a>][<a href="https://github.com/RILAB/AJB_MutationalTargetSize_GenomeSize">github</a>][<a href="https://www.youtube.com/watch?v=aXLuWruOmO4_">YouTube</a>][<a href="https://www.slideshare.net/jrossibarra/adaptation-in-plant-genomes-bigger-is-different">slides</a>]<br />
<strong>Mei W</strong>, <strong>Stetter MG</strong>, <strong>Gates DJ</strong>, <strong>Stitzer MC</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://academic.oup.com/mbe/article/34/8/1825/3742530/Gene-Fractionation-and-Function-in-the-Ancient?guestAccessKey=391f8f04-5fd6-4bc7-bac0-0057fb3102ab">Gene fractionation and function in the ancient subgenomes of maize</a> [<a href="http://biorxiv.org/content/early/2016/12/19/095547">preprint</a>]<br />
<strong>Renny-Byfield S</strong>, Rodgers-Melnick E, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177896">Genomic abundance is not predictive of tandem repeat localization in grass genomes</a> [<a href="https://peerj.com/preprints/2314/">preprint</a>] [<a href="https://github.com/paulbilinski/Github_centrepeat">github</a>]<br />
<strong>Bilinski P</strong>, Han Y, <strong>Hufford MB</strong>, <strong>Lorant A</strong>, Zhang P, Estep MC, Jiang J, <strong>Ross-Ibarra J</strong></p></li>
</ul>
<hr>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
