<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research</title>

<script src="site_libs/header-attrs-2.30/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Ross-Ibarra Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="people.html">People</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="https://github.com/RILAB/lab-docs">Lab Docs</a>
</li>
<li>
  <a href="rehab.html">R.E.H.A.B</a>
</li>
<li>
  <a href="news.html">News</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research</h1>

</div>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<hr>
<div id="genetics-of-adaptation" class="section level2">
<h2><strong>Genetics of adaptation</strong></h2>
<p><img src="images/teosinte.jpg" width="150" align="right"/></p>
<p>Maize spread rapidly after domestication, adapting to a wide range of
environments. Today maize is grown across a broader geographic breadth
than any of the world’s other staple crops, from sea level to altitudes
of <span class="math inline">\(&gt;4,000\)</span>m and from deserts to
near-flooded conditions. The wild relatives of maize have also adapted
to environments varying widely in elevation, temperature, and moisture
availability. The lab works on a number of projects using maize and its
wild relatives to understand the genetic basis of adaptation.</p>
<div id="selected-recent-publications" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1011714">The
utility of environmental data from traditional varieties for
climate-adaptive maize breeding</a> [<a
href="https://www.biorxiv.org/content/10.1101/2024.09.19.613351v1">preprint</a>]<br />
<strong>Li F</strong>, <strong>Gates DJ</strong>, Buckler ES …[8
authors]… Hearne S, <strong>Ross-Ibarra J</strong>, Runcie DE</p></li>
<li><p><a href="https://elifesciences.org/reviewed-preprints/92405">Not
so local: the population genetics of convergent adaptation in maize and
teosinte.</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.09.09.459637v1">preprint</a>]<br />
<strong>Tittes S</strong>, <strong>Lorant A</strong>, <strong>McGinty
S</strong> …[4 authors]… Tenaillon MI, <strong>Ross-Ibarra
J</strong></p></li>
<li><p><a
href="https://www.nature.com/articles/s41588-022-01184-y">Genome
sequencing reveals evidence of adaptive variation in the genus
<em>Zea</em></a> [<a
href="https://www.biorxiv.org/content/10.1101/2022.06.03.494450v1">preprint</a>]<br />
Chen L, Luo J, Minliang Jin, <strong>Yang N</strong> …[27 authors
including <strong>Phillips AR</strong> and <strong>Cameron B</strong>]…
<strong>Ross-Ibarra J</strong>, Yan J.</p></li>
<li><p><a
href="https://academic.oup.com/mbe/article/39/11/msac239/6795225">Allele-specific
expression reveals multiple paths to highland adaptation in maize</a>
[<a
href="https://www.biorxiv.org/content/10.1101/2022.07.15.500250v1?rss=1">preprint</a>]<br />
Hu H, Crow T, Nojoomi S, …[5 authors]… Estévez-Palmas JM,
<strong>Ross-Ibarra J</strong>, Runcie DE.</p></li>
<li><p>The origins and adaptive consequences of polyploidy in a dominant
prairie grass. [<a
href="https://www.biorxiv.org/content/10.1101/2025.11.25.690567v1">preprint</a>]<br />
<strong>Phillips A</strong>, AuBuchon-Elder T, …[23 authors including
<strong>Cameron B</strong>, <strong>Cryan EP</strong>, <strong>Julianna
Porter</strong>]… Kellogg EA, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="">Teosinte populations exhibit weak local adaptation to
their rhizosphere biota despite strong effects of biota source on
teosinte fitness and traits</a> [<a
href="https://www.biorxiv.org/content/10.1101/2021.04.20.440703v1">preprint</a>]<br />
<strong>O’Brien AM</strong>, Sawers RJH, Gasca-Pineda J, Baxter, I,
Eguiarte LE, <strong>Ross-Ibarra J</strong>, Strauss SY</p></li>
</ul>
<hr>
</div>
</div>
<div id="experimental-evolution" class="section level2">
<h2><strong>Experimental Evolution</strong></h2>
<p><img src="images/corn.jpg" width="150" align="right"/></p>
<p>Plant domestication and modern breeding represent examples of
experimentally evolved populations. Studying these populations provides
an opportunity to understand not only the genetic basis of evolutionary
change but also how the processes of evolution interact to shape modern
genetic and phenotypic diversity.</p>
<div id="selected-recent-publications-1" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p><a
href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msad170/7231441?utm_source=authortollfreelink&amp;utm_campaign=mbe&amp;utm_medium=email&amp;guestAccessKey=4c4e0dab-22b0-4c6d-be4f-009119c216fd">Unraveling
prevalence and effects of deleterious mutations in maize elite lines
across decades of modern breeding</a><br />
Sun S, Wang B, Li C, Xu G, Yang J, Hufford MBH, <strong>Ross-Ibarra
J</strong>, Wang H, Wang L</p></li>
<li><p><a
href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyaf085/8126661">Genetic
variation at transcription factor binding sites largely explains
phenotypic heritability in maize</a> [<a
href="https://biorxiv.org/cgi/content/short/2023.08.08.551183v1">preprint</a>]<br />
Engelhorn J, <strong>Snodgrass SJ</strong>, Kok A, Seetharam AS …[15
authors]… Frommer WB, <strong>Ross-Ibarra J</strong>, Hartwig T</p></li>
<li><p><a href="http://rilab.ucdavis.edu/pdfs/li22natplants.pdf">Genomic
Insights into Historical Improvement of Heterotic Groups during Modern
Hybrid Maize Breeding</a><br />
Li C, Guan H, Jing X, Li Y …[18 authors]… <strong>Ross-Ibarra
J</strong>, Li Y, Wang T, Wang H</p></li>
</ul>
<hr>
</div>
</div>
<div id="human-maize-coevolution." class="section level2">
<h2><strong>Human-Maize Coevolution</strong>.</h2>
<p><img src="images/aztec.jpg" width="150" align="right"/></p>
<p>As a domesticate maize is entirely dependent on humans for survival
and dispersal. In turn, human populations became increasingly dependent
on maize as a staple crop. Much of the current work in the lab
investigates the dynamics of how each of these two species have impacted
the evolution of the other.</p>
<div id="selected-recent-publications-2" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2503748122">An
ancient origin of the naked grains of maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/2024.12.02.626434v1">preprint</a>]<br />
<strong>Fairbanks R</strong>, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a href="https://www.science.org/doi/10.1126/science.adg8940">Two
teosintes made modern maize</a>. [<a
href="https://www.biorxiv.org/content/10.1101/2023.01.31.526540v1">preprint</a>]<br />
<strong>Yang N</strong>*, Wang Y*, Liu X* …[20 authors including
<strong>Mambakkam S</strong> and <strong>Menon M</strong>]… Stitzer MC,
Runcie DE, Yan J, <strong>Ross-Ibarra J</strong></p></li>
</ul>
<hr>
</div>
</div>
<div id="genome-evolution" class="section level2">
<h2><strong>Genome Evolution</strong></h2>
<p><img src="images/panel.jpeg" width="150" align="right"/></p>
<p>In addition to discerning the genetic basis of phenotypic evolution,
we are interested in understanding the processes that shape evolution of
the genome itself. From copy number variation and inversions to the
evolution of recombination rate and coevolution between transposable
elements and their hosts, the diversity and evolutionary lability of
genomes offers a lot to explore.</p>
<div id="selected-recent-publications-3" class="section level3">
<h3>Selected Recent Publications</h3>
<ul>
<li><p>Genome-wide selection on transposable elements in maize. [<a
href="https://www.biorxiv.org/content/10.1101/2025.09.16.676665v1">preprint</a>]<br />
<strong>Liu B</strong>, Munasinghe M, <strong>Fairbanks RA</strong>,
Hirsch CN, <strong>Ross-Ibarra J</strong></p></li>
<li><p><a
href="https://academic.oup.com/genetics/advance-article/doi/10.1093/genetics/iyaf085/8126661?login=false">Molecular
evolution of a reproductive barrier in maize and related species</a> [<a
href="https://www.biorxiv.org/content/10.1101/2024.12.02.626474v1">preprint</a>]<br />
<strong>Cryan, E</strong>, <strong>Phinney G</strong>, Seetharam AS,
Evans MMS, Kellogg EA, Zhan J, Meyers BC, Kliebenstein D,
<strong>Ross-Ibarra J</strong></p></li>
<li><p>Extensive genome evolution distinguishes maize within a stable
tribe of grasses [<a
href="https://www.biorxiv.org/content/10.1101/2025.01.22.633974v1">preprint</a>]<br />
Stitzer MC, Seetharam AS, Scheben A …[35 authors including
<strong>Phillips AR</strong>]… <strong>Ross-Ibarra J</strong>, Romay MC,
Kellogg EA, Buckler ES, Hufford MB</p></li>
<li><p><a href="https://rdcu.be/dQg54">Teosinte Pollen Drive guides
maize domestication and evolution by RNAi</a> [<a
href="https://www.biorxiv.org/content/10.1101/2023.07.12.548689v1">preprint</a>]<br />
Berube B, Ernst E, Cahn J …[3 authors]… Scheben A, Siepel A,
<strong>Ross-Ibarra J</strong>, Kermicle J, Martienssen RA.</p></li>
<li><p><a
href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001814#ack">Conflict
over fertilization underlies the transient evolution of
reinforcement.</a> [<a
href="https://www.biorxiv.org/content/10.1101/2020.11.10.377481v1">preprint</a>]<br />
<strong>Rushworth C</strong>, Wardlaw AW, <strong>Ross-Ibarra
J</strong>, Brandvain YB</p></li>
<li><p><a
href="https://www.nature.com/articles/s41467-020-19333-4">Adaptive
evolution of DNA methylation reshaped gene regulation in maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/2020.03.13.991117v1">preprint</a>]
[<a href="https://github.com/jyanglab/msfs_teo">github</a>]<br />
Xu G, Lyu J, Li Q, Liu H, Wang D, Zhang M, Springer NM,
<strong>Ross-Ibarra J</strong>, Yang J</p></li>
<li><p><a href="https://www.genetics.org/content/214/4/1019">The
temporal dynamics of background selection in non-equilibrium
populations</a> [<a
href="https://www.biorxiv.org/content/10.1101/618389v1">preprint</a>][<a
href="https://github.com/RILAB/BGS_sims/">github</a>]<br />
Torres R, <strong>Stetter MG</strong>, Hernandez R, <strong>Ross-Ibarra
J</strong></p></li>
<li><p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009768">The
Genomic Ecosystem of Transposable Elements in Maize</a> [<a
href="https://www.biorxiv.org/content/10.1101/559922v1">preprint</a>]
<strong>Stitzer MC</strong>, Anderson SN, Springer NM,
<strong>Ross-Ibarra J</strong></p></li>
</ul>
<hr>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
