<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>research_opps.knit</title>

<script src="site_libs/header-attrs-2.26/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>










<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<script>
  window.onload = choosePic;
  var myPix = new Array("images/image1.jpg","images/image2.jpg","images/image3.jpg","images/image4.jpg","images/image5.jpg","images/image6.jpg","images/image7.jpg","images/image8.jpg","images/image8.jpg","images/image10.jpg");
  function choosePic() {
    var randomNum = Math.floor(Math.random() * myPix.length);
    document.getElementById("myPicture").src = myPix[randomNum];
  }
</script>

<div class="navbar navbar-default  navbar-float-top" role="navigation">
    <div class="container" overflow="auto" align="left" style="width: inherit;">
      <img id="myPicture" src="" align='center'>
      </div>

  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>

      </button>

      <a class="navbar-brand" href="index.html"></a>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav" align="center">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="maize_history.html">Maize History</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs/wiki">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">




</div>


<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-5300959-3', 'auto');
  ga('send', 'pageview');

</script>
<div id="current-undergraduate-research-opportunities"
class="section level1">
<h1>Current Undergraduate Research opportunities</h1>
<div id="transposable-elements" class="section level2">
<h2>Transposable Elements</h2>
<div id="expression-of-transposable-elements-tes-in-different-tissues"
class="section level3">
<h3>Expression of transposable elements (TEs) in different tissues:</h3>
<p>Transcription is a vital step in the process of transposition, as TE
encoded proteins need to be generated. Although RNA-seq datasets exist
for a large number of tissues and developmental stages in maize,
particularly the inbred line B73, most analysis pipelines focus only on
protein coding genes. This project would use <a
href="https://pachterlab.github.io/kallisto/about">Kallisto</a>, a
relatively new program for quickly measuring gene expression
differences, to quantify how much transcription is occurring for a given
TE copy or TE family. For a TE to make it to the next generation, it has
to transpose in tissue that will become pollen or eggs, so we would
focus on these tissues and their precursors. Candidate TE families would
be Dotted, Ac, and a number of LTR retrotransposon families. It would be
neat to follow up on a result from Kunze that has never been published,
of increased transcription of Ds when Ac is present in the genome. This
seems counter-intuitive to host control of a TE.</p>
</div>
<div id="b73-polymorphism" class="section level3">
<h3>B73 polymorphism:</h3>
<p>The maize genome consists of over 85% TEs, and 75% LTR
retrotransposons, but little is known about the transpositional activity
of LTR retrotransposons. Only one LTR TE (bs1) has been observed to
jump. This project would use short read resequencing of B73 to identify
candidate TE regions that appear absent across accessions of the inbred
line B73. Although B73 is often thought of as a completely inbred stock,
differences have accumulated, as labs across the world grow these
plants, and generate their <a
href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0157942">own
stocks</a>. Transposition may contribute to differences between these
stocks.</p>
</div>
<div id="identifying-the-autonomous-partner-to-rdt"
class="section level3">
<h3>Identifying the autonomous partner to rDt:</h3>
<p><a href="http://www.genetics.org/content/23/4/377.long">Dotted</a> is
a DNA TE discovered by Marcus Rhoades in the 1940’s, named such because
it generates small dots on the aleurone. Now, we know this is due to a
small nonautonomous TE, <a
href="http://link.springer.com/article/10.1007/BF00339723">rDt</a>, in
an anthocyanin pigment producing gene. Experiments in the 1980’s showed
that unlike other elements like Ac and Mu which stop transposing once
there are high copy numbers in the genome, Dotted just keeps jumping
more no matter how many copies introduced into one background. This
project would include genotyping Dotted insertions across lines, and
testing for their presence and transposition in a number of controlled
crosses.</p>
</div>
<div id="characterizing-the-genomic-niche-of-a-retrotransposon-family"
class="section level3">
<h3>Characterizing the genomic niche of a retrotransposon family:</h3>
<p>The maize genome is a majority DNA derived from LTR retrotransposons,
but there is a stunning diversity of types and ecological strategies of
these TEs. Simply knowing a sequence is from an LTR TE doesn’t tell you
much. This project would entail characterizing every insertion in the
genome of a small family of LTR retrotransposon (~10-30 insertions), and
identifying whether they are present in the same positions in other
individuals. This would be done through a combination of bioinformatic
(e.g. <a href="https://github.com/JinfengChen/RelocaTE2">relocaTE</a>,
<a href="https://github.com/ListerLab/TEPID">TEPID</a>, <a
href="https://github.com/bergmanlab/mcclintock">mcclintock</a>) and
wet-lab techniques to verify position in different genotypes. This would
address whether TEs that always insert into the 5’ UTR of one maize
inbred line do this in every inbred line, or whether the ecological
environment of another genome disrupts the insertion preference of the
TE. A somewhat similar approach is described <a
href="http://www.maydica.org/articles/54_417.pdf">here</a>.</p>
</div>
<div id="molecular-evolution-of-te-proteins" class="section level3">
<h3>Molecular evolution of TE proteins:</h3>
<p>We have evidence of activity of some TE families in maize, but not
all. Is this due to innate deficiencies in their protein coding sequence
for transposon proteins? This project would mine genomic sequences to
identify TE protein coding domains, and use population genetic methods
to measure the extent of purifying and positive selection acting on
individual families of TEs. There are lots of ways to take this, but I
like the theoretical framework proposed <a
href="http://gbe.oxfordjournals.org/content/5/1/77.full">here</a>.</p>
</div>
</div>
<div id="tree-genetics" class="section level2">
<h2>Tree Genetics</h2>
<div id="identify-of-locus-controlling-bittersweet-kernel-sk-in-almond"
class="section level3">
<h3>Identify of locus controlling bitter/sweet kernel (Sk) in
almond</h3>
<p>In almond breeding one of the first phenotypes evaluated is whether
an almond tree produces sweet or bitter kernels. Obviously only those
genotypes with sweet kernels are advanced for further evaluation and
possibly for variety development. The kernel sweetness is controlled in
a dominant/co-dominant manner at the Sk locus with the sweet kernel
being dominant. (Literature indicates that the sweet kernel allele may
be mostly dominant and produce a bittersweet kernel phenotype when in
combination with the bitter kernel allele but distinction between
bittersweet and bitter phenotypes is unclear.) Markers with some linkage
to the phenotype have been identified, identifying an approximately 2 Mb
region in the peach genome. Identifying the genetic control of the Sk
locus and alleles contributing to bitter and sweet kernels would be of
great benefit to almond breeding programs and could be used to
investigate the origin and evolution of the sweet kernel allele(s) in
almond.</p>
</div>
</div>
<div id="inversions" class="section level2">
<h2>Inversions</h2>
<p>When chromosomes break, sometimes the cell’s repair machinery doesn’t
work as accurately as expected, and occasionally the broken chunk of
chromosome flips around before it is stuck back in place. These changes,
called inversions, can play an important role in evolutionary genetics,
such as disrupting gene function via breakage, modifying gene action by
rearranging regulatory sequence, suppressing recombination and fomenting
adaptive divergence and introgression. Many cool examples of inversions
have been identified in the nature populations such as perenniality in
<em>Mimulus</em>, mating behavior in birds, marine and fresh water
divergence in sticklebacks, and mimicry polymorphism of wing patterns in
butterflies.</p>
<p>So, what about inversions in maize and maize wild relative teosinte?
So far, we have identified nearly 130Mb of <em>Zea</em> inversions,
which is nearly the entire genome length of the <em>Arabidopsis</em>.
The biggest inversion identified so far on Chr1 is about 50Mb. This
inversion identified in teosinte an altitudinal cline and associates
with phenotype culm diameter, a trait difference between maize and
teosinte. When we surveyed a handful of teosinte populations using
relatively low-density genotyping, we easily pick up several
megabase-scale inversions. Of course, smaller inversions will be harder
to detect, but also less likely to be disrupted by crossing over and
probably much more abundant in the genome. So far, we have developed an
R package to identify inversions based on genotype data. The plan is to
use this package to identify novel inversions across a large diverse
panel of inbred lines, then verify our approach using population
genetics and comparison to several fully sequenced genomes.</p>
</div>
<div id="adaptation-to-uv-radiation" class="section level2">
<h2>Adaptation to UV radiation</h2>
<p>In order to photosynthesize enough to survive, plants must be able to
capture energy from solar radiation while avoiding the harmful effects
of UV radiation that is also present. Models of future climates suggest
that, as in current high elevation environments, increase UV radiation
will be damaging to plant growth and lead to decreased plant
productivity and increased mortality. These fitness reductions have
obvious consequences for agricultural output and the long term stability
of natural plant populations. Domesticated maize and its close relatives
the teosintes have been shown to have a number of adaptations to
mitigate UV damage like an increase in chemical sunscreen compounds as
well as a dense layer of hair on stems and leaves. While the genes that
control many of these traits have been characterized by coarse genetic
surveys of mutant plants, the specific genetic changes that have been
naturally selected to confer these advantages are still unknown. In this
experiment we will survey high elevation maize and teosinte plants’
ability to avoid the harmful effects of UV damage in leaves and pollen.
This data, combined with sequencing data from these same plants, will
allow us to characterize genetic predictors of adaptive UV resistance.
With this knowledge we will be able to characterize the size and scope
of UV adaptive responses in high elevation maize genomes and begin to
generate improvement strategies to meet the demands of agricultural
output in changing climates.</p>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
