<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <link rel="icon" type="image/ico" href="http://www.rilab.org/favicon.ico"/>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="research.html#adaptation"> <img src="http://www.rilab.org/images/teosinte.jpg" style="width: 150px;"></a> <a href="research.html#experimental"> <img src="http://www.rilab.org/images/corn.jpg" style="width: 150px;"></a> <a href="research.html#csomes"> <img src="http://www.rilab.org/images/csomes.jpg" style="width: 150px;"></a></p>
  <h4 id="home-people-research-publications-lab-docs-r.e.h.a.b.-news-press"><a href="index.html">Home</a> || <a href="people.html">People</a> || <a href="research.html">Research</a> || <a href="pubs.html">Publications</a> || <a href="https://github.com/RILAB/lab-docs">Lab Docs</a> || <a href="rehab.html">R.E.H.A.B.</a> || <a href="news.html">News</a> || <a href="press.html">Press</a></h4>
  </body>
  </html>
</head>
<body>
<h1 id="current-undergraduate-research-opportunities">Current Undergraduate Research opportunities</h1>
<h2 id="transposable-elements">Transposable Elements</h2>
<h3 id="expression-of-transposable-elements-tes-in-different-tissues">Expression of transposable elements (TEs) in different tissues:</h3>
<p>Transcription is a vital step in the process of transposition, as TE encoded proteins need to be generated. Although RNA-seq datasets exist for a large number of tissues and developmental stages in maize, particularly the inbred line B73, most analysis pipelines focus only on protein coding genes. This project would use <a href="https://pachterlab.github.io/kallisto/about">Kallisto</a>, a relatively new program for quickly measuring gene expression differences, to quantify how much transcription is occurring for a given TE copy or TE family. For a TE to make it to the next generation, it has to transpose in tissue that will become pollen or eggs, so we would focus on these tissues and their precursors. Candidate TE families would be Dotted, Ac, and a number of LTR retrotransposon families. It would be neat to follow up on a result from Kunze that has never been published, of increased transcription of Ds when Ac is present in the genome. This seems counter-intuitive to host control of a TE.</p>
<h3 id="b73-polymorphism">B73 polymorphism:</h3>
<p>The maize genome consists of over 85% TEs, and 75% LTR retrotransposons, but little is known about the transpositional activity of LTR retrotransposons. Only one LTR TE (bs1) has been observed to jump. This project would use short read resequencing of B73 to identify candidate TE regions that appear absent across accessions of the inbred line B73. Although B73 is often thought of as a completely inbred stock, differences have accumulated, as labs across the world grow these plants, and generate their <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0157942">own stocks</a>. Transposition may contribute to differences between these stocks.</p>
<h3 id="identifying-the-autonomous-partner-to-rdt">Identifying the autonomous partner to rDt:</h3>
<p><a href="http://www.genetics.org/content/23/4/377.long">Dotted</a> is a DNA TE discovered by Marcus Rhoades in the 1940’s, named such because it generates small dots on the aleurone. Now, we know this is due to a small nonautonomous TE, <a href="http://link.springer.com/article/10.1007/BF00339723">rDt</a>, in an anthocyanin pigment producing gene. Experiments in the 1980’s showed that unlike other elements like Ac and Mu which stop transposing once there are high copy numbers in the genome, Dotted just keeps jumping more no matter how many copies introduced into one background. This project would include genotyping Dotted insertions across lines, and testing for their presence and transposition in a number of controlled crosses.</p>
<h3 id="characterizing-the-genomic-niche-of-a-retrotransposon-family">Characterizing the genomic niche of a retrotransposon family:</h3>
<p>The maize genome is a majority DNA derived from LTR retrotransposons, but there is a stunning diversity of types and ecological strategies of these TEs. Simply knowing a sequence is from an LTR TE doesn’t tell you much. This project would entail characterizing every insertion in the genome of a small family of LTR retrotransposon (~10-30 insertions), and identifying whether they are present in the same positions in other individuals. This would be done through a combination of bioinformatic (e.g. <a href="https://github.com/JinfengChen/RelocaTE2">relocaTE</a>, <a href="https://github.com/ListerLab/TEPID">TEPID</a>, <a href="https://github.com/bergmanlab/mcclintock">mcclintock</a>) and wet-lab techniques to verify position in different genotypes. This would address whether TEs that always insert into the 5’ UTR of one maize inbred line do this in every inbred line, or whether the ecological environment of another genome disrupts the insertion preference of the TE. A somewhat similar approach is described <a href="http://www.maydica.org/articles/54_417.pdf">here</a>.</p>
<h3 id="molecular-evolution-of-te-proteins">Molecular evolution of TE proteins:</h3>
<p>We have evidence of activity of some TE families in maize, but not all. Is this due to innate deficiencies in their protein coding sequence for transposon proteins? This project would mine genomic sequences to identify TE protein coding domains, and use population genetic methods to measure the extent of purifying and positive selection acting on individual families of TEs. There are lots of ways to take this, but I like the theoretical framework proposed <a href="http://gbe.oxfordjournals.org/content/5/1/77.full">here</a>.</p>
<h2 id="tree-genetics">Tree Genetics</h2>
<h3 id="identify-of-locus-controlling-bittersweet-kernel-sk-in-almond">Identify of locus controlling bitter/sweet kernel (Sk) in almond</h3>
<p>In almond breeding one of the first phenotypes evaluated is whether an almond tree produces sweet or bitter kernels. Obviously only those genotypes with sweet kernels are advanced for further evaluation and possibly for variety development. The kernel sweetness is controlled in a dominant/co-dominant manner at the Sk locus with the sweet kernel being dominant. (Literature indicates that the sweet kernel allele may be mostly dominant and produce a bittersweet kernel phenotype when in combination with the bitter kernel allele but distinction between bittersweet and bitter phenotypes is unclear.) Markers with some linkage to the phenotype have been identified, identifying an approximately 2 Mb region in the peach genome. Identifying the genetic control of the Sk locus and alleles contributing to bitter and sweet kernels would be of great benefit to almond breeding programs and could be used to investigate the origin and evolution of the sweet kernel allele(s) in almond.</p>
<h2 id="inversions">Inversions</h2>
<p>When chromosomes break, sometimes the cell’s repair machinery doesn’t work as accurately as expected, and occasionally the broken chunk of chromosome flips around before it is stuck back in place. These changes, called inversions, can play an important role in evolutionary genetics, such as disrupting gene function via breakage, modifying gene action by rearranging regulatory sequence, suppressing recombination and fomenting adaptive divergence and introgression. Many cool examples of inversions have been identified in the nature populations such as perenniality in <em>Mimulus</em>, mating behavior in birds, marine and fresh water divergence in sticklebacks, and mimicry polymorphism of wing patterns in butterflies.</p>
<p>So, what about inversions in maize and maize wild relative teosinte? So far, we have identified nearly 130Mb of <em>Zea</em> inversions, which is nearly the entire genome length of the <em>Arabidopsis</em>. The biggest inversion identified so far on Chr1 is about 50Mb. This inversion identified in teosinte an altitudinal cline and associates with phenotype culm diameter, a trait difference between maize and teosinte. When we surveyed a handful of teosinte populations using relatively low-density genotyping, we easily pick up several megabase-scale inversions. Of course, smaller inversions will be harder to detect, but also less likely to be disrupted by crossing over and probably much more abundant in the genome. So far, we have developed an R package to identify inversions based on genotype data. The plan is to use this package to identify novel inversions across a large diverse panel of inbred lines, then verify our approach using population genetics and comparison to several fully sequenced genomes.</p>
<h2 id="adaptation-to-uv-radiation">Adaptation to UV radiation</h2>
<p>In order to photosynthesize enough to survive, plants must be able to capture energy from solar radiation while avoiding the harmful effects of UV radiation that is also present. Models of future climates suggest that, as in current high elevation environments, increase UV radiation will be damaging to plant growth and lead to decreased plant productivity and increased mortality. These fitness reductions have obvious consequences for agricultural output and the long term stability of natural plant populations. Domesticated maize and its close relatives the teosintes have been shown to have a number of adaptations to mitigate UV damage like an increase in chemical sunscreen compounds as well as a dense layer of hair on stems and leaves. While the genes that control many of these traits have been characterized by coarse genetic surveys of mutant plants, the specific genetic changes that have been naturally selected to confer these advantages are still unknown. In this experiment we will survey high elevation maize and teosinte plants’ ability to avoid the harmful effects of UV damage in leaves and pollen. This data, combined with sequencing data from these same plants, will allow us to characterize genetic predictors of adaptive UV resistance. With this knowledge we will be able to characterize the size and scope of UV adaptive responses in high elevation maize genomes and begin to generate improvement strategies to meet the demands of agricultural output in changing climates.</p>
</body>
</html>
