<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="research.html#corn"> <img src="http://www.rilab.org/images/corn.jpg" style="width: 100px;"></a> <a href="research.html#csomes"> <img src="http://www.rilab.org/images/csomes.jpg" style="width: 100px;"></a> <a href="research.html#teosinte"> <img src="http://www.rilab.org/images/teosinte.jpg" style="width: 100px;"></a> <a href="research.html#domestication"> <img src="http://www.rilab.org/images/domestication.jpg" style="width: 100px;"></a> <a href="research.html#tes"> <img src="http://www.rilab.org/images/tes.jpg" style="width: 100px;"></a> <a href="research.html#trees"> <img src="http://www.rilab.org/images/trees.jpg" style="width: 100px;"></a></p>
  <h4 id="home-people-research-publications-lab-docs-r.e.h.a.b.-news-press"><a href="index.html">Home</a> || <a href="people.html">People</a> || <a href="research.html">Research</a> || <a href="pubs.html">Publications</a> || <a href="https://github.com/RILAB/lab-docs">Lab Docs</a> || <a href="rehab.html">R.E.H.A.B.</a> || <a href="news.html">News</a> || <a href="press.html">Press</a></h4>
  </body>
  </html>
</head>
<body>
<p><a href="http://onlinelibrary.wiley.com/doi/10.1111/tpj.12537/abstract">Hawkins et al.</a> evaluate patterns of allele-specific expression in 8 genes around the well-studied bronze locus in maize. They show evidence for allelic differences in tissue-specific expression, and variation of expression dominance among tissues for a subset of loci.</p>
<p>The introduction includes a number of questionalbe statements about rates of change (see <a href="http://www.ncbi.nlm.nih.gov/pubmed/20666839">Purugganan and Fuller</a> for some data on the matter), the relative importance of point mutations vs transposable elements in creating functional variation, and the utility of maize as a model for speciation. We disagreed with most of these claims, and felt they needed more discussion and better citation.</p>
<p>The authors measure expression for 8 genes in four different tissues (root, leaf, immature ear, immature tassel). We were a bit concerned that their variation in technical reps (from the same ground tissue) varied more (14%) than did biological variation (12%), and made us a bit dubious about the claim of “exceptional statistical power” in the discussion.</p>
<p>A number of places the paper could use clarification. For example, they claim that:</p>
<p>“Our results indicate that allelic expression is more highly variable that that described previously for average levels of allelic expression variability across the maize genome.”</p>
<p>This would be a nice claim to make, but it is unclear what they are comparing too, as there is no citation or further discussion. Perhaps more worryingly, as it is central to the main claim of the title, is the statement:</p>
<p>“Perhaps the most notable finding from the present study is the direct correlation between recombination frequency and the extent of allelic expression variation.”</p>
<p>The authors find that the greates allelic variation (of 8 genes) is found in a region that had been previously identified as a hotspot. If we interpreted this correctly, it essentially provides a sample size of n=2 (they also show low levels of variation in a single cold spot) such that no meaningful correlation can be done. Though we haven’t dug into databases to verify this, it seemed to us that genetic maps and genome-wide expression data would allow a broader test of this hypothesis given existing data.</p>
<p>In the end, we found that the reported variation in allele-specific expression was interesting, but will wait for a genome-wide analysis before believing any of the recombination results.</p>
</body>
</html>
