<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="research.html#corn"> <img src="http://www.rilab.org/images/corn.jpg" style="width: 100px;"></a> <a href="research.html#csomes"> <img src="http://www.rilab.org/images/csomes.jpg" style="width: 100px;"></a> <a href="research.html#teosinte"> <img src="http://www.rilab.org/images/teosinte.jpg" style="width: 100px;"></a> <a href="research.html#domestication"> <img src="http://www.rilab.org/images/domestication.jpg" style="width: 100px;"></a> <a href="research.html#tes"> <img src="http://www.rilab.org/images/tes.jpg" style="width: 100px;"></a> <a href="research.html#trees"> <img src="http://www.rilab.org/images/trees.jpg" style="width: 100px;"></a></p>
  <h4 id="home-people-research-publications-lab-docs-r.e.h.a.b.-news-press"><a href="index.html">Home</a> || <a href="people.html">People</a> || <a href="research.html">Research</a> || <a href="pubs.html">Publications</a> || <a href="https://github.com/RILAB/lab-docs">Lab Docs</a> || <a href="rehab.html">R.E.H.A.B.</a> || <a href="news.html">News</a> || <a href="press.html">Press</a></h4>
  </body>
  </html>
</head>
<body>
<p><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003661">Chari and Dworkin</a> seek to better our understanding of how genetic background plays a role in genetic interactions.  They take two varieties of fruit flies, each with an identical allele, sdE3, at a locus important for wing phenotype, and cross them to a deletion panel.  Despite having the same allele, each line in the panel has a very different wing phenotype due to their unique genetic backgrounds.  They observe how each deletion impacts wing phenotype, and find that 74% of deletions with consequences are genetic background dependent.  Interestingly, a particular deletion in one line can act as an enhancer of the wing phenotype while that same deletion is a suppressor in the other genetic background (that is probably the coolest part).</p>
<p>I believe the research questions here are very interesting, particularly to the fields of personalized medicine and crop breeding, where the success of a medicine or cross will depend on our ability to predict how they interact with a given genome.  I think fruit flies are a great way to investigate genetic background interactions.  However, the use of a large effect allele makes me wonder how applicable these findings are to other genes of smaller consequence.  The authors selected an allele of a gene that has a huge impact on phenotype, and the gene itself is likely very central to the wing developmental network.  I wonder what role, if any, the centrality of the pathway has on how much of the genome interacts with our allele of interest.  After all, I would think that any other gene that stunts overall body growth is likely to also have an impact on wing development without actually interacting directly with the sd pathway.  Variation in nature is much less drastic than the sdE3 allele, and I would be curious to see if genetic background makes much less of an impact on genes less central to a network.</p>
<p>Unfortunately, the writing of the paper also frustrated me.  I am a plant biologist, not a fly person, and was lost with the quantity of specific background the authors assumed readers would know.  Admittedly, I didn’t dive into the supplement or the works cited, but the writing failed to inform me about some basic facts about these fly lines.  For example, how much genetic differentiation is there between the 2 lines used?  Do they have copy number variation in genes associated with the wing phenotype network?  Could we expect that the deletion lines would better complement one of the two varieties than the other?</p>
<p>Another interesting outcome of the study was that deletions impacted phenotype in one variety much more than the other.  They called this phenomenon robustness.  I would be very interested in hearing what the authors think could account for the difference in robustness between the fly lines.  My own storytelling leans toward network complexity and extra or substitute copies of genes in the more robust variety.</p>
<p>Though I remain skeptical of the broad brush the authors used in painting their findings to all of genetics, I found the paper useful for my own conclusions about different varieties having different network architectures of very important traits. If networks can be jostled, reshaped, and interact uniquely in flies, what does that mean for a plant genome with huge structural variability?  </p>
</body>
</html>
