<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <link rel="shortcut icon" href="favicon.ico">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="research.html#adaptation"> <img src="http://www.rilab.org/images/teosinte.jpg" style="width: 150px;"></a> <a href="research.html#experimental"> <img src="http://www.rilab.org/images/corn.jpg" style="width: 150px;"></a> <a href="research.html#csomes"> <img src="http://www.rilab.org/images/csomes.jpg" style="width: 150px;"></a></p>
  <h4 id="home-people-research-publications-lab-docs-r.e.h.a.b.-news-press"><a href="index.html">Home</a> || <a href="people.html">People</a> || <a href="research.html">Research</a> || <a href="pubs.html">Publications</a> || <a href="https://github.com/RILAB/lab-docs">Lab Docs</a> || <a href="rehab.html">R.E.H.A.B.</a> || <a href="news.html">News</a> || <a href="press.html">Press</a></h4>
  </body>
  </html>
</head>
<body>
<p><a href="http://www.pnas.org/content/110/19/E1743.full?sid=bfb4a537-ab08-4c97-811f-cd0b63ba9e9a">Yeaman</a> investigates the commonly observed “<a href="http://scholar.google.com/scholar?hl=en&amp;q=genomic+islands+of+divergence&amp;btnG=&amp;as_sdt=1%2C5&amp;as_sdtp=">genomic islands of divergence</a>”, a phenomenon that occurs when highly divergent alleles between adapted populations are found in relatively large, and potentially linked, clusters of loci.</p>
<p>Yeaman begins with a relatively thorough introduction that includes a description of four different mechanisms that can lead to genomic islands of divergence. However, he chooses to investigate only two of these mechanisms as a potential explanation for genomic islands of divergence: 1) divergence hitchhiking, and 2) the evolution of genome architecture, which he describes as genomic rearrangements that may result in tight linkage between locally adaptive loci. Yeaman acknowledges that this may only provide a partial explanation of the phenomenon (two of the four mechanisms are not explored). This is important to keep in mind while reading the paper—the models and results given provide a plausible explanation for genomic islands of divergence, though they don’t necessarily rule out others as equally (or perhaps more) relevant.</p>
<h4 id="part-i-divergence-hitchhiking">Part I: Divergence Hitchhiking</h4>
<p>Yeaman points out that the divergence hitchhiking explanation for genomic islands of divergence should lead to an enrichment of clustered adaptive loci relative to unlinked adaptive loci. He estimates the divergence hitchhiking advantage by modeling the establishment probability of advantageous alleles that are linked to a strongly-selected focal locus, relative to the establishment probability of advantageous alleles that are unlinked to that locus. He shows that the advantage is generally minor (less than three-fold), except in the rare situation when migration is high relative to the strength of selection on the linked mutation. The advantage that Yeaman models is the average over all potentially linked mutations, from 0.001 cM to 50 cM. An interesting analysis that was not included would have been the same averaged over distances that are not so large. i.e., would the advantage be as minor of only loci up to, say, 10 cM distal from the focal locus were included in the average?</p>
<p>Using the result above, a rough approximation is provided which suggests that, for threespine stickleback, ~13 unlinked alleles should be established in a population for every clustered allele that establishes. This is because, roughly speaking, with only a threefold advantage for linked loci, these would only be expected to be established as often as unlinked loci if there are 1/3 as many of them in the genome. But, for threespine stickleback (which has 21 chromosomes), the vast majority of the genome is unlinked to any given focal locus.</p>
<p>Based on these results, Yeaman asserts that divergence hitchhiking is unlikely to adequately describe genomic islands of divergence, an assertion I found moderately convincing. If the model accurately describes the underlying biological processes, the result would be tremendously strong. However, there are at least three potential shortcomings to the model. For one, as mentioned previously, by averaging over distances up to 50 cM distal from the focal locus, the establishment probability for loci in a tighter cluster is certainly underestimated. Secondly, Yeaman acknowledges that he only considers a single focal locus, rather than several. A verbal argument for why this is unlikely to dramatically impact conclusions is provided, and it does have some merit, but the argument would be stronger if additional focal loci were explicitly included in the model. Finally, Yeaman’s analysis considers a strongly selected focal locus coupled with weakly selected linked loci. A more thorough modeling may have considered combinations thereof, such as pairs of moderately selected focal loci coupled with weakly selected linked loci located between them.</p>
<h4 id="part-ii-evolution-of-genome-architecture">Part II: Evolution of genome architecture</h4>
<p>After providing a compelling argument for why divergence hitchhiking is unlikely to explain genomic islands of divergence, Yeaman investigates the evolution of genome architecture, or mutations that rearrange the genome, as a potential explanation. This analysis begins with an elegant simplified model, and then expands to include a simulation approach with fewer assumptions. The model estimates the time required for a population to evolve a clustered architecture, which depends on the genome size, population size, migration rate, rearrangement rate, and length of completely linked genomic segments. Yeaman emphasizes that the model, or heuristic, is only an approximation that should not be expected to give exact values. He uses the model to demonstrate that clustering can occur in a relatively short time, given the right model parameters. It is difficult to judge how realistic these parameters are, however, especially the values for the rearrangement rate. Still, Yeaman has more or less proven that under certain scenarios, genome architecture can evolve as a result of divergent selection, and that this can lead to genomic islands of divergence.</p>
<p>Without going into great depth, the simulations that Yeaman performs agree with the model, and allow additional insights. For instance, the simulations demonstrate that even when environments fluctuate in time and space (and therefore so does selection), genome architecture may still evolve. Among my fellow R.E.H.A.B. participants, the biggest complaint with the simulations is that they are based on a cut-and-paste method to approximate genomic rearrangements. Whether or not this reasonably reflects how genome architecture actually evolves is left to speculation. In fact, Yeaman describes six known mechanisms for modifications of genome architecture – four of which are at least somewhat approximated by cut-and-paste – but highlights that the relative abundance of each mechanism in nature is unknown. When gene duplication was simulated (rather than cut-and-paste), gene deletion also had to be included for genome architecture to evolve.</p>
<h4 id="part-iii-discussion-and-implications">Part III: Discussion and Implications</h4>
<p>Like with the introduction, Yeaman provides a thorough and fun to read discussion. The most glaring omission, however, is that there is no mention of the two mechanisms that could describe genomic islands of divergence which he did not investigate (though he acknowledged them in the introduction): increased persistence time of locally adapted alleles following secondary contact, and competition among combinations of alleles with similar fitness effects but different linkage relationships. Therefore, if one accepts the methods and conclusions, this paper provides a plausible and well justified explanation for clusters of adaptive loci. But since some explanations are untouched, the findings do not guarantee that genome evolution is the only, or even the most common, explanation. Overall, this paper provides a positive contribution to the understanding of evolution, and it may (and should!) lead to researchers more seriously considering the evolution of genome architecture as an explanation for the clustering pattern of divergent loci that is often observed.</p>
<p><a href="http://scholar.google.com/citations?user=PHAEOXIAAAAJ&amp;hl=en&amp;oi=ao">Tim Beissinger</a> <a href="https://twitter.com/timbeissinger"><img src="http://www.rilab.org/images/Twitter_logo_blue.png" style="width: 15px;"></a></p>
</body>
</html>
