<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <title></title>
  <style type="text/css">code{white-space: pre;}</style>
  <link rel="stylesheet" href="../pandoc.css" type="text/css" />
  <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
  <html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <link rel="icon" type="image/ico" href="http://www.rilab.org/favicon.ico"/>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Style-Type" content="text/css" />
    <meta name="generator" content="pandoc" />
    <title></title>
    <style type="text/css">code{white-space: pre;}</style>
    <link rel="stylesheet" href="pandoc.css" type="text/css" />
  </head>
  <body>
  <p><a href="research.html#adaptation"> <img src="http://www.rilab.org/images/teosinte.jpg" style="width: 150px;"></a> <a href="research.html#experimental"> <img src="http://www.rilab.org/images/corn.jpg" style="width: 150px;"></a> <a href="research.html#csomes"> <img src="http://www.rilab.org/images/csomes.jpg" style="width: 150px;"></a></p>
  <h4 id="home-people-research-publications-lab-docs-r.e.h.a.b.-news-press"><a href="index.html">Home</a> || <a href="people.html">People</a> || <a href="research.html">Research</a> || <a href="pubs.html">Publications</a> || <a href="https://github.com/RILAB/lab-docs">Lab Docs</a> || <a href="rehab.html">R.E.H.A.B.</a> || <a href="news.html">News</a> || <a href="press.html">Press</a></h4>
  </body>
  </html>
</head>
<body>
<p>What is a mutation rate? We’re used to thinking of the per base mutation rate as averaged across all nucleotides in a genome, but what about the averaging that occurs across individuals in these estimates? How do differences in the genetic condition of individuals impact the mutation rate measured from them? <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002419">Sharp and Agrawal (2016)</a> analyze mutation accumulation experiments in <em>Drosophila</em> and show that mutation rate varies with the condition of the fly. They do this using fun fly tricks, primarily the ability to passage a chromosome through many generations of males without meiotic recombination. They introduce one chromosome, a ‘loaded’ chromosome, with mutations in genes known to affect fitness, but importantly, not in DNA repair or other pathways with direct influences on the mutation rate. After 52 generations, they sequence the flies, and count mutations on another focal autosome. They find that focal chromosomes that accumulated mutations in the presence of the low-quality ‘loaded’ chromosome had no differences in the nucleotide substitution rate or rates of transposition, but had lower rates of gene conversion, and higher rates of small indels than focal chromosomes spending the experiment in high-quality genetic backgrounds with ‘unloaded’ chromosomes. Although they manipulated one or two genes on the ‘loaded’ chromsomes, these flies had lower body mass, which they interpret as a general correlate of condition or fitness. Altogether, this produces an elegant model where low condition flies are utilizing quicker, less energy efficient DNA repair pathways, slamming broken chromsomes together using nonhomologous end joining, while high quality flies have the luxury of waiting and repairing chromosome breaks with higher fidelity homology, resulting in gene conversion.</p>
<p>While Sharp and Agrawal raise the possibility in their introduction of different quality flies utilizing DNA polymerases with variable fidelity during DNA replication, it seems like most of the results focus on repair pathways. In light of their observations, this focus makes sense, and differences in repair explain a lot, but it still seems like replication could be impacted as well. As an audience of non-drosophilists, we wondered whether the condition of the flies impacted the amount of time spent in different phases of the cell cycle. If cell number and body size are correlated, then it seems like cells in small flies could be descendents of fewer mitoses and would have spent less time with a replicated homologous chromosome for HR. But these were mostly the rambling thoughts of some plant biologists who have been thinking too much about cell size and cell production rates in maize.</p>
<p>Also, as maize biologists, we keyed in on the lack of differences in transposition based on condition of the flies. While the <a href="http://www.nature.com/nature/journal/v461/n7267/full/nature08479.html">activation</a> <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004115">of</a> <a href="http://www.karger.com/Article/FullText/84957">certain</a> <a href="http://www.sciencedirect.com/science/article/pii/S0378111913000759">transposable</a> <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004915">elements</a> <a href="http://science.sciencemag.org/content/226/4676/792">under</a> <a href="http://www.sciencedirect.com/science/article/pii/S0960982202006383">stress</a> is unequivocal in plants, the evidence is less clear in Drosophila. We thought about their lack of TE increase in these lines, and whether we would expect it. At the time, I couldn’t remember the specifics of Drosophila TEs, but <a href="http://www.nature.com/hdy/journal/v108/n5/full/hdy201189a.html">Guerreiro (2012)</a> states pretty clearly that while “… there is a quasi-unanimous opinion about increases of transposition associated to abiotic and biotic stresses in plants, it is not the case in Drosophila…” Despite the fact that the canonical example of genotypic differences generating bursts of transposition comes from Drosophila in the form of the P-element hybrid dysgenic system, other abiotic stresses are less consistently shown to activate transposition. To me, it seems like there’s lots of suggestions of stress activation and many experiments that generate correlations between increased TE copy number and stress, but little direct proof. But I haven’t thought too much about it in flies, and it was fun to do so.</p>
<p>We were a little annoyed at the terminology of nonsynonymous applied to indels, but maybe we should reconsider, as this was terminology used by another <a href="http://www.amnat.org/announcements/AwaAnn.html">Jasper Loftus-Hill award winner</a> we read two weeks ago in <a href="http://mbe.oxfordjournals.org/content/32/6/1436">Bailey et al 2015</a>. Is this terminology becoming more common as we’re able to move beyond SNPs in variant calling pipelines? Or maybe this terminology is more reflective of the mutation accumulation field?</p>
<p>In total, we liked the paper and the ideas it raises about differences between individuals, local adaptation and evolutionary rescue, and mutational meltdown. Although we’re always skeptical of how to know the appropriate mutation rate to apply (for example, in maize, per base pair point estimates range from <a href="http://mbe.oxfordjournals.org/content/22/11/2304.full">3.3e-8</a> to <a href="http://www.pnas.org/content/93/19/10274.full.pdf">6.5e-9</a> depending on approach), thinking about the mechanism and consequences of differences between individuals that go into any calculation are really worthy of consideration.</p>
<p>Michelle Stitzer</p>
</body>
</html>
