

# Current Undergraduate Research opportunities

## Transposable Elements

### Expression of transposable elements (TEs) in different tissues:
Transcription is a vital step in the process of transposition, as TE encoded proteins need to be generated. Although RNA-seq datasets exist for a large number of tissues and developmental stages in maize, particularly the inbred line B73, most analysis pipelines focus only on protein coding genes. This project would use Kallisto, a relatively new program for quickly measuring gene expression differences, to quantify how much transcription is occurring for a given TE copy or TE family. For a TE to make it to the next generation, it has to transpose in tissue that will become pollen or eggs, so we would focus on these tissues and their precursors. Candidate TE families would be Dotted, Ac, and a number of LTR retrotransposon families. It would be neat to follow up on a result from Kunze that has never been published, of increased transcription of Ds when Ac is present in the genome. To me, this seems counterintuitive to host control of a TE.

### B73 polymorphism:
The maize genome consists of over 85% TEs, and 75% LTR retrotransposons, but little is known about the transpositional activity of LTR retrotransposons. Only one LTR TE (bs1) has been observed to jump. This project would use short read resequencing of B73 to identify candidate TE regions that appear absent across accessions of the inbred line B73. Although B73 is often thought of as a completely inbred stock, differences have accumulated, as labs across the world grow these plants, and generate their [own stocks](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0157942). Transposition may contribute to differences between these stocks.

### Identifying the autonomous partner to rDt:
Dotted is a DNA TE discovered by Marcus Rhoades in the 1940’s, named such because it generates small dots on the aleurone. Now, we know this is due to a small nonautonomous TE, rDt in an anthocyanin pigment producing gene. Experiments in the 1980’s showed that unlike other elements like Ac and Mu which stop transposing once there are high copy numbers in the genome, Dotted just keeps jumping more no matter how many copies introduced into one background. This project would include genotyping Dotted insertions across lines, and testing for their presence and transposition in a number of controlled crosses.

### Characterizing the genomic niche of a retrotransposon family:  
The maize genome is a majority DNA derived from LTR retrotransposons, but there is a stunning diversity of types and ecological strategies of these TEs. Simply knowing a sequence is from an LTR TE doesn’t tell you much. This project would entail characterizing every insertion in the genome of a small family of LTR retrotransposon (~10-30 insertions), and identifying whether they are present in the same positions in other individuals. This would be done through a combination of bioinformatic (e.g. relocaTE, TEPID, mcclintock)  and wet-lab techniques to verify position in different genotypes. This would address whether TEs that always insert into the 5’ UTR of one maize inbred line do this in every inbred line, or whether the ecological environment of another genome disrupts the insertion preference of the TE. A somewhat similar approach is described here.

### Molecular evolution of TE proteins:

We have evidence of activity of some TE families in maize, but not all. Is this due to innate deficiencies in their protein coding sequence for transposon proteins? This project would mine genomic sequences to identify TE protein coding domains, and use population genetic methods to measure the extent of purifying and positive selection acting on individual families of TEs. There are lots of ways to take this, but I like the theoretical framework proposed here.

## Tree Genetics

### Identification of locus controlling bitter/sweet kernel (Sk) in almond

In almond breeding one of the first phenotypes evaluated is whether an almond tree produces sweet or bitter kernels. Obviously only those genotypes with sweet kernels are advanced for further evaluation and possibly for variety development. The kernel sweetness is controlled in a dominant/co-dominant manner at the Sk locus with the sweet kernel being dominant. (Literature indicates that the sweet kernel allele may be mostly dominant and produce a bittersweet kernel phenotype when in combination with the bitter kernel allele but distinction between bittersweet and bitter phenotypes is unclear.) Markers with some linkage to the phenotype have been identified, identifying an approximately 2 Mb region in the peach genome. Identifying the genetic control of the Sk locus and alleles contributing to bitter and sweet kernels would be of great benefit to almond breeding programs and could be used to investigate the origin and evolution of the sweet kernel allele(s) in almond.

##### Proposed methodology:
Use VCF from P. dulcis to identify variants in the region of interest between almond varieties identified as homozygous for either the sweet kernel or bitter kernel genotype (two each). Then identify which, if any, of these variants correlate with sweet/bitter in other almond varieties. Determine candidate genes in close proximity to the variants of interest, identify their function(s) and how it/they may contribute to sweetness/bitterness phenotype.

##### Breeding aspect:
Develop a simple genotyping method, e.g. CAPS, and validate co-segregation with the phenotype. Then use with an expanded set of germplasm with known phenotypes to further validate utility.

##### Evolution aspect:
Utilize genomic sequence in the candidate region of different available species in genus/subgenus to estimate when sweetness may have evolved and/or been selected in almond and how this timing fits with domestication history or other events. (Sweet kernels are found in apricot, peach, and possibly other Prunus so, like S-alleles, it may be another trans-specific evolution situation. This section not well-developed at this point.)
