<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title></title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>




<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <img alt="Brand" src="images/cornkey_icon.png">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      
      </button>
      
      <a class="navbar-brand" href="index.html">Ross-Ibarra Lab</a>
    </div>
    
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="people.html">People</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="pubs.html">Publications</a></li>
        <li><a href="https://github.com/RILAB/lab-docs">Lab Docs</a></li>
        <li><a href="rehab.html">R.E.H.A.B</a></li>
        <li><a href="news.html">News</a></li>
        <li><a href="press.html">Press</a></li>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<div id="research" class="section level2">
<h2>Research</h2>
<hr>
<p><a name="adaptation"></a> <img src="images/teosinte.jpg" /></p>
<div id="genetics-of-adaptation" class="section level3">
<h3>Genetics of adaptation</h3>
<p>Maize spread rapidly after domestication, adapting to a wide range of environments. Today maize is grown across a broader geographic breadth than any of the world’s other staple crops, from sea level to altitudes of <span class="math inline">\(&gt;4,000\)</span>m and from deserts to near-flooded conditions. The wild relatives of maize have also adapted to environments varying widely in elevation, temperature, and moisture availability. Our previous work has shown that adaptation in teosinte is often restricted to discrete local populations and has often made use of regulatory variation. We also find evidence that inversion polymorphisms are common and associated with environmental gradients and phenotypic variation. Because maize – and likely other complex plant genomes – can apparently circumvent cytological problems associated with large-scale structure variation, we suspect such variants are common in plant genomes. Work is underway both to characterize the genomic and phenotypic effects of individual inversions and to more broadly characterize the scope of structural variation within natural populations using both resequencing and <em>de novo</em> genome assembly.</p>
<p>In many instances of local adaptation, independent populations have converged on similar phenotypic adaptations. We have worked with maize populations adapted to high elevation environments in Mexico and South America, seeking to understand whether convergent phenotypic evolution is associated with convergent changes at the molecular level. Our previous efforts pointed to a key role for adaptive introgression from wild teosinte in enabling maize to adapt to the mountains of central Mexico, but in spite of similar phenotypes we find no overlap between Mexican and Andean populations in genes showing evidence of selection. Our analyses further suggest that most local adaptation in these populations has not been mutation limited, but that instead stochastic differences in local founding populations, each with abundant standing genetic variation, likely explain the lack of convergent genetics.</p>
<ul>
<li><p><a href="http://www.genetics.org/content/200/4/1297">Independent molecular basis of convergent highland adaptation in maize</a><br />
<strong>Takuno S</strong>, Ralph P, Swarts K, Elshire RJ, Glaubitz JC, Buckler ES, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>.<br />
Genetics 200:1297-1312</p></li>
<li><p><a href="http://www.rilab.org/pdfs/fonseca2015.pdf">The origin and evolution of maize in the American Southwest</a><br />
Fonseca RR, Smith B, Wales N, Cappellini E …[12 authors]… Hufford MB, Albrechtsen A, <strong>Ross-Ibarra J</strong>, Gilbert MT.<br />
<em>Nature Plants</em> 1</p></li>
<li><p><a href="https://peerj.com/articles/900/">Natural variation in teosinte at the domestication locus teosinte branched1 (tb1)</a><br />
<strong>Vann LE</strong>, <strong>Kono TJ</strong>, <strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>.<br />
PeerJ 3:e900</p></li>
<li><p><a href="http://www.rilab.org/pdfs/tiffin2014.pdf">Advances and limits of using population genetics to understand local adaptation</a><br />
Tiffin P, <strong>Ross-Ibarra J</strong><br />
<em>Trends in Ecology and Evolution</em> 29:673-680</p></li>
<li><p><a href="http://gbe.oxfordjournals.org/content/5/9/1594">Complex patterns of local adaptation in teosinte</a><br />
<strong>Pyhäjärvi T</strong>, <strong>Hufford MB</strong>, <strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong><br />
<em>Genome Biology and Evolution</em> 5:1594-1609 [<a href="http://www.oxfordjournals.org/our_journals/gbe/resource/gbe-cover_dec-2013_expansion.jpg">cover</a>]</p></li>
<li><p><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003477">The genomic signature of crop-wild introgression in maize</a><br />
<strong>Hufford MB</strong>, Lubinsky P, <strong>Pyhäjärvi T</strong>, <strong>Devengenzo MT</strong>, Ellstrand NC, <strong>Ross-Ibarra J</strong><br />
<em>PLoS Genetics</em> 9(5): e1003477</p></li>
<li><p><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy20132a.html">Diversity and abundance of the Abnormal chromosome 10 meiotic drive complex in <em>Zea mays</em></a><br />
Kanizay LB, <strong>Pyhäjärvi T</strong>, Lowry E, <strong>Hufford MB</strong>, Peterson DG, <strong>Ross-Ibarra J</strong>, Dawe RK<br />
<em>Heredity</em> 110: 570-577</p></li>
<li><p><a href="http://www.genetics.org/content/191/3/883.abstract">Megabase-scale inversion polymorphism in the wild ancestor of maize</a><br />
Fang Z, <strong>Pyhäjärvi T</strong>, Weber AL, Dawe RK, Glaubitz JC, Sianchez Gonzalez J, <strong>Ross-Ibarra C</strong>, Doebley J, Morrell PL, <strong>Ross-Ibarra J</strong> <em>Genetics</em> 191:883-894</p></li>
<li><p><a href="http://www.rilab.org/pdfs/hufford2012.pdf">Teosinte as a model system for population and ecological genomics</a><br />
<strong>Hufford MB</strong>, <strong>Bilinski P</strong>, <strong>Pyhäjärvi T</strong>, <strong>Ross-Ibarra J</strong><br />
<em>Trends in Genetics</em> 12: 606-615</p></li>
<li><p><a href="http://www.rilab.org/pdfs/Van-Heerwaarden-et-al-2010-951.pdf">Fine scale genetic structure in the wild ancestor of maize (<strong>Zea mays</strong> ssp. <strong>parviglumis</strong>)</a><br />
<strong>van Heerwaarden J</strong>, <strong>Ross-Ibarra J</strong>, Doebley J, Glaubitz JC, Sanchez Gonzalez J, Gaut BS, Eguiarte LE<br />
<em>Molecular Ecology</em> 19: 1162-1173</p></li>
<li><p><a href="http://www.rilab.org/pdfs/hufford2011.pdf">Influence of cryptic population structure on observed mating patterns in the wild progenitor of maize</a><br />
<strong>Hufford MB</strong>, Gepts P, <strong>Ross-Ibarra J</strong><br />
<em>Molecular Ecology</em> 20: 46-55</p></li>
</ul>
<hr>
<p><a name="experimental"></a> <img src="images/corn.jpg" /></p>
</div>
<div id="experimental-evolution" class="section level3">
<h3>Experimental Evolution</h3>
<p>Plant domestication and modern breeding represent examples of experimentally evolved populations. Studying these populations provides an opportunity to understand not only the genetic basis of evolutionary change but also how the processes of evolution interact to shape modern genetic and phenotypic diversity.</p>
<p>We have worked to understand selection during domestication, documenting its polygenic nature, the contributions of regulatory variation, and the importance of processes such as convergent molecular evolution and selection on standing genetic variation. Our detailed analyses of inbred lines have revealed little evidence for strong selection during modern maize breeding, instead highlighting the effect of small population sizes in partitioning diversity into distinct populations of increasingly narrow ancestry.</p>
<p>Our current work on domestication focuses on how demographic change interacts with selection to shape genetic and phenotypic diversity. Maize underwent a demographic bottleneck during domestication, reducing its effective population size and thus the efficacy of purifying selection. Purifying selection in teosinte is thus stronger due to its larger effective population size, resulting in both a deeper and wider valley of diversity around conserved genes. But maize population size grew quickly after domestication, eventually exceeding that of teosinte. New mutations in maize are thus subject to stronger selection than in teosinte, a shift reflected in patterns of variation in recent low frequency variants such as singleton polymorphisms.<br />
Population demographic change can also impact the effect size, number, and dominance of loci underlying phenotypic traits . Our future work will utilize data from large-scale association mapping studies, coupled with population genomic inference of demographic change and selection, to study how the process of domestication may have shaped the genetic architecture of phenotypic traits.</p>
<p>Building on our work highlighting the importance of deleterious alleles in phenotypic variation, we have used experimentally evolved populations to track haplotype frequencies over time and assess the genetic basis of gain in hybrid yield. Our recent analysis finds little overlap in selected haplotypes between two populations bred for increasing hybrid yield, consistent with a model for complementation of deleterious variants brought to high frequency by hitchhiking. Current efforts include work to build a pedigree of more than 4,000 maize lines with which to extend these haplotype analyses across many populations.</p>
<ul>
<li><p>Gerke JP, Edwards JW, Guill KE, <strong>Ross-Ibarra J</strong>, McMullen MD. <a href="http://arxiv.org/abs/1307.7313">The genomic impacts of drift and selection for hybrid performance in maize</a></p></li>
<li><p>Seed filling in domesticated maize and rice depends on SWEET-mediated hexose transport<br />
Sosso D, Luo D, Li Q-B, Sasse J, <strong>Yang J</strong> … (6 authors)… <strong>Ross-Ibarra J</strong>, Yang B, Frommer WB.<br />
Nature Genetics <em>Accepted for publication</em></p></li>
<li><p><a href="http://elifesciences.org/content/4/e05861">Genetic, evolutionary and plant breeding insights from the domestication of maize</a><br />
Hake S, <strong>Ross-Ibarra J</strong>.<br />
<em>eLife</em> 2015;4:e05861</p></li>
<li><p><strong>Mezmouk S</strong>, <strong>Ross-Ibarra J</strong>. <a href="http://arxiv.org/abs/1308.0380">The pattern and distribution of deleterious mutations in maize</a>.</p></li>
<li><p><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1003604">From many, one: genetic control of prolificacy during maize domestication</a><br />
Wills DM, Whipple C, <strong>Takuno S</strong>, Kursel LE, Shannon LM, <strong>Ross-Ibarra J</strong>, Doebley JF.<br />
<em>PLoS Genetics</em> [9(6): e1003604</p></li>
<li><p>Waters AJ, <strong>Bilinski P</strong>, Eichten SR, Vaugh MW, <strong>Ross-Ibarra J</strong>, Gehring M, Springer NM. <a href="http://arxiv.org/abs/1307.7678">Conservation and divergence of imprinting in maize</a>. <em>PNAS</em> <em>In Press</em></p></li>
<li><p><a href="http://media.wix.com/ugd/fe9228_24240aa930bf4728be83b4b57da55f17.pdf">Comparative population genomics of maize domestication and improvement</a><br />
<strong>Hufford MB</strong>, Xun X, <strong>van Heerwaarden J</strong>, <strong>Pyhäjärvi T</strong> …[17 authors]… Ware D, Buckler ES, Yang S, <strong>Ross-Ibarra J</strong><br />
<em>Nature Genetics</em> 44:808-811</p></li>
<li><p>Cook JP, McMullen MD, Holland JB, Tian F, Bradbury P, <strong>Ross-Ibarra J</strong>, Buckler ES, Flint-Garcia SA (2012) <a href="http://www.plantphysiol.org/content/158/2/824.full">Genetic architecture of maize kernel composition in the Nested Association Mapping and Inbred Association panels</a>. <em>Plant Physiology</em> 158: 824-834</p></li>
<li><p><strong>van Heerwaarden J</strong>, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong> (2012) <a href="http://www.pnas.org/content/early/2012/07/12/1209275109">Historical genomics of North American maize</a>. <em>PNAS</em> 109: 12420-12425</p></li>
<li><p><a href="http://www.rilab.org/pdfs/swansonwagner2012.pdf">Reshaping of the maize transcriptome by domestication</a><br />
Swanson-Wagner R, Briskine R, Schaefer R, <strong>Hufford MB</strong>, <strong>Ross-Ibarra J</strong>, Myers CL, Tiffin P, Springer NM<br />
<em>PNAS</em> 109: 11878-11883</p></li>
<li><p><a href="http://media.wix.com/ugd/fe9228_14630f30e97f4e04b8b5980632002f7a.pdf">Identification of a functional transposon insertion in the maize domestication gene <em>tb1</em></a><br />
Studer A, Zhao Q, <strong>Ross-Ibarra J</strong>, Doebley J<br />
<em>Nature Genetics</em> 43:1160-1163</p></li>
<li><p><a href="http://media.wix.com/ugd/fe9228_4916e7f7acec4aee869da040d519e0be.pdf">Genetic signals of origin, spread and introgression in a large sample of maize landraces</a><br />
<strong>van Heerwaarden J</strong>, Doebley J, Briggs WH, Glaubitz JC, Goodman MM, Sanchez Gonzalez JJ, <strong>Ross-Ibarra J</strong><br />
<em>PNAS</em> 108: 1088-1092</p></li>
<li><p><a href="http://www.nature.com/hdy/journal/v104/n1/full/hdy2009110a.html">Genetic diversity in a crop metapopulation</a><br />
<strong>van Heerwaarden J</strong>, van Eeuwijk FA, <strong>Ross-Ibarra J</strong><br />
<em>Heredity</em> [104: 28-39]</p></li>
<li><p><a href="http://www.genetics.org/content/181/4/1399.abstract">Historical divergence and gene flow in the genus Zea</a><br />
<strong>Ross-Ibarra J</strong>, Tenaillon M, Gaut BS<br />
<em>Genetics</em> 181: 1399-1413</p></li>
<li><p><a href="http://media.wix.com/ugd/fe9228_352f4ee9b69a425589e55a4bac60bc5d.pdf">A first-generation haplotype map of maize</a><br />
Gore MA, Chia JM, Elshire RJ, Sun Q …[4 authors]… Grills GS, <strong>Ross-Ibarra J</strong>, Ware DH, Buckler ES<br />
<em>Science 326</em>: 1115-1117</p></li>
<li><p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2614810/">Multiple domestications do not appear monophyletic</a><br />
<strong>Ross-Ibarra J</strong>, Gaut BS<br />
<em>PNAS</em> (Letter) 105: E105</p></li>
<li><p><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1876441/">Plant domestication, a unique opportunity to identify the genetic basis of adaptation</a><br />
<strong>Ross-Ibarra J</strong>, Morrell PL, Gaut BS<br />
<em>PNAS</em> 104</p></li>
<li><p><a href="http://www.rilab.org/pdfs/Ross-Ibarra-2005-15881692.pdf">Quantitative trait loci and the study of plant domestication</a><br />
<strong>Ross-Ibarra J</strong> (2005)<br />
<em>Genetica</em> 123: 197-204</p></li>
<li><p><a href="http://www.rilab.org/pdfs/Ross-Ibarra-2004.pdf">The evolution of recombination under domestication: a test of two hypotheses</a><br />
<strong>Ross-Ibarra J</strong><br />
<em>American Naturalist</em> 163: 105-112</p></li>
</ul>
<hr>
<p><a name="csomes"></a> <img src="images/csomes.jpg" /></p>
</div>
<div id="genome-evolution" class="section level3">
<h3>Genome Evolution</h3>
<p>In addition to discerning the genetic basis of phenotypic evolution, we are interested in understanding the processes that shape evolution of the genome itself. We have shown that differences in deletion bias can effect large changes in genome size over phylogenetic time scales and documented extensive variation in copy number across diverse maize and teosinte. Current work characterizing copy number variation in a single wild population of teosinte has revealed problems with population genetic methods that are unable to account for biologically missing data. Statistics such as Tajima’s D, for example, show a strong correlations with the frequency of deletions, potentially leading to false interpretations of selection or demographic change.</p>
<p>To better understand processes shaping genome evolution across populations, we are currently working to model genome size as a phenotype. We have developed methods to quantify the abundance of different repetitive fractions of the genome and test for selection on repeat abundance. Our initial results find evidence that overall genome size and heterochromatic knob abundance both are under selection for smaller genomes across altitudinal gradients, perhaps as a means of accelerating development and flowering.</p>
<p>While transposable elements (TEs) as a group do not appear to be under selection for genome size variation, they nonetheless make up the majority of most flowering plant genomes. We have previously shown important functional consequences of individual TE insertions on phenotype and gene expression, but we are just beginning to understand their genome-wide significance. Methods developed in other model organisms invariably fail to detect TE insertions in complex genomes such as maize, but we have developed approaches that take advantage of our <em>de novo</em> hand-curated annotation of TEs to accurately identify insertions in high-coverage resequencing data. TE polymorphism is abundant: individual lines contain hundreds of thousands of new insertions, including thousands of insertions into protein-coding genes. Analysis across a small set of lines has already revealed the impact of new insertions on gene expression and strong differences in insertion preference, methylation state, and allele frequency among different TE families. We aim to expand these analyses to include population genetic methods to identify selection on individual insertions, thus enabling us to better understand the role TEs play in driving genomic change and potentially adaptation in both natural and domesticated populations.</p>
<ul>
<li><p><a href="http://www.plosgenetics.org/article/info%3Adoi%2F10.1371%2Fjournal.pgen.1004915">Transposable Elements Contribute to Activation of Maize Genes in Response to Abiotic Stress</a><br />
Makarevitch I, Waters M, West P, <strong>Stitzer M</strong>, <strong>Ross-Ibarra J</strong>, Springer NM.<br />
<em>PLoS Genetics</em> 11(1): e1004915</p></li>
<li><p><a href="http://rilab.org/pdfs/Bilinski2014.pdf">Diversity and evolution of centromere repeats in the maize genome</a><br />
<strong>Bilinski P</strong>, <strong>Distor KD</strong>, <strong>Gutierez-Lopez J</strong>, <strong>Mendoza Mendoza G</strong>, Shi J, Dawe K, <strong>Ross-Ibarra J</strong><br />
<em>Chromosoma</em> 0009-5915</p></li>
<li><p><a href="http://genomebiology.com/2013/14/1/R10">Patterns of centromere tandem repeat evolution in 282 animal and plant genomes</a><br />
Melters DP, Bradnam KR, Young HA, Telis N …[9 authors]… Tobias C, <strong>Ross-Ibarra J</strong>, Korf IF, Chan SW-L.<br />
<em>Genome Biology</em> 14: R10</p></li>
<li><p><a href="http://www.pnas.org/content/110/48/19639.abstract">Comprehensive analysis of imprinted genes in maize reveals allelic variation and limited conservation with other species</a><br />
Waters AJ, <strong>Bilinski P</strong>, Eichten SR, Vaugh MW, <strong>Ross-Ibarra J</strong>, Gehring M, Springer NM<br />
<em>PNAS</em> 110:19639-19644</p></li>
<li><p><a href="http://www.nature.com/hdy/journal/vaop/ncurrent/full/hdy20132a.html">Diversity and abundance of the Abnormal chromosome 10 meiotic drive complex in <em>Zea mays</em></a><br />
Kanizay LB, <strong>Pyhäjärvi T</strong>, Lowry E, <strong>Hufford MB</strong>, Peterson DG, <strong>Ross-Ibarra J</strong>, Dawe RK<br />
<em>Heredity</em> 110: 570-577</p></li>
<li><p><a href="http://media.wix.com/ugd/fe9228_86a609db506740ce96c549567c246b4c.pdf">Maize HapMap2 identifies extant variation from a genome in flux</a><br />
Chia J-M, Song C, Bradbury P, Costich D …[29 authors]… Buckler ES, Zhang G, Xu Y, Ware, D<br />
<em>Nature Genetics</em> 44:803-807 [<a href="http://www.nature.com/ng/journal/v44/n7/covers/index.html">cover</a>]</p></li>
<li><p><a href="http://www.rilab.org/pdf/Munoz_Diez_R1-1.pdf">Using nextgen sequencing to investigate genome size variation and transposable element content</a><br />
Munoz Diez C, Vitte C, <strong>Ross-Ibarra J</strong>, Gaut BS, Tenaillon MI<br />
<em>Topics in Current Genetics: Plant Transposable Elements - Impact on Genome Structure &amp; Function</em> 24: 41-58</p></li>
<li><p><a href="http://gbe.oxfordjournals.org/cgi/pmidlookup?view=long&amp;pmid=21296765">Genome size and TE content as determined by high-throughput sequencing in maize and <em>Zea luxurians</em></a><br />
Tenaillon MI, <strong>Hufford MB</strong>, Gaut BS, <strong>Ross-Ibarra J</strong><br />
<em>Genome Biology and Evolution</em> 3: 219-229</p></li>
<li><p><a href="http://www.rilab.org/pdfs/Whitney-et-al-2010-402.pdf">A role for nonadaptive processes in plant genome size evolution?</a><br />
Whitney KD, Baack EJ, Hamrick JL, Godt, MJW …[3 authors]… Goodwillie C, Kalisz S, Leitch I, <strong>Ross-Ibarra J</strong><br />
<em>Evolution</em> 64: 2097-2109</p></li>
<li><p><a href="http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1000327">Widespread gene conversion in centromere cores</a><br />
Shi J, Wolf S, Burke J, Presting G, <strong>Ross-Ibarra J</strong>, Dawe RK<br />
<em>PLoS Biology</em> 8: e1000327</p></li>
<li><p><a href="http://www.rilab.org/pdfs/Ross-Ibarra_Gaut_2008.pdf">Selection on major components of angiosperm genomes</a><br />
Gaut BS, <strong>Ross-Ibarra J</strong> <em>Science</em> 320: 484-486</p></li>
<li><p><a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2006.01275.x/abstract">Genome size and recombination in angiosperms: a second look</a><br />
<strong>Ross-Ibarra J</strong><br />
<em>Journal of Evolutionary Biology</em> 20: 800-806</p></li>
</ul>
<hr>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
